--- pciutils-2.2.4~pre4.orig/lspci.c
+++ pciutils-2.2.4~pre4/lspci.c
@@ -121,7 +121,11 @@
   d->present = xmalloc(64);
   memset(d->present, 1, 64);
   if (!pci_read_block(p, 0, d->config, 64))
-    die("Unable to read the standard configuration space header");
+    {
+      fprintf(stderr, "Unable to read 64 bytes of configuration space\n");
+      return NULL;
+    }
+     
   if ((d->config[PCI_HEADER_TYPE] & 0x7f) == PCI_HEADER_TYPE_CARDBUS)
     {
       /* For cardbus bridges, we need to fetch 64 bytes more to get the
--- pciutils-2.2.4~pre4.orig/Makefile
+++ pciutils-2.2.4~pre4/Makefile
@@ -1,7 +1,7 @@
 # Makefile for The PCI Utilities
 # (c) 1998--2006 Martin Mares <mj@ucw.cz>
 
-OPT=-O2
+OPT=-O2 -g
 CFLAGS=$(OPT) -Wall -W -Wno-parentheses -Wstrict-prototypes -Wmissing-prototypes -Winline
 
 VERSION=2.2.4-pre4
@@ -34,6 +34,7 @@
 export
 
 all: $(PCILIB) lspci setpci lspci.8 setpci.8 update-pciids update-pciids.8 pci.ids
+all: pcimodules pcimodules.8
 
 $(PCILIB): $(PCIINC) force
 	$(MAKE) -C lib all
@@ -45,9 +46,11 @@
 
 lspci: lspci.o common.o $(PCILIB)
 setpci: setpci.o common.o $(PCILIB)
+pcimodules: pcimodules.o common.o $(PCILIB)
 
 lspci.o: lspci.c pciutils.h $(PCIINC)
 setpci.o: setpci.c pciutils.h $(PCIINC)
+pcimodules.o: pcimodules.c pciutils.h $(PCIINC)
 common.o: common.c pciutils.h $(PCIINC)
 
 update-pciids: update-pciids.sh
@@ -59,22 +62,26 @@
 clean:
 	rm -f `find . -name "*~" -o -name "*.[oa]" -o -name "\#*\#" -o -name TAGS -o -name core`
 	rm -f update-pciids lspci setpci lib/config.* lib/example *.8 pci.ids.*
+	rm -f pcimodules
 	rm -rf maint/dist
 
 distclean: clean
 
 install: all
 # -c is ignored on Linux, but required on FreeBSD
-	$(DIRINSTALL) -m 755 $(SBINDIR) $(IDSDIR) $(MANDIR)/man8
-	$(INSTALL) -c -m 755 -s lspci setpci $(SBINDIR)
-	$(INSTALL) -c -m 755 update-pciids $(SBINDIR)
-	$(INSTALL) -c -m 644 pci.ids $(IDSDIR)
-	$(INSTALL) -c -m 644 lspci.8 setpci.8 update-pciids.8 $(MANDIR)/man8
+	$(DIRINSTALL) -m 755 $(DESTDIR)/$(SBINDIR) $(DESTDIR)/$(IDSDIR) $(DESTDIR)/$(MANDIR)/man8
+	$(INSTALL) -c -m 755 -s lspci setpci $(DESTDIR)/$(SBINDIR)
+	$(INSTALL) -c -m 755 update-pciids $(DESTDIR)/$(SBINDIR)
+	$(INSTALL) -c -m 644 pci.ids $(DESTDIR)/$(IDSDIR)
+	$(INSTALL) -c -m 644 lspci.8 setpci.8 update-pciids.8 $(DESTDIR)/$(MANDIR)/man8
+	$(INSTALL) -c -m 755 -s pcimodules $(DESTDIR)/$(SBINDIR)
+	$(INSTALL) -c -m 644 pcimodules.8 $(DESTDIR)/$(MANDIR)/man8
 
 uninstall: all
-	rm -f $(SBINDIR)/lspci $(SBINDIR)/setpci $(SBINDIR)/update-pciids
-	rm -f $(IDSDIR)/pci.ids
-	rm -f $(MANDIR)/man8/lspci.8 $(MANDIR)/man8/setpci.8 $(MANDIR)/man8/update-pciids.8
+	rm -f $(DESTDIR)/$(SBINDIR)/lspci $(DESTDIR)/$(SBINDIR)/setpci $(DESTDIR)/$(SBINDIR)/update-pciids
+	rm -f $(DESTDIR)/$(IDSDIR)/pci.ids
+	rm -f $(DESTDIR)/$(MANDIR)/man8/lspci.8 $(DESTDIR)/$(MANDIR)/man8/setpci.8 $(DESTDIR)/$(MANDIR)/man8/update-pciids.8
+	rm -f $(DESTDIR)/$(SBINDIR)/pcimodules $(DESTDIR)/$(MANDIR)/man8/pcimodules.8
 
 get-ids:
 	cp ~/tree/pciids/pci.ids pci.ids
--- pciutils-2.2.4~pre4.orig/ChangeLog
+++ pciutils-2.2.4~pre4/ChangeLog
@@ -1,3 +1,7 @@
+2006-08-01  Martin Mares  <mj@ucw.cz>
+
+	* lib/i386-io-hurd.h: Fixed a silly typo.
+
 2006-07-30  Martin Mares  <mj@ucw.cz>
 
 	* lib/Makefile: Added a missing rule for obsd-device.o.
--- pciutils-2.2.4~pre4.orig/lib/Makefile
+++ pciutils-2.2.4~pre4/lib/Makefile
@@ -7,6 +7,7 @@
 INCL=internal.h pci.h config.h header.h sysdep.h types.h
 
 PCILIB=libpci.a
+CFLAGS += -fPIC
 
 ifdef PCI_HAVE_PM_LINUX_SYSFS
 OBJS += sysfs.o
--- pciutils-2.2.4~pre4.orig/lib/names.c
+++ pciutils-2.2.4~pre4/lib/names.c
@@ -17,8 +17,8 @@
 struct id_entry {
   struct id_entry *next;
   u32 id12, id34;
-  byte cat;
-  byte name[1];
+  byte cat;	/* Actually an enum id_entry_type */
+  char name[1];
 };
 
 enum id_entry_type {
@@ -81,7 +81,7 @@
   return h % HASH_SIZE;
 }
 
-static byte *id_lookup(struct pci_access *a, int cat, int id1, int id2, int id3, int id4)
+static byte *id_lookup(struct pci_access *a, enum id_entry_type cat, int id1, int id2, int id3, int id4)
 {
   struct id_entry *n;
   u32 id12 = id_pair(id1, id2);
@@ -95,7 +95,7 @@
   return n ? n->name : NULL;
 }
 
-static int id_insert(struct pci_access *a, int cat, int id1, int id2, int id3, int id4, byte *text)
+static int id_insert(struct pci_access *a, enum id_entry_type cat, int id1, int id2, int id3, int id4, char *text)
 {
   u32 id12 = id_pair(id1, id2);
   u32 id34 = id_pair(id3, id4);
@@ -117,7 +117,7 @@
   return 0;
 }
 
-static int id_hex(byte *p, int cnt)
+static int id_hex(char *p, int cnt)
 {
   int x = 0;
   while (cnt--)
@@ -143,8 +143,8 @@
 
 static const char *id_parse_list(struct pci_access *a, FILE *f, int *lino)
 {
-  byte line[MAX_LINE];
-  byte *p;
+  char line[MAX_LINE];
+  char *p;
   int id1=0, id2=0, id3=0, id4=0;
   int cat = -1;
   int nest;
--- pciutils-2.2.4~pre4.orig/lib/example.c
+++ pciutils-2.2.4~pre4/lib/example.c
@@ -7,7 +7,7 @@
 
 #include <stdio.h>
 
-#include "pci.h"
+#include "pci/pci.h"
 
 int main(void)
 {
--- pciutils-2.2.4~pre4.orig/lib/i386-io-hurd.h
+++ pciutils-2.2.4~pre4/lib/i386-io-hurd.h
@@ -25,15 +25,15 @@
   mach_port_t device;
 
   if ((errno = get_privileged_ports(NULL, &device)))
-    a->warn("i386-io-hurd: Can't get_privileged_ports(): %m");
+    a->warning("i386-io-hurd: Can't get_privileged_ports(): %m");
 
   if (!errno && (errno = device_open(device, D_READ | D_WRITE, "io", &io_port)))
-    a->warn("i386-io-hurd: Can't device_open(): %m");
+    a->warning("i386-io-hurd: Can't device_open(): %m");
 
   mach_port_deallocate(mach_task_self(), device);
 
   if (!errno && (errno = i386_io_port_add(mach_thread_self(), io_port)))
-    a->warn("i386-io-hurd: Can't i386_io_port_add(): %m");
+    a->warning("i386-io-hurd: Can't i386_io_port_add(): %m");
 
   return errno ? 0 : 1;
 }
@@ -42,7 +42,7 @@
 intel_cleanup_io(struct pci_access *a)
 {
   if ((errno = i386_io_port_remove(mach_thread_self(), io_port)))
-    a->warn("i386-io-hurd: Can't i386_io_port_remove(): %m");
+    a->warning("i386-io-hurd: Can't i386_io_port_remove(): %m");
 
   mach_port_deallocate(mach_task_self(), io_port);
 
--- pciutils-2.2.4~pre4.orig/pci.ids
+++ pciutils-2.2.4~pre4/pci.ids
@@ -11,7 +11,7 @@
 #	This file can be distributed under either the GNU General Public License
 #	(version 2 or higher) or the 3-clause BSD License.
 #
-#	$Id: pci.ids,v 1.54 2006/05/05 15:20:27 mares Exp $
+#	Daily snapshot on Fri 2006-08-11 01:05:07
 #
 
 # Vendors, devices and subsystems. Please keep sorted.
@@ -56,9 +56,11 @@
 045e  Microsoft
 	006e  MN-510 802.11b wireless USB paddle
 	00c2  MN-710 wireless USB paddle
+0482  Kyocera
 04cf  Myson Century, Inc
 	8818  CS8818 USB2.0-to-ATAPI Bridge Controller with Embedded PHY
 050d  Belkin
+	001a  FSD7000 802.11g PCI Wireless card
 	0109  F5U409-CU USB/Serial Portable Adapter
 	7050  F5D7050 802.11g Wireless USB Adapter
 05e3  CyberDoor
@@ -88,6 +90,9 @@
 	0405  Rhino R8FXX
 	0505  Rhino R24FXX
 	0506  Rhino R2T1
+	0605  Rhino R2T1
+# =
+	0705  Rhino R24FXS
 0b49  ASCII Corporation
 	064f  Trance Vibrator
 0e11  Compaq Computer Corporation
@@ -201,6 +206,7 @@
 	f150  NetFlex-3/P ThunderLAN 2.3
 0e21  Cowon Systems, Inc.
 0e55  HaSoTec GmbH
+0eac  SHF Communication Technologies AG
 # Formerly NCR
 1000  LSI Logic / Symbios Logic
 	0001  53c810
@@ -213,6 +219,7 @@
 	0006  53c860
 		1000 1000  LSI53C860E PCI to Ultra SCSI I/O Processor
 	000a  53c1510
+		0e11 b143  Integrated Dual Channel Wide Ultra2 SCSI Controller
 		1000 1000  LSI53C1510 PCI to Dual Channel Wide Ultra2 SCSI Controller (Nonintelligent mode)
 	000b  53C896/897
 		0e11 6004  EOB003 Series SCSI host adapter
@@ -246,10 +253,13 @@
 		1000 1000  LSI53C875A PCI to Ultra SCSI Controller
 	0020  53c1010 Ultra3 SCSI Adapter
 		1000 1000  LSI53C1010-33 PCI to Dual Channel Ultra160 SCSI Controller
+		107b 1040  Server Onboard 53C1010-33
 		1de1 1020  DC-390U3W
 	0021  53c1010 66MHz  Ultra3 SCSI Adapter
 		1000 1000  LSI53C1000/1000R/1010R/1010-66 PCI to Ultra160 SCSI Controller
 		1000 1010  Asus TR-DLS onboard 53C1010-66
+		103c 1330  Ultra160 SCSI [A7059A]
+		103c 1340  Ultra160 SCSI [A7060A]
 		124b 1070  PMC-USCSI3
 		4c53 1080  CT8 mainboard
 		4c53 1300  P017 mezzanine (32-bit PMC)
@@ -261,6 +271,7 @@
 		1028 016c  PowerEdge 1850 MPT Fusion SCSI/RAID (Perc 4)
 		1028 0183  PowerEdge 1800
 		1028 1010  LSI U320 SCSI Controller
+		103c 12c5  Ultra320 SCSI [A7173A]
 		124b 1170  PMC-USCSI320
 		1734 1052  Primergy RX300 S2
 	0031  53c1030ZC PCI-X Fusion-MPT Dual Ultra320 SCSI
@@ -537,8 +548,9 @@
 		1028 00ce  PowerEdge 1400
 		1028 00d1  PowerEdge 2550
 		1028 00d9  PowerEdge 2500
-		1028 0134  Poweredge SC600
+		1028 0134  PowerEdge 600SC
 		103c 10e1  NetServer Rage XL
+		107b 6400  6400 Server
 		1734 007a  Primergy RX300
 		8086 3411  SDS2 Mainboard
 		8086 3427  S875WP1-E mainboard
@@ -583,7 +595,12 @@
 	4a4d  R420 JM [FireGL X3]
 	4a4e  M18 JN [Radeon Mobility 9800]
 	4a50  R420 JP [Radeon X800XT]
+	4a54  R420 [Radeon X800 VE]
+	4a69  R420 [Radeon X800 PRO/GTO] (Secondary)
+	4a6a  R420 [Radeon X800] (Secondary)
+	4a6b  R420 [Radeon X800] (Secondary)
 	4a70  R420 [X800XT-PE] (Secondary)
+	4a74  R420 [Radeon X800 VE] (Secondary)
 	4b49  R480 [Radeon X850XT]
 	4b4b  R480 [Radeon X850Pro]
 	4b4c  R481 [Radeon X850XT-PE]
@@ -643,7 +660,7 @@
 	4c5a  Radeon Mobility M6 LZ
 	4c64  Radeon R250 Ld [Radeon Mobility 9000 M9]
 	4c65  Radeon R250 Le [Radeon Mobility 9000 M9]
-	4c66  Radeon R250 [Radeon Mobility 9200]
+	4c66  Radeon R250 [Mobility FireGL 9000]
 	4c67  Radeon R250 Lg [Radeon Mobility 9000 M9]
 # Secondary chip to the Lf
 	4c6e  Radeon R250 Ln [Radeon Mobility 9000 M9] [Secondary]
@@ -787,6 +804,7 @@
 		174b 7112  RV100 QY [Sapphire Radeon VE 7000]
 		174b 7c28  Sapphire Radeon VE 7000 DDR
 		1787 0202  RV100 QY [Excalibur Radeon 7000]
+		17ee 1001  Radeon 7000 64MB DDR + DVI
 	515a  Radeon RV100 QZ [Radeon 7000/VE]
 	515e  ES1000
 	515f  ES1000
@@ -855,9 +873,11 @@
 	5551  R423 UQ [FireGL V7200 (PCIE)]
 	5552  R423 UR [FireGL V5100 (PCIE)]
 	5554  R423 UT [FireGL V7100 (PCIE)]
+	5569  R423 UI [Radeon X800PRO (PCIE)] Secondary
 	556b  Radeon R423 UK (PCIE) [X800 SE] (Secondary)
 	556d  R430 [Radeon X800 XL] (PCIe) Secondary
 	556f  R430 [Radeon X800 (PCIE) Secondary]
+	5571  R423GL-SE ATI FIREGL V5100 PCI-EX Secondary
 	564a  M26 [Mobility FireGL V5000]
 	564b  M26 [Mobility FireGL V5000]
 	564f  M26 [Radeon Mobility X700 XL] (PCIE)
@@ -917,6 +937,7 @@
 	5969  ES1000
 	5974  RS482 [Radeon Xpress 200]
 	5975  RS482 [Radeon Xpress 200M]
+	5a33  Radeon Xpress 200 Host Bridge
 	5a34  RS480 PCI-X Root Port
 # Comes in pair with 5a3f
 	5a36  RS480 PCI Bridge
@@ -933,13 +954,15 @@
 		1043 032e  Extreme AX300/TD
 		1462 0400  RX300SE-TD128E (MS-8940 REV:200)
 		1462 0402  RX300SE-TD128E (MS-8940)
-	5b62  RV370 5B62 [Radeon X600 (PCIE)]
+		196d 1086  X300SE HM
+	5b62  RV380 [Radeon X600 (PCIE)]
 	5b63  RV370 [Sapphire X550 Silent]
 	5b64  RV370 5B64 [FireGL V3100 (PCIE)]
 	5b65  RV370 5B65 [FireGL D1100 (PCIE)]
 	5b70  RV370 [Radeon X300SE]
 		1462 0403  RX300SE-TD128E (MS-8940) (secondary display)
-	5b72  Radeon X600(RV380)
+		196d 1087  X300SE HM
+	5b72  RV380 [Radeon X600]
 	5b73  RV370 secondary [Sapphire X550 Silent]
 	5b74  RV370 5B64 [FireGL V3100 (PCIE)] (Secondary)
 	5c61  M9+ 5C61 [Radeon Mobility 9200 (AGP)]
@@ -981,11 +1004,13 @@
 	700f  PCI Bridge [IGP 320M]
 	7010  PCI Bridge [IGP 340M]
 	7100  R520 [Radeon X1800]
+	7104  R520 GL ATI FireGL V7200 Primary
 	7105  R520 [FireGL]
 	7109  R520 [Radeon X1800]
 		1002 0322  All-in-Wonder X1800XL
 		1002 0d02  Radeon X1800 CrossFire Edition
 	7120  R520 [Radeon X1800] (Secondary)
+	7124  R520 GL ATI FireGL V7200 Secondary
 	7129  R520 [Radeon X1800] (Secondary)
 		1002 0323  All-in-Wonder X1800XL (Secondary)
 		1002 0d03  Radeon X1800 CrossFire Edition (Secondary)
@@ -998,16 +1023,26 @@
 	714a  M52 [ATI Mobility Radeon X1300]
 	714b  M52 [ATI Mobility Radeon X1300]
 	714c  M52 [ATI Mobility Radeon X1300]
+	7152  RV515 GL ATI FireGL V3300 Primary
 	7162  RV515 [Radeon X1300] (Secondary)
 		1002 0323  All-in-Wonder 2006 PCI-E Edition (Secondary)
 	7166  RV515 [Radeon X1300] (Secondary)
 		1002 0323  All-in-Wonder 2006 PCI-E Edition (Secondary)
+	7172  RV515 GL ATI FireGL V3300 Secondary
+	7181  RV516 XT Radeon X1600 Series Primary
+	71a1  RV516 XT Radeon X1600 Series Secondary
 	71c0  RV530 [Radeon X1600]
 	71c2  RV530 [Radeon X1600]
 	71c4  M56GL [ATI Mobility FireGL V5200]
 	71c5  M56P [Radeon Mobility X1600]
+	71d5  M66-P ATI Mobility Radeon X1700
+	71d6  M66-XT ATI Mobility Radeon X1700
 	71e0  RV530 [Radeon X1600] (Secondary)
 	71e2  RV530 [Radeon X1600] (Secondary)
+	7249  R580 [Radeon X1900 XT] Primary
+	724e  R580 [FireGL V7300/V7350] Primary (PCIE)
+	7269  R580 [Radeon X1900 XT] Secondary
+	726e  R580 [FireGL V7300/V7350] Secondary (PCIE)
 	7833  Radeon 9100 IGP Host Bridge
 	7834  Radeon 9100 PRO IGP
 	7835  Radeon Mobility 9200 IGP
@@ -1174,6 +1209,9 @@
 		1385 2100  FA510
 		1395 0001  10/100 Ethernet CardBus PC Card
 		13d1 ab01  EtherFast 10/100 Cardbus (PCMPC200)
+		1498 000a  TPMC880-10 10/100Base-T and 10Base2 PMC Ethernet Adapter
+		1498 000b  TPMC880-11 Single 10/100Base-T PMC Ethernet Adapter
+		1498 000c  TPMC880-12 Single 10Base2 PMC Ethernet Adapter
 		14cb 0100  LNDL-100N 100Base-TX Ethernet PC Card
 		8086 0001  EtherExpress PRO/100 Mobile CardBus 32
 	001a  Farallon PN9000SX Gigabit Ethernet
@@ -1240,6 +1278,7 @@
 		153b 1136  SiXPack 5.1+
 		1681 0050  Game Theater XP
 		1681 a011  Fortissimo III 7.1
+		5053 3357  Santa Cruz
 	6004  CS 4614/22/24 [CrystalClear SoundFusion Audio Accelerator]
 	6005  Crystal CS4281 PCI Audio
 		1013 4281  Crystal CS4281 PCI Audio
@@ -1367,6 +1406,7 @@
 		1014 02c1  PCI-X DDR 3Gb SAS Adapter (572A/572C)
 		1014 02c2  PCI-X DDR 3Gb SAS RAID Adapter (572B/571D)
 	0302  Winnipeg PCI-X Host Bridge
+	0308  CalIOC2 PCI-E Root Port
 	0314  ZISC 036 Neural accelerator card
 	3022  QLA3022 Network Adapter
 	4022  QLA3022 Network Adapter
@@ -1441,6 +1481,7 @@
 		1259 2701  AT-2700FX 100Mb Ethernet
 		1259 2702  AT-2700FTX 10/100 Mb Fiber/Copper Fast Ethernet
 		1259 2703  AT-2701FX
+		1259 2704  AT-2701FTX 10/100 Mb Fiber/Copper Fast Ethernet
 		4c53 1000  CC7/CR7/CP7/VC7/VP7/VR7 mainboard
 		4c53 1010  CP5/CR6 mainboard
 		4c53 1020  VR6 mainboard
@@ -1820,6 +1861,9 @@
 		102b 2538  Parhelia APVe
 		102b 3007  QID Low-profile PCIe
 	4536  VIA Framegrabber
+	4cdc  Morphis Vision System Jpeg2000
+	4fc5  Morphis Vision System
+	5e10  Morphis Vision System Aux/IO
 	6573  Shark 10/100 Multiport SwitchNIC
 102c  Chips and Technologies
 	00b8  F64310
@@ -2121,11 +2165,15 @@
 	121c  NetServer PCI COM Port Decoder
 	1229  zx1 System Bus Adapter
 	122a  zx1 I/O Controller
-	122e  zx1 Local Bus Adapter
+	122e  PCI-X Local Bus Adapter
+	127b  sx1000 System Bus Adapter
 	127c  sx1000 I/O Controller
 	1290  Auxiliary Diva Serial Port
 	1291  Auxiliary Diva Serial Port
 	12b4  zx1 QuickSilver AGP8x Local Bus Adapter
+	12eb  sx2000 System Bus Adapter
+	12ec  sx2000 I/O Controller
+	12ee  PCI-X 2.0 Local Bus Adapter
 	12f8  Broadcom BCM4306 802.11b/g Wireless LAN
 	12fa  BCM4306 802.11b/g Wireless LAN Controller
 	2910  E2910A PCIBus Exerciser
@@ -2134,6 +2182,9 @@
 	3085  Realtek RTL8139/8139C/8139C+
 	3220  Hewlett-Packard Smart Array P600
 	3230  Hewlett-Packard Smart Array Controller
+	4030  zx2 System Bus Adapter
+	4031  zx2 I/O Controller
+	4037  PCIe Local Bus Adapter
 103e  Solliday Engineering
 103f  Synopsys/Logic Modeling Group
 1040  Accelgraphics Inc.
@@ -2347,13 +2398,24 @@
 		103c 099c  NX6110/NC6120
 	8036  PCI6515 Cardbus Controller
 	8038  PCI6515 SmartCard Controller
+	8039  PCIxx12 Cardbus Controller
+		103c 309f  nx9420
+	803a  PCIxx12 OHCI Compliant IEEE 1394 Host Controller
+		103c 309f  nx9420
 	803b  5-in-1 Multimedia Card Reader (SD/MMC/MS/MS PRO/xD)
+		103c 309f  nx9420
+# Description as Windows says it
+	803c  PCIxx12 SDA Standard Compliant SD Host Controller
+		103c 309f  nx9420
+# Description as Windows says it
+	803d  PCIxx12 GemCore based SmartCard controller
+		103c 309f  nx9420
 	8201  PCI1620 Firmware Loading Function
 	8204  PCI7410,7510,7610 PCI Firmware Loading Function
 		1028 0139  Latitude D400
 		1028 014e  Latitude D800
-	8231  XIO2000(A)/XIO2200 PCI Express-to-PCI Bridge
-	8235  XIO2200 IEEE-1394a-2000 Controller (PHY/Link)
+	8231  XIO2000(A)/XIO2200(A) PCI Express-to-PCI Bridge
+	8235  XIO2200(A) IEEE-1394a-2000 Controller (PHY/Link)
 	8400  ACX 100 22Mbps Wireless Interface
 		1186 3b00  DWL-650+ PC Card cardbus 22Mbs Wireless Adapter [AirPlus]
 		1186 3b01  DWL-520+ 22Mbps PCI Wireless Adapter
@@ -2431,6 +2493,7 @@
 	ac52  PCI1451 PC card Cardbus Controller
 	ac53  PCI1421 PC card Cardbus Controller
 	ac54  PCI1620 PC Card Controller
+		103c 08b0  tc1100 tablet
 	ac55  PCI1520 PC card Cardbus Controller
 		1014 0512  ThinkPad T30/T40
 	ac56  PCI1510 PC card Cardbus Controller
@@ -2527,6 +2590,7 @@
 		ecc0 0072  Mona rev.2
 	18c0  MPC8265A/8266/8272
 	18c1  MPC8271/MPC8272
+	3055  SM56 Data Fax Modem
 	3410  DSP56361 Digital Signal Processor
 		ecc0 0050  Gina24 rev.0
 		ecc0 0051  Gina24 rev.1
@@ -3112,6 +3176,7 @@
 		1461 0003  AverMedia UltraTV PCI 350
 		1461 0004  AVerTV WDM Video Capture
 		1461 0761  AverTV DVB-T
+		1461 0771  AverMedia AVerTV DVB-T 771
 		14f1 0001  Bt878 Mediastream Controller NTSC
 		14f1 0002  Bt878 Mediastream Controller PAL BG
 		14f1 0003  Bt878a Mediastream Controller PAL BG
@@ -3174,6 +3239,7 @@
 		1461 0002  Avermedia PCTV98 Audio Capture
 		1461 0004  AVerTV WDM Audio Capture
 		1461 0761  AVerTV DVB-T
+		1461 0771  AverMedia AVerTV DVB-T 771
 		14f1 0001  Bt878 Video Capture (Audio Section)
 		14f1 0002  Bt878 Video Capture (Audio Section)
 		14f1 0003  Bt878 Video Capture (Audio Section)
@@ -3259,8 +3325,6 @@
 	4002  TIO-CE PCI Express Port
 	8001  O2 1394
 	8002  G-net NT
-	8010  Broadcom e-net [SGI IO9/IO10 BaseIO]
-	8018  Broadcom e-net [SGI A330 Server BaseIO]
 10aa  ACC Microelectronics
 	0000  ACCM 2188
 10ab  Digicom
@@ -3385,6 +3449,11 @@
 		129d 0002  Aculab PCI Prosidy card
 		12d9 0002  PCI Prosody Card
 		12df 4422  4422PCI ["Do-All" Telemetry Data Aquisition System]
+		1517 000b  ECSG-1R3ADC-PMC Clock synthesizer
+	9656  PCI <-> IOBus Bridge
+		1517 000f  ECDR-GC314-PMC Receiver
+		1885 0700  Tsunami FPGA PMC with Altera Stratix S40
+		1885 0701  Tsunami FPGA PMC with Altera Stratix S30
 	bb04  B&B 3PCIOSD1A Isolated PCI Serial
 10b6  Madge Networks
 	0001  Smart 16/4 PCI Ringnode
@@ -3476,6 +3545,8 @@
 	900a  3c900B-FL 10base-FL [Cyclone]
 	9050  3c905 100BaseTX [Boomerang]
 	9051  3c905 100BaseT4 [Boomerang]
+	9054  3C905B-TX Fast Etherlink XL PCI
+		10b7 9054  3C905B-TX Fast Etherlink XL PCI
 	9055  3c905B 100BaseTX [Cyclone]
 		1028 0080  3C905B Fast Etherlink XL 10/100
 		1028 0081  3C905B Fast Etherlink XL 10/100
@@ -3638,7 +3709,7 @@
 	5251  M5251 P1394 OHCI 1.0 Controller
 	5253  M5253 P1394 OHCI 1.1 Controller
 	5261  M5261 Ethernet Controller
-	5263  M5263 Ethernet Controller
+	5263  ULi 1689,1573 integrated ethernet.
 	5281  ALi M5281 Serial ATA / RAID Host Controller
 	5287  ULi 5287 SATA
 	5288  ULi M5288 SATA
@@ -3741,6 +3812,7 @@
 	1200  ASC1200 [(abp940) Fast SCSI-II]
 	1300  ABP940-U / ABP960-U
 		10cd 1310  ASC1300 SCSI Adapter
+		1195 1320  Ultra-SCSI CardBus PC Card REX CB31
 	2300  ABP940-UW
 	2500  ABP940-U2W
 10ce  Radius
@@ -3890,43 +3962,53 @@
 		1043 815a  K8N4-E Mainboard
 		1458 0c11  GA-K8N Ultra-9 Mainboard
 		1462 7100  MSI K8N Diamond
+		147b 1c1a  KN8-Ultra Mainboard
 	0051  CK804 ISA Bridge
 	0052  CK804 SMBus
 		1043 815a  K8N4-E Mainboard
 		1458 0c11  GA-K8N Ultra-9 Mainboard
 		1462 7100  MSI K8N Diamond
+		147b 1c1a  KN8-Ultra Mainboard
 	0053  CK804 IDE
 		1043 815a  K8N4-E Mainboard
 		1458 5002  GA-K8N Ultra-9 Mainboard
 		1462 7100  MSI K8N Diamond
+		147b 1c1a  KN8-Ultra Mainboard
 	0054  CK804 Serial ATA Controller
 		1458 b003  GA-K8N Ultra-9 Mainboard
 		1462 7100  MSI K8N Diamond
+		147b 1c1a  KN8-Ultra Mainboard
 	0055  CK804 Serial ATA Controller
 		1043 815a  K8N4-E Mainboard
 		1458 b003  GA-K8N Ultra-9 Mainboard
+		147b 1c1a  KN8-Ultra Mainboard
 	0056  CK804 Ethernet Controller
 	0057  CK804 Ethernet Controller
 		1043 8141  K8N4-E Mainboard
 		1458 e000  GA-K8N Ultra-9 Mainboard
 		1462 7100  MSI K8N Diamond
+		147b 1c1a  KN8-Ultra Mainboard
 	0058  CK804 AC'97 Modem
 	0059  CK804 AC'97 Audio Controller
 		1043 812a  K8N4-E Mainboard
+		147b 1c1a  KN8-Ultra Mainboard
 	005a  CK804 USB Controller
 		1043 815a  K8N4-E Mainboard
 		1458 5004  GA-K8N Ultra-9 Mainboard
 		1462 7100  MSI K8N Diamond
+		147b 1c1a  KN8-Ultra Mainboard
 	005b  CK804 USB Controller
 		1043 815a  K8N4-E Mainboard
 		1458 5004  GA-K8N Ultra-9 Mainboard
 		1462 7100  MSI K8N Diamond
+		147b 1c1a  KN8-Ultra Mainboard
 	005c  CK804 PCI Bridge
 	005d  CK804 PCIE Bridge
 	005e  CK804 Memory Controller
 		10f1 2891  Thunder K8SRE Mainboard
 		1458 5000  GA-K8N Ultra-9 Mainboard
 		1462 7100  MSI K8N Diamond
+		147b 1c1a  KN8-Ultra Mainboard
 	005f  CK804 Memory Controller
 	0060  nForce2 ISA Bridge
 		1043 80ad  A7N8X Mainboard
@@ -3943,6 +4025,7 @@
 		1043 0c11  A7N8X Mainboard
 		a0a0 03b4  UK79G-1394 motherboard
 	006a  nForce2 AC97 Audio Controler (MCP)
+		1043 8095  nForce2 AC97 Audio Controler (MCP)
 		a0a0 0304  UK79G-1394 motherboard
 	006b  nForce Audio Processing Unit
 		10de 006b  nForce2 MCP Audio Processing Unit
@@ -3983,6 +4066,7 @@
 	00c9  NV41.9 [GeForce Go 6800 Ultra]
 	00cc  NV41 [Quadro FX Go1400]
 	00cd  NV41 [Quadro FX 3450/4000 SDI]
+		10de 029b  wx4300 Workstation
 	00ce  NV41GL [Quadro FX 1400]
 	00d0  nForce3 LPC Bridge
 	00d1  nForce3 Host Bridge
@@ -3997,24 +4081,32 @@
 	00da  nForce3 Audio
 	00dd  nForce3 PCI Bridge
 	00df  CK8S Ethernet Controller
+		105b 0c43  Winfast NF3250K8AA
 		147b 1c0b  NF8 Mainboard
 	00e0  nForce3 250Gb LPC Bridge
+		10de 0c11  Winfast NF3250K8AA
 		147b 1c0b  NF8 Mainboard
 	00e1  nForce3 250Gb Host Bridge
 		147b 1c0b  NF8 Mainboard
 	00e2  nForce3 250Gb AGP Host to PCI Bridge
 	00e3  CK8S Serial ATA Controller (v2.5)
+		105b 0c43  Winfast NF3250K8AA
 		147b 1c0b  NF8 Mainboard
 	00e4  nForce 250Gb PCI System Management
+		105b 0c43  Winfast NF3250K8AA
 		147b 1c0b  NF8 Mainboard
 	00e5  CK8S Parallel ATA Controller (v2.5)
+		105b 0c43  Winfast NF3250K8AA
 		147b 1c0b  NF8 Mainboard
 	00e6  CK8S Ethernet Controller
 	00e7  CK8S USB Controller
+		105b 0c43  Winfast NF3250K8AA
 		147b 1c0b  NF8 Mainboard
 	00e8  nForce3 EHCI USB 2.0 Controller
+		105b 0c43  Winfast NF3250K8AA
 		147b 1c0b  NF8 Mainboard
 	00ea  nForce3 250Gb AC'97 Audio Controller
+		105b 0c43  Winfast NF3250K8AA
 		147b 1c0b  NF8 Mainboard
 	00ed  nForce3 250Gb PCI-to-PCI Bridge
 	00ee  CK8S Serial ATA Controller (v2.5)
@@ -4097,6 +4189,7 @@
 		1048 0c52  Gladiac-64
 		107d 2840  WinFast GeForce2 GTS with TV output
 		107d 2842  WinFast GeForce 2 Pro
+		10de 002e  GeForce2 GTS
 		1462 8831  Creative GeForce2 Pro
 	0151  NV15DDR [GeForce2 Ti]
 		1043 405f  V7700Ti
@@ -4104,8 +4197,9 @@
 	0152  NV15BR [GeForce2 Ultra, Bladerunner]
 		1048 0c56  GLADIAC Ultra
 	0153  NV15GL [Quadro2 Pro]
-	0161  GeForce 6200 TurboCache(TM)
-	0162  NV43 [GeForce 6200 SE]
+	0161  NV44 [GeForce 6200 TurboCache(TM)]
+	0162  NV44 [GeForce 6200 SE TurboCache (TM)]
+	0163  NV44 [GeForce 6200 LE]
 	0164  NV44 [GeForce Go 6200]
 	0165  NV44 [Quadro NVS 285]
 	0166  NV43 [GeForce Go 6400]
@@ -4124,6 +4218,7 @@
 	0174  NV17 [GeForce4 440 Go]
 	0175  NV17 [GeForce4 420 Go]
 	0176  NV17 [GeForce4 420 Go 32M]
+		103c 08b0  tc1100 tablet
 		4c53 1090  Cx9 / Vx9 mainboard
 	0177  NV17 [GeForce4 460 Go]
 	0178  NV17GL [Quadro4 550 XGL]
@@ -4165,6 +4260,8 @@
 	01c2  nForce USB Controller
 	01c3  nForce Ethernet Controller
 	01d1  GeForce 7300 LE
+# MSI 7300LE PCI Express Graphics Adapter
+		1462 0345  John Ennew
 	01d7  Quadro NVS 110M / GeForce Go 7300
 	01d8  GeForce Go 7400
 	01da  Quadro NVS 110M
@@ -4275,6 +4372,7 @@
 	029c  Quadro FX 5500
 	029d  Quadro FX 3500
 	029e  Quadro FX 1500
+	029f  Quadro FX 4500 X2
 # Xbox Graphics Processing Unit (Integrated). GeForce3 derivative (NV20 < NV2A < NV25).
 	02a0  NV2A [XGPU]
 	02e1  GeForce 7600 GS
@@ -4369,19 +4467,55 @@
 	0368  MCP55 SMBus
 	0369  MCP55 Memory Controller
 	036a  MCP55 Memory Controller
+	036b  MCP55 SMU
 	036c  MCP55 USB Controller
 	036d  MCP55 USB Controller
 	036e  MCP55 IDE
+	0370  MCP55 PCI bridge
 	0371  MCP55 High Definition Audio
 	0372  MCP55 Ethernet
 	0373  MCP55 Ethernet
+	0374  MCP55 PCI Express bridge
+	0375  MCP55 PCI Express bridge
+	0376  MCP55 PCI Express bridge
+	0377  MCP55 PCI Express bridge
+	0378  MCP55 PCI Express bridge
 	037a  MCP55 Memory Controller
 	037e  MCP55 SATA Controller
 	037f  MCP55 SATA Controller
 	0391  G70 [GeForce 7600 GT]
 	0392  G70 [GeForce 7600 GS]
+		1462 0622  NX7600GS-T2D256EH
 	0398  G70 [GeForce Go 7600]
 	039e  Quadro FX 560
+	03a0  C55 Host Bridge
+	03a1  C55 Host Bridge
+	03a2  C55 Host Bridge
+	03a3  C55 Host Bridge
+	03a4  C55 Host Bridge
+	03a5  C55 Host Bridge
+	03a6  C55 Host Bridge
+	03a7  C55 Host Bridge
+	03a8  C55 Memory Controller
+	03a9  C55 Memory Controller
+	03aa  C55 Memory Controller
+	03ab  C55 Memory Controller
+	03ac  C55 Memory Controller
+	03ad  C55 Memory Controller
+	03ae  C55 Memory Controller
+	03af  C55 Memory Controller
+	03b0  C55 Memory Controller
+	03b1  C55 Memory Controller
+	03b2  C55 Memory Controller
+	03b3  C55 Memory Controller
+	03b4  C55 Memory Controller
+	03b5  C55 Memory Controller
+	03b6  C55 Memory Controller
+	03b7  C55 PCI Express bridge
+	03b8  C55 PCI Express bridge
+	03b9  C55 PCI Express bridge
+	03ba  C55 Memory Controller
+	03bb  C55 PCI Express bridge
 	03e0  MCP61 LPC Bridge
 	03e1  MCP61 LPC Bridge
 	03e2  MCP61 LPC Bridge
@@ -4390,6 +4524,8 @@
 	03e5  MCP61 Ethernet
 	03e6  MCP61 Ethernet
 	03e7  MCP61 SATA Controller
+	03e8  MCP61 PCI Express bridge
+	03e9  MCP61 PCI Express bridge
 	03ea  MCP61 Memory Controller
 	03eb  MCP61 SMBus
 	03ec  MCP61 IDE
@@ -4398,9 +4534,42 @@
 	03f0  MCP61 High Definition Audio
 	03f1  MCP61 USB Controller
 	03f2  MCP61 USB Controller
+	03f3  MCP61 PCI bridge
+	03f4  MCP61 SMU
 	03f5  MCP61 Memory Controller
 	03f6  MCP61 SATA Controller
 	03f7  MCP61 SATA Controller
+	0440  MCP65 LPC Bridge
+	0441  MCP65 LPC Bridge
+	0442  MCP65 LPC Bridge
+	0443  MCP65 LPC Bridge
+	0444  MCP65 Memory Controller
+	0445  MCP65 Memory Controller
+	0446  MCP65 SMBus
+	0447  MCP65 SMU
+	0448  MCP65 IDE
+	0449  MCP65 PCI bridge
+	044a  MCP65 High Definition Audio
+	044b  MCP65 High Definition Audio
+	044c  MCP65 AHCI Controller
+	044d  MCP65 AHCI Controller
+	044e  MCP65 AHCI Controller
+	044f  MCP65 AHCI Controller
+	0450  MCP65 Ethernet
+	0451  MCP65 Ethernet
+	0452  MCP65 Ethernet
+	0453  MCP65 Ethernet
+	0454  MCP65 USB Controller
+	0455  MCP65 USB Controller
+	0456  MCP65 USB Controller
+	0457  MCP65 USB Controller
+	0458  MCP65 PCI Express bridge
+	0459  MCP65 PCI Express bridge
+	045a  MCP65 PCI Express bridge
+	045c  MCP65 SATA Controller
+	045d  MCP65 SATA Controller
+	045e  MCP65 SATA Controller
+	045f  MCP65 SATA Controller
 10df  Emulex Corporation
 	1ae5  LP6000 Fibre Channel Host Adapter
 	f085  LP850 Fibre Channel Host Adapter
@@ -4446,11 +4615,15 @@
 10e2  Aptix Corporation
 10e3  Tundra Semiconductor Corp.
 	0000  CA91C042 [Universe]
+	0108  Tsi108 Host Bridge for Single PowerPC
 	0148  Tsi148 [Tempe]
 	0860  CA91C860 [QSpan]
+		25d8 0110  HuSCI PMC reflective memory
 	0862  CA91C862A [QSpan-II]
+		25d8 0110  HuSCI PMC reflective memory
 	8260  CA91L8200B [Dual PCI PowerSpan II]
 	8261  CA91L8260B [Single PCI PowerSpan II]
+	a108  Tsi109 Host Bridge for Dual PowerPC
 10e4  Tandem Computers
 	8029  Realtek 8029 Network Card
 10e5  Micro Industries Corporation
@@ -4501,6 +4674,7 @@
 	8111  Twist3 Frame Grabber
 10ec  Realtek Semiconductor Co., Ltd.
 	0139  Zonet Zen3200
+		1025 1605  TravelMate 5600 series
 	8029  RTL-8029(AS)
 		10b8 2011  EZ-Card (SMC1208)
 		10ec 8029  RTL-8029(AS)
@@ -4509,6 +4683,7 @@
 		1259 2400  AT-2400
 	8129  RTL-8129
 		10ec 8129  RT8129 Fast Ethernet Adapter
+	8136  RTL8101E PCI Express Fast Ethernet controller
 	8138  RT8139 (B/C) Cardbus Fast Ethernet Adapter
 		10ec 8138  RT8139 (B/C) Fast Ethernet Adapter
 	8139  RTL-8139/8139C/8139C+
@@ -4546,6 +4721,8 @@
 		8e2e 7000  KF-230TX
 		8e2e 7100  KF-230TX/2
 		a0a0 0007  ALN-325C
+	8167  RTL-8169SC Gigabit Ethernet
+	8168  RTL8111/8168B PCI Express Gigabit Ethernet controller
 	8169  RTL-8169 Gigabit Ethernet
 		1025 0079  Aspire 5024WLMi
 		1259 c107  CG-LAPCIGT
@@ -4600,6 +4777,7 @@
 1100  Jazz Multimedia
 1101  Initio Corporation
 	1060  INI-A100U2W
+	1622  INI-1623 PCI SATA-II Controller
 	9100  INI-9100/9100W
 	9400  INI-940
 	9401  INI-950
@@ -4632,6 +4810,9 @@
 		1102 0058  SB0090 Audigy Player/OEM
 		1102 1007  SB0240 Audigy 2 Platinum 6.1
 		1102 2002  SB Audigy 2 ZS (SB0350)
+	0005  SB X-Fi
+		1102 0021  X-Fi Platinum
+		1102 1003  X-Fi XtremeMusic
 	0006  [SB Live! Value] EMU10k1X
 	0007  SB Audigy LS
 		1102 0007  SBLive! 24bit
@@ -5749,10 +5930,15 @@
 	0017  GCNB-LE Host Bridge
 	0036  HT1000 PCI/PCI-X bridge
 	0101  CIOB-X2 PCI-X I/O Bridge
+	0103  EPB PCI-Express to PCI-X Bridge
 	0104  HT1000 PCI/PCI-X bridge
 	0110  CIOB-E I/O Bridge with Gigabit Ethernet
-	0130  HT1000 PCI-X bridge
-	0132  HT1000 PCI-Express bridge
+	0130  HT2000 PCI-X bridge
+	0132  HT2000 PCI-Express bridge
+		1166 0132  HT2000 PCI-Express bridge
+	0140  HT2100 PCI-Express Bridge
+	0141  HT2100 PCI-Express Bridge
+	0142  HT2100 PCI-Express Bridge
 	0200  OSB4 South Bridge
 	0201  CSB5 South Bridge
 		4c53 1080  CT8 mainboard
@@ -5763,6 +5949,7 @@
 	0212  CSB5 IDE Controller
 		4c53 1080  CT8 mainboard
 	0213  CSB6 RAID/IDE Controller
+		1028 4134  PowerEdge 600SC
 		1028 c134  Poweredge SC600
 		1734 1012  Primergy RX300
 	0214  HT1000 Legacy IDE controller
@@ -5901,6 +6088,7 @@
 		1186 1301  DFE-530TX+ 10/100 Ethernet Adapter
 		1186 1303  DFE-528TX 10/100 Fast Ethernet PCI Adapter
 	1340  DFE-690TXD CardBus PC Card
+	1405  DFE-520TX Fast Ethernet PCI Adapter
 	1541  DFE-680TXD CardBus PC Card
 	1561  DRP-32TXD Cardbus PC Card
 	2027  AirPlus Xtreme G DWL-G520 Adapter
@@ -6488,6 +6676,7 @@
 1215  Interware Co., Ltd
 1216  Purup Prepress A/S
 1217  O2 Micro, Inc.
+	00f7  Firewire (IEEE 1394)
 	6729  OZ6729
 	673a  OZ6730
 	6832  OZ6832/6833 CardBus Controller
@@ -6506,7 +6695,10 @@
 	7112  OZ711EC1/M1 SmartCardBus/MemoryCardBus Controller
 	7113  OZ711EC1 SmartCardBus Controller
 	7114  OZ711M1/MC1 4-in-1 MemoryCardBus Controller
+	7120  Integrated MMC/SD Controller
+	7130  Integrated MS/xD Controller
 	7134  OZ711MP1/MS1 MemoryCardBus Controller
+	7135  Cardbus bridge
 	71e2  OZ711E2 SmartCardBus Controller
 	7212  OZ711M2 4-in-1 MemoryCardBus Controller
 	7213  OZ6933E CardBus Controller
@@ -7116,6 +7308,7 @@
 		12b9 0091  USR 56k Internal Voice WinModem (Model 2978)
 	1007  USR 56k Internal WinModem
 		12b9 00a3  USR 56k Internal WinModem (Model 3595)
+		12b9 00c4  U.S. Robotics 56K Voice Win Int (2884a)
 	1008  56K FaxModem Model 5610
 		12b9 00a2  USR 56k Internal FAX Modem (Model 2977)
 		12b9 00aa  USR 56k Internal Voice Modem (Model 2976)
@@ -7379,17 +7572,17 @@
 1312  Acuity Imaging, Inc
 1313  Yaskawa Electric Co.
 1316  Teradyne Inc
-1317  Linksys
+1317  ADMtek
 	0981  21x4x DEC-Tulip compatible 10/100 Ethernet
 	0985  NC100 Network Everywhere Fast Ethernet 10/100
 		1734 100c  Scenic N300 ADMtek AN983 10/100 Mbps PCI Adapter
 	1985  21x4x DEC-Tulip compatible 10/100 Ethernet
 	2850  HSP MicroModem 56
-	5120  ADMtek ADM5120 OpenGate System-on-Chip
-	8201  ADMtek ADM8211 802.11b Wireless Interface
+	5120  ADM5120 OpenGate System-on-Chip
+	8201  ADM8211 802.11b Wireless Interface
 		10b8 2635  SMC2635W 802.11b (11Mbps) wireless lan pcmcia (cardbus) card
 		1317 8201  SMC2635W 802.11b (11mbps) wireless lan pcmcia (cardbus) card
-	8211  ADMtek ADM8211 802.11b Wireless Interface
+	8211  ADM8211 802.11b Wireless Interface
 	9511  21x4x DEC-Tulip compatible 10/100 Ethernet
 1318  Packet Engines Inc.
 	0911  GNIC-II PCI Gigabit Ethernet [Hamachi]
@@ -7557,6 +7750,7 @@
 	0204  GPS170PCI GPS Receiver
 	0301  TCR510PCI IRIG Timecode Reader
 	0302  TCR167PCI IRIG Timecode Reader
+	0303  TCR511PCI IRIG Timecode Reader
 1361  Soliton Systems K.K.
 1362  Fujifacom Corporation
 1363  Phoenix Technology Ltd
@@ -7567,6 +7761,7 @@
 1368  Skyware Corporation
 1369  Digigram
 136a  High Soft Tech
+	0004  HST Saphir VII mini PCI
 136b  Kawasaki Steel Corporation
 	ff01  KL5A72002 Motion JPEG
 136c  Adtek System Science Co Ltd
@@ -7651,6 +7846,8 @@
 	630a  GA630 Gigabit Ethernet
 	6b00  WG311v3 54 Mbps Wireless PCI Adapter
 	6d00  WPNT511 RangeMax 240 Mbps Wireless PC Card
+	7b00  WN511B RangeMax Next 280 Mbps Wireless PC Card
+	7c00  WN511T RangeMax Next 300 Mbps Wireless PC Card
 	f004  FA310TX
 1386  Video Domain Technologies
 1387  Systran Corp
@@ -7931,7 +8128,11 @@
 1414  Microsoft Corporation
 1415  Oxford Semiconductor Ltd
 	8403  VScom 011H-EP1 1 port parallel adaptor
-	9501  OX16PCI954 (Quad 16950 UART) function 0
+	9500  OX16PCI954 (Quad 16950 UART) function 0 (Disabled)
+	9501  OX16PCI954 (Quad 16950 UART) function 0 (Uart)
+		12c4 0202  Titan/cPCI (4 port)
+		12c4 0203  Titan/cPCI (8 port)
+		12c4 0210  Titan/104-Plus (8 port, p1-4)
 		131f 2050  CyberPro (4-port)
 # Model IO1085, Part No: JJ-P46012
 		131f 2051  CyberSerial 4S Plus
@@ -7940,9 +8141,13 @@
 	950a  EXSYS EX-41092 Dual 16950 Serial adapter
 	950b  OXCB950 Cardbus 16950 UART
 	9510  OX16PCI954 (Quad 16950 UART) function 1 (Disabled)
-	9511  OX16PCI954 (Quad 16950 UART) function 1
+		12c4 0200  Titan/cPCI (Unused)
+	9511  OX16PCI954 (Quad 16950 UART) function 1 (8bit bus)
+		12c4 0211  Titan/104-Plus (8 port, p5-8)
 		15ed 2000  MCCR Serial p4-7 of 8
 		15ed 2001  MCCR Serial p4-15 of 16
+	9512  OX16PCI954 (Quad 16950 UART) function 1 (32bit bus)
+	9513  OX16PCI954 (Quad 16950 UART) function 1 (parallel port)
 	9521  OX16PCI952 (Dual 16950 UART)
 	9523  OX16PCI952 Integrated Parallel Port
 1416  Multiwave Innovation pte Ltd
@@ -8053,6 +8258,8 @@
 	9110  GeFORCE FX5200
 	9119  NVIDIA NV31 [GeForce FX 5600XT] VGA Adapter
 	9123  NVIDIA NV31 [GeForce FX 5600] FX5600-VTDR128 [MS-8912]
+	9510  Radeon 9600XT
+	9511  Radeon 9600XT
 	9591  nVidia Corporation NV36 [GeForce FX 5700LE]
 1463  Fast Corporation
 1464  Interactive Circuits & Systems Ltd
@@ -8113,6 +8320,7 @@
 1498  TEWS Datentechnik GmBH
 	0330  TPMC816 2 Channel CAN bus controller.
 	0385  TPMC901 Extended CAN bus with 2/4/6 CAN controller
+	21cc  TCP460 CompactPCI 16 Channel Serial Interface RS232/RS422
 	21cd  TCP461 CompactPCI 8 Channel Serial Interface RS232/RS422
 	30c8  TPCI200
 1499  EMTEC CO., Ltd
@@ -8176,8 +8384,9 @@
 14bf  SPIDER Communications Inc.
 14c0  COMPAL Electronics Inc
 14c1  MYRICOM Inc.
-	0008  Myri-10G Dual-Protocol Interconnect
+	0008  Myri-10G Dual-Protocol NIC (10G-PCIE-8A)
 	8043  Myrinet 2000 Scalable Cluster Interconnect
+		103c 1240  Myrinet M2L-PCI64/2-3.0 LANai 7.4 (HP OEM)
 14c2  DTK Computer
 14c3  MEDIATEK Corp.
 14c4  IWASAKI Information Systems Co Ltd
@@ -8254,6 +8463,7 @@
 	0811  Sentry5 External Interface Core
 	0816  BCM3302 Sentry5 MIPS32 CPU
 	1600  NetXtreme BCM5752 Gigabit Ethernet PCI Express
+		107b 5048  E4500 Onboard
 	1601  NetXtreme BCM5752M Gigabit Ethernet PCI Express
 	1644  NetXtreme BCM5700 Gigabit Ethernet
 		1014 0277  Broadcom Vigil B5700 1000Base-T
@@ -8316,6 +8526,10 @@
 		0e11 00cf  NC7772 Gigabit Server Adapter (PCI-X, 10,100,1000-T)
 		0e11 00d0  NC7782 Gigabit Server Adapter (PCI-X, 10,100,1000-T)
 		0e11 00d1  NC7783 Gigabit Server Adapter (PCI-X, 10,100,1000-T)
+		10a9 8013  Dual Port Gigabit Ethernet (PCI-X,Copper)
+		10a9 8018  Dual Port Gigabit Ethernet (A330)
+		10a9 801a  Dual Port Gigabit Ethernet (IA-blade)
+		10a9 801b  Quad Port Gigabit Ethernet (PCI-E,Copper)
 		10b7 2000  3C998-T Dual Port 10/100/1000 PCI-X
 		10b7 3000  3C999-T Quad Port 10/100/1000 PCI-X
 		1166 1648  NetXtreme CIOB-E 1000Base-T
@@ -8367,6 +8581,7 @@
 	1696  NetXtreme BCM5782 Gigabit Ethernet
 		103c 12bc  HP d530 CMT (DG746A)
 		14e4 000d  NetXtreme BCM5782 1000Base-T
+	169a  NetLink BCM5786 Gigabit Ethernet PCI Express
 	169b  NetLink BCM5787 Gigabit Ethernet PCI Express
 	169c  NetXtreme BCM5788 Gigabit Ethernet
 		103c 308b  MX6125
@@ -8384,6 +8599,8 @@
 		14e4 000b  NetXtreme BCM5703 1000Base-T
 		14e4 800a  NetXtreme BCM5703 1000Base-T
 	16a8  NetXtreme BCM5704S Gigabit Ethernet
+		10a9 8014  Dual Port Gigabit Ethernet (PCI-X,Fiber)
+		10a9 801c  Quad Port Gigabit Ethernet (PCI-E,Fiber)
 		10b7 2001  3C998-SX Dual Port 1000-SX PCI-X
 	16aa  NetXtreme II BCM5706S Gigabit Ethernet
 		103c 3102  NC370F MultifuNCtion Gigabit Server Adapter
@@ -8422,6 +8639,7 @@
 	4306  BCM4307 Ethernet Controller
 	4307  BCM4307 802.11b Wireless LAN Controller
 	4310  BCM4310 Chipcommon I/OController
+	4311  Dell Wireless 1390 WLAN Mini-PCI Card
 	4312  BCM4310 UART
 	4313  BCM4310 Ethernet Controller
 	4315  BCM4310 USB Controller
@@ -8449,6 +8667,7 @@
 		1737 4320  WPC54G
 		1799 7001  Belkin F5D7001 High-Speed Mode Wireless G Network Card
 		1799 7010  Belkin F5D7010 54g Wireless Network card
+		1799 7011  F5D7011 54g+ Wireless Network card
 		185f 1220  TravelMate 290E WLAN Mini-PCI Card
 	4321  BCM4306 802.11a Wireless LAN Controller
 	4322  BCM4306 UART
@@ -8460,7 +8679,9 @@
 		1414 0004  Wireless PCI Adapter MN-730
 # probably this is a correct ID...
 	4326  BCM4307 Chipcommon I/O Controller?
+	4329  BCM43XG
 	4401  BCM4401 100Base-T
+		103c 08b0  tc1100 tablet
 		1043 80a8  A7V8X motherboard
 	4402  BCM4402 Integrated 10/100BaseT
 	4403  BCM4402 V.90 56k Modem
@@ -8890,6 +9111,7 @@
 1540  PROVIDEO MULTIMEDIA Co Ltd
 1541  MACHONE Communications
 1542  Concurrent Computer Corporation
+	9260  RCIM-II Real-Time Clock & Interrupt Module
 1543  SILICON Laboratories
 	3052  Intel 537 [Winmodem]
 	4c22  Si3036 MC'97 DAA
@@ -9184,6 +9406,8 @@
 167b  ZyDAS Technology Corp.
 	2102  ZyDAS ZD1202
 		187e 3406  ZyAIR B-122 CardBus 11Mbs Wireless LAN Card
+167d  Samsung Electro-Mechanics Co., Ltd.
+	a000  IPW2200 miniPCI Wireless
 1681  Hercules
 	0010  Hercules 3d Prophet II Ultra 64MB (350 MHz NV15BR core)
 1682  XFX Pine Group Inc.
@@ -9203,6 +9427,7 @@
 		1186 3a17  D-Link AirPremier DWL-G680 Wireless Cardbus Adapter
 		1186 3a18  D-Link AirPremier DWL-G550 Wireless PCI Adapter
 		1186 3a63  D-Link AirPremier DWL-AG660 Wireless Cardbus Adapter
+		1186 3a93  Conceptronic C54I Wireless 801.11g PCI card
 		1186 3a94  C54C Wireless 801.11g cardbus
 		1186 3ab0  Allnet ALL0281 Wireless PCI Card
 		1385 4d00  Netgear WG311T Wireless PCI Adapter
@@ -9210,10 +9435,11 @@
 		14b7 0a60  8482-WD ORiNOCO 11a/b/g Wireless PCI Adapter
 		168c 0013  AirPlus XtremeG DWL-G650 Wireless PCMCIA Adapter
 		168c 1025  DWL-G650B2 Wireless CardBus Adapter
-		168c 1027  Netgate NL-3054CB ARIES b/g CardBus Adapter
+		168c 1027  Engenius NL-3054CB ARIES b/g CardBus Adapter
+		168c 1042  Ubiquiti Networks SuperRange a/b/g Cardbus Adapter
 		168c 2026  Netgate 5354MP ARIES a(108Mb turbo)/b/g MiniPCI Adapter
-		168c 2041  Netgate 5354MP Plus ARIES2 b/g MiniPCI Adapter
-		168c 2042  Netgate 5354MP Plus ARIES2 a/b/g MiniPCI Adapter
+		168c 2041  Engenius 5354MP Plus ARIES2 b/g MiniPCI Adapter
+		168c 2042  Engenius 5354MP Plus ARIES2 a/b/g MiniPCI Adapter
 		16ab 7302  Trust Speedshare Turbo Pro Wireless PCI Adapter
 		185f 2012  Wistron NeWeb WLAN a+b+g model CB9
 	001a  AR5005G 802.11abg NIC
@@ -9223,12 +9449,13 @@
 		1186 3a16  D-Link AirPlus G DWL-G510 Wireless PCI Adapter(rev.B)
 		1186 3a23  D-Link AirPlus G DWL-G520+A Wireless PCI Adapter
 		1186 3a24  D-Link AirPlus G DWL-G650+A Wireless Cardbus Adapter
+		168c 001a  Belkin FD7000
 		168c 1052  TP-Link TL-WN510G Wireless CardBus Adapter
 	001b  AR5006X 802.11abg NIC
 		1186 3a19  D-Link AirPremier AG DWL-AG660 Wireless Cardbus Adapter
 		1186 3a22  D-Link AirPremier AG DWL-AG530 Wireless PCI Adapter
-		168c 2062  EnGenius EMP-8602 (400mw)
-		168c 2063  EnGenius EMP-8602 (400mw)
+		168c 2062  EnGenius EMP-8602 (400mw) or Compex WLM54AG (SuperAG)
+		168c 2063  EnGenius EMP-8602 (400mw) or Compex WLM54AG
 	0020  AR5005VL 802.11bg Wireless NIC
 	1014  AR5212 802.11abg NIC
 1695  EPoX Computer Co., Ltd.
@@ -9320,6 +9547,7 @@
 	6020  Wireless PCMCIA Card - F5D6020
 	6060  Wireless PDA Card - F5D6060
 	7000  Wireless PCI Card - F5D7000
+	700a  Wireless PCI Card - F5D7000UK
 	7010  BCM4306 802.11b/g Wireless Lan Controller F5D7010
 179c  Data Patterns
 	0557  DP-PCI-557 [PCI 1553B]
@@ -9340,6 +9568,8 @@
 17c0  Wistron Corp.
 17c2  Newisys, Inc.
 17cb  Airgo Networks Inc
+	0001  AGN100 802.11 a/b/g True MIMO Wireless Card
+	0002  AGN300 802.11 a/b/g True MIMO Wireless Card
 17cc  NetChip Technology, Inc
 	2280  USB 2.0
 17cf  Z-Com, Inc.
@@ -9355,7 +9585,11 @@
 17d5  S2io Inc.
 	5831  Xframe 10 Gigabit Ethernet PCI-X
 		103c 12d5  HP PCI-X 133MHz 10GbE SR Fiber
+		10a9 8020  Single Port 10 Gigabit Ethernet (PCI-X, Fiber)
+		10a9 8024  Single Port 10 Gigabit Ethernet (PCI-X, Fiber)
 	5832  Xframe II 10Gbps Ethernet
+		10a9 8021  Single Port 10 Gigabit Ethernet II (PCI-X, Fiber)
+17db  Cray Inc
 17de  KWorld Computer Co. Ltd.
 17ee  Connect Components Ltd
 17f2  Albatron Corp.
@@ -9390,6 +9624,7 @@
 	0301  RT2561/RT61 802.11g PCI
 		1186 3c08  DWL-G630 Rev E
 		1186 3c09  DWL-G510 Rev C
+		1737 0055  WMP54G ver 4.1
 	0302  RT2561/RT61 rev B 802.11g
 		1186 3c08  DWL-G630 Rev E
 		1186 3c09  DWL-G510 Rev C
@@ -9401,11 +9636,14 @@
 # HFC-based ISDN card
 	3069  ISDN PCI DC-105V2
 	9790  WL-121 Wireless Network Adapter 100g+ [Ver.3]
+182e  Raza Microelectronics, Inc.
+	0008  XLR516 Processor
 1830  Credence Systems Corporation
 183b  MikroM GmbH
 	08a7  MVC100 DVI
 	08a8  MVC101 SDI
 	08a9  MVC102 DVI+Audio
+	08b0  MVC200-DC
 1849  ASRock Incorporation
 1851  Microtune, Inc.
 1852  Anritsu Corp.
@@ -9436,7 +9674,8 @@
 18a1  Astute Networks Inc.
 18ac  DViCO Corporation
 	d500  FusionHDTV 5
-	d810  FusionHDTV 3 Gold
+	d800  FusionHDTV 3 Gold
+	d810  FusionHDTV 3 Gold-Q
 	d820  FusionHDTV 3 Gold-T
 18b8  Ammasso
 	b001  AMSO 1100 iWARP/RDMA Gigabit Ethernet Coprocessor
@@ -9468,6 +9707,10 @@
 		18ec d002  COMBO-4SFP
 		18ec d003  COMBO-4SFPRO
 		18ec d004  COMBO-2XFP
+18f6  NextIO
+	1000  [Nexsis] Switch Virtual P2P PCIe Bridge
+	1050  [Nexsis] Switch Virtual P2P PCI Bridge
+	2000  [Nexsis] Switch Integrated Mgmt. Endpoint
 18f7  Commtech, Inc.
 	0001  Fastcom ESCC-PCI-335
 	0002  Fastcom 422/4-PCI-335
@@ -9476,8 +9719,11 @@
 	000a  Fastcom 232/4-PCI-335
 18fb  Resilience Corporation
 1904  Hangzhou Silan Microelectronics Co., Ltd.
+	8139  RTL8139D [Realtek] PCI 10/100BaseTX ethernet adaptor
 1923  Sangoma Technologies Corp.
+	0040  A200/Remora FXO/FXS Analog AFT card
 	0100  A104d QUAD T1/E1 AFT card
+	0300  A101 single-port T1/E1
 	0400  A104u Quad T1/E1 AFT
 1924  Level 5 Networks Inc.
 192e  TransDimension
@@ -9500,6 +9746,7 @@
 196a  Sensory Networks Inc.
 	0101  NodalCore C-1000 Content Classification Accelerator
 	0102  NodalCore C-2000 Content Classification Accelerator
+196d  Club-3D BV
 197b  JMicron Technologies, Inc.
 	2360  JMicron 20360/20363 AHCI Controller
 	2361  JMB361 AHCI/IDE
@@ -9562,6 +9809,7 @@
 	0014  HiNT HC4 PCI to ISDN bridge, Network controller
 	0020  HB6 Universal PCI-PCI bridge (transparent mode)
 	0021  HB6 Universal PCI-PCI bridge (non-transparent mode)
+		1775 ce90  CE9
 		4c53 1050  CT7 mainboard
 		4c53 1080  CT8 mainboard
 		4c53 1090  Cx9 mainboard
@@ -9674,6 +9922,7 @@
 		12ab ffff  MPG600
 		9005 0092  VideOh! AVC-2010
 		9005 0093  VideOh! AVC-2410
+		ffab 0600  ZAP-TV1001 MCE
 	0803  iTVC15 MPEG-2 Encoder
 		0070 4000  WinTV PVR-350
 		0070 4001  WinTV PVR-250
@@ -9899,6 +10148,7 @@
 7063  pcHDTV
 	2000  HD-2000
 	3000  HD-3000
+	5500  HD5500 HDTV
 7604  O.N. Electronic Co Ltd.
 7bde  MIDAC Corporation
 7fed  PowerTV
@@ -10016,8 +10266,8 @@
 		1014 0265  PRO/1000 MT Network Connection
 		1014 0267  PRO/1000 MT Network Connection
 		1014 026a  PRO/1000 MT Network Connection
-		1024 0134  Poweredge SC600
 		1028 002e  Optiplex GX260
+		1028 0134  PowerEdge 600SC
 		1028 0151  PRO/1000 MT Network Connection
 		107b 8920  PRO/1000 MT Desktop Adapter
 		8086 001e  PRO/1000 MT Desktop Adapter
@@ -10037,8 +10287,8 @@
 		4c53 1080  CT8 mainboard
 		4c53 10a0  CA3/CR3 mainboard
 		8086 1011  PRO/1000 MT Dual Port Server Adapter
-		8086 1012  Primergy RX300
-		8086 101a  PRO/1000 MT Dual Port Network Adapter
+		8086 1012  PRO/1000 MT Dual Port Server Adapter
+		8086 101a  PRO/1000 MT Dual Port Network Connection
 		8086 3424  SE7501HG2 Mainboard
 	1011  82545EM Gigabit Ethernet Controller (Fiber)
 		1014 0268  iSeries Gigabit Ethernet Adapter
@@ -10049,9 +10299,11 @@
 		8086 1012  PRO/1000 MF Dual Port Server Adapter
 	1013  82541EI Gigabit Ethernet Controller (Copper)
 		8086 0013  PRO/1000 MT Network Connection
-		8086 1013  IBM ThinkCentre Network Card
+		8086 1013  PRO/1000 MT Network Connection
 		8086 1113  PRO/1000 MT Desktop Adapter
 	1014  82541ER Gigabit Ethernet Controller
+		8086 0014  PRO/1000 MT Desktop Connection
+		8086 1014  PRO/1000 MT Network Connection
 	1015  82540EM Gigabit Ethernet Controller (LOM)
 	1016  82540EP Gigabit Ethernet Controller (LOM)
 		1014 052c  PRO/1000 MT Mobile Connection
@@ -10088,7 +10340,7 @@
 		8086 1003  PRO/1000 MF Server Adapter(LX)
 		8086 1027  PRO/1000 MF Server Adapter
 	1028  82545GM Gigabit Ethernet Controller
-		8086 1028  PRO/1000 MB Server Adapter
+		8086 1028  PRO/1000 MB Server Connection
 	1029  82559 Ethernet Controller
 	1030  82559 InBusiness 10/100
 	1031  82801CAM (ICH3) PRO/100 VE (LOM) Ethernet Controller
@@ -10115,15 +10367,19 @@
 	103b  82801DB PRO/100 VM (LOM) Ethernet Controller
 	103c  82801DB PRO/100 VM (CNR) Ethernet Controller
 	103d  82801DB PRO/100 VE (MOB) Ethernet Controller
+		1014 0522  Thinkpad R50e model 1634
 	103e  82801DB PRO/100 VM (MOB) Ethernet Controller
 	1040  536EP Data Fax Modem
 		16be 1040  V.9X DSP Data Fax Modem
 	1043  PRO/Wireless LAN 2100 3B Mini PCI Adapter
+		103c 08b0  tc1100 tablet
 		8086 2527  MIM2000/Centrino
 	1048  PRO/10GbE LR Server Adapter
 		8086 a01f  PRO/10GbE LR Server Adapter
 		8086 a11f  PRO/10GbE LR Server Adapter
-	104b  Ethernet Controller
+	104a  82566DM Gigabit Network Connection
+	104b  82566DC Gigabit Network Connection
+	104c  82562V 10/100 Network Connection
 	1050  82562EZ 10/100 Ethernet Controller
 		1462 728c  865PE Neo2 (MS-6728)
 		1462 758c  MS-6758 (875P Neo)
@@ -10135,9 +10391,22 @@
 	1053  PRO/100 VM Network Connection
 	1059  82551QM Ethernet Controller
 	105e  82571EB Gigabit Ethernet Controller
+		103c 7044  NC360T PCI Express Dual Port Gigabit Server Adapter
 		1775 6003  Telum GE-QT
+		8086 005e  PRO/1000 PT Dual Port Server Connection
+		8086 105e  PRO/1000 PT Dual Port Network Connection
+		8086 115e  PRO/1000 PT Dual Port Server Adapter
+		8086 116e  PRO/1000 PT Dual Port Server Adapter
+		8086 125e  PRO/1000 PT Dual Port Server Adapter
+		8086 135e  PRO/1000 PT Dual Port Server Adapter
 	105f  82571EB Gigabit Ethernet Controller
+		8086 115f  PRO/1000 PF Dual Port Server Adapter
+		8086 116f  PRO/1000 PF Dual Port Server Adapter
+		8086 125f  PRO/1000 PF Dual Port Server Adapter
+		8086 135f  PRO/1000 PF Dual Port Server Adapter
 	1060  82571EB Gigabit Ethernet Controller
+		8086 0060  PRO/1000 PB Dual Port Server Connection
+		8086 1060  PRO/1000 PB Dual Port Server Connection
 	1064  82562ET/EZ/GT/GZ - PRO/100 VE (LOM) Ethernet Controller
 		1043 80f8  P5GD1-VW Mainboard
 	1065  82562ET/EZ/GT/GZ - PRO/100 VE Ethernet Controller
@@ -10157,16 +10426,18 @@
 		8086 0076  PRO/1000 MT Network Connection
 		8086 1076  PRO/1000 MT Network Connection
 		8086 1176  PRO/1000 MT Desktop Adapter
-		8086 1276  PRO/1000 MT Desktop Adapter
+		8086 1276  PRO/1000 MT Network Adapter
 	1077  82541GI Gigabit Ethernet Controller
 		1179 0001  PRO/1000 MT Mobile Connection
 		8086 0077  PRO/1000 MT Mobile Connection
 		8086 1077  PRO/1000 MT Mobile Connection
 	1078  82541EI Gigabit Ethernet Controller
-		8086 1078  PRO/1000 MT Network Connection
+		8086 1078  82541ER-based Network Connection
 	1079  82546GB Gigabit Ethernet Controller
 		103c 12a6  HP Dual Port 1000Base-T [A9900A]
 		103c 12cf  HP Core Dual Port 1000Base-T [AB352A]
+		1775 10d0  V5D Single Board Computer Gigabit Ethernet
+		1775 ce90  CE9
 		1fc1 0027  Niagara 2261 Failover NIC
 		4c53 1090  Cx9 / Vx9 mainboard
 		4c53 10b0  CL9 mainboard
@@ -10182,30 +10453,40 @@
 		8086 007b  PRO/1000 MB Dual Port Server Connection
 		8086 107b  PRO/1000 MB Dual Port Server Connection
 	107c  82541PI Gigabit Ethernet Controller
+		8086 1376  PRO/1000 GT Desktop Adapter
+		8086 1476  PRO/1000 GT Desktop Adapter
 	107d  82572EI Gigabit Ethernet Controller
+		8086 1082  PRO/1000 PT Server Adapter
+		8086 1092  PRO/1000 PT Server Adapter
 	107e  82572EI Gigabit Ethernet Controller
+		8086 1084  PRO/1000 PF Server Adapter
+		8086 1094  PRO/1000 PF Server Adapter
 	107f  82572EI Gigabit Ethernet Controller
 	1080  FA82537EP 56K V.92 Data/Fax Modem PCI
-	1081  Enterprise Southbridge LAN Copper
-	1082  Enterprise Southbridge LAN fiber
-	1083  Enterprise Southbridge LAN SERDES
-	1084  Enterprise Southbridge IDE Redirection
-	1085  Enterprise Southbridge Serial Port Redirection
-	1086  Enterprise Southbridge IPMI/KCS0
-	1087  Enterprise Southbridge UHCI Redirection
-	1089  Enterprise Southbridge BT
+	1081  631xESB/632xESB LAN Controller Copper
+	1082  631xESB/632xESB LAN Controller fiber
+	1083  631xESB/632xESB LAN Controller SERDES
+	1084  631xESB/632xESB IDE Redirection
+	1085  631xESB/632xESB Serial Port Redirection
+	1086  631xESB/632xESB IPMI/KCS0
+	1087  631xESB/632xESB UHCI Redirection
+	1089  631xESB/632xESB BT
 	108a  82546EB Gigabit Ethernet Controller
+		8086 108a  PRO/1000 P Dual Port Server Adapter
+		8086 118a  PRO/1000 P Dual Port Server Adapter
 	108b  82573V Gigabit Ethernet Controller (Copper)
 	108c  82573E Gigabit Ethernet Controller (Copper)
 	108e  82573E KCS (Active Management)
 	108f  Intel(R) Active Management Technology - SOL
 	1092  Intel(R) PRO/100 VE Network Connection
-	1096  PRO/1000 EB Network Connection with I/O Acceleration
-	1097  Enterprise Southbridge DPT LAN fiber
-	1098  PRO/1000 EB Backplane Connection with I/O Acceleration
+	1096  631xESB/632xESB DPT LAN Controller Copper
+	1097  631xESB/632xESB DPT LAN Controller fiber
+	1098  631xESB/632xESB DPT LAN Controller SERDES
 	1099  82546GB Quad Port Server Adapter
+		8086 1099  PRO/1000 GT Quad Port Server Adapter
 	109a  82573L Gigabit Ethernet Controller
 		17aa 207e  Thinkpad X60s
+		8086 109a  PRO/1000 PL Network Connection
 	109b  82546GB PRO/1000 GF Quad Port Server Adapter
 	10a0  82571EB PRO/1000 AT Quad Port Bypass Adapter
 	10a1  82571EB PRO/1000 AF Quad Port Bypass Adapter
@@ -10215,6 +10496,11 @@
 	10b4  82573L PRO/1000 PL Network Connection
 	10b5  82546GB PRO/1000 GT Quad Port Server Adapter
 		103c 3109  NC340T PCI-X Quad-port Gigabit Server Adapter
+		8086 1099  PRO/1000 GT Quad Port Server Adapter
+		8086 1199  PRO/1000 GT Quad Port Server Adapter
+	10b9  82572EI Gigabit Ethernet Controller (Copper)
+		8086 1083  PRO/1000 PT Desktop Adapter
+		8086 1093  PRO/1000 PT Desktop Adapter
 	1107  PRO/1000 MF Server Adapter (LX)
 	1130  82815 815 Chipset Host Bridge and Memory Controller Hub
 		1025 1016  Travelmate 612 TX
@@ -10315,6 +10601,7 @@
 		144d 2501  SEM-2000 MiniPCI LAN Adapter
 		144d 2502  SEM-2100IL MiniPCI LAN Adapter
 		1668 1100  EtherExpress PRO/100B (TX) (MiniPCI Ethernet+Modem)
+		1775 ce90  CE9
 		4c53 1080  CT8 mainboard
 		4c53 10e0  PSL09 PrPMC
 		8086 0001  EtherExpress PRO/100B (TX)
@@ -10460,7 +10747,7 @@
 	1a30  82845 845 (Brookdale) Chipset Host Bridge
 		1028 010e  Optiplex GX240
 	1a31  82845 845 (Brookdale) Chipset AGP Bridge
-	1a38  Server DMA Engine
+	1a38  5000 Series Chipset DMA Engine
 	1a48  PRO/10GbE SR Server Adapter
 	2410  82801AA ISA Bridge (LPC)
 	2411  82801AA IDE
@@ -10488,6 +10775,7 @@
 	2442  82801BA/BAM USB (Hub #1)
 		1014 01c6  Netvista A40/A40p
 		1025 1016  Travelmate 612 TX
+		1028 00c7  Dimension 8100
 		1028 010e  Optiplex GX240
 		1043 8027  TUSL2-C Mainboard
 		104d 80df  Vaio PCG-FX403
@@ -10497,6 +10785,7 @@
 	2443  82801BA/BAM SMBus
 		1014 01c6  Netvista A40/A40p
 		1025 1016  Travelmate 612 TX
+		1028 00c7  Dimension 8100
 		1028 010e  Optiplex GX240
 		1043 8027  TUSL2-C Mainboard
 		104d 80df  Vaio PCG-FX403
@@ -10505,6 +10794,7 @@
 		8086 4557  D815EGEW Mainboard
 	2444  82801BA/BAM USB (Hub #2)
 		1025 1016  Travelmate 612 TX
+		1028 00c7  Dimension 8100
 		1028 010e  Optiplex GX240
 		1043 8027  TUSL2-C Mainboard
 		104d 80df  Vaio PCG-FX403
@@ -10560,6 +10850,7 @@
 		104d 80df  Vaio PCG-FX403
 	244b  82801BA IDE U100
 		1014 01c6  Netvista A40/A40p
+		1028 00c7  Dimension 8100
 		1028 010e  Optiplex GX240
 		1043 8027  TUSL2-C Mainboard
 		147b 0507  TH7II-RAID
@@ -10632,12 +10923,14 @@
 	24c1  82801DBL (ICH4-L) IDE Controller
 	24c2  82801DB/DBL/DBM (ICH4/ICH4-L/ICH4-M) USB UHCI Controller #1
 		1014 0267  NetVista A30p
+		1014 052d  Thinkpad R50e model 1634
 		1025 005a  TravelMate 290
 		1028 0126  Optiplex GX260
 		1028 0163  Latitude D505
 		1028 0196  Inspiron 5160
 		103c 088c  NC8000 laptop
 		103c 0890  NC6000 laptop
+		103c 08b0  tc1100 tablet
 		1071 8160  MIM2000
 		1462 5800  845PE Max (MS-6580)
 		1509 2990  Averatec 5110H laptop
@@ -10647,10 +10940,12 @@
 		8086 4541  Latitude D400
 	24c3  82801DB/DBL/DBM (ICH4/ICH4-L/ICH4-M) SMBus Controller
 		1014 0267  NetVista A30p
+		1014 052d  Thinkpad R50e model 1634
 		1025 005a  TravelMate 290
 		1028 0126  Optiplex GX260
 		103c 088c  NC8000 laptop
 		103c 0890  NC6000 laptop
+		103c 08b0  tc1100 tablet
 		1071 8160  MIM2000
 		1458 24c2  GA-8PE667 Ultra
 		1462 5800  845PE Max (MS-6580)
@@ -10659,12 +10954,14 @@
 		4c53 1090  Cx9 / Vx9 mainboard
 	24c4  82801DB/DBL/DBM (ICH4/ICH4-L/ICH4-M) USB UHCI Controller #2
 		1014 0267  NetVista A30p
+		1014 052d  Thinkpad R50e model 1634
 		1025 005a  TravelMate 290
 		1028 0126  Optiplex GX260
 		1028 0163  Latitude D505
 		1028 0196  Inspiron 5160
 		103c 088c  NC8000 laptop
 		103c 0890  NC6000 laptop
+		103c 08b0  tc1100 tablet
 		1071 8160  MIM2000
 		1462 5800  845PE Max (MS-6580)
 		1509 2990  Averatec 5110H
@@ -10674,32 +10971,38 @@
 	24c5  82801DB/DBL/DBM (ICH4/ICH4-L/ICH4-M) AC'97 Audio Controller
 		0e11 00b8  Analog Devices Inc. codec [SoundMAX]
 		1014 0267  NetVista A30p
+		1014 055f  Thinkpad R50e model 1634
 		1025 005a  TravelMate 290
 		1028 0139  Latitude D400
 		1028 0163  Latitude D505
 		1028 0196  Inspiron 5160
 		103c 088c  NC8000 laptop
 		103c 0890  NC6000 laptop
+		103c 08b0  tc1100 tablet
 		1071 8160  MIM2000
 		1458 a002  GA-8PE667 Ultra
 		1462 5800  845PE Max (MS-6580)
 		1734 1005  D1451 (SCENIC N300, i845GV) Sigmatel STAC9750T
 		1734 1055  Amilo M1420
 	24c6  82801DB/DBL/DBM (ICH4/ICH4-L/ICH4-M) AC'97 Modem Controller
+		1014 0559  Thinkpad R50e model 1634
 		1025 003c  Aspire 2001WLCi (Compal CL50 motherboard) implementation
 		1025 005a  TravelMate 290
 		1028 0196  Inspiron 5160
 		103c 088c  NC8000 laptop
 		103c 0890  NC6000 laptop
+		103c 08b0  tc1100 tablet
 		1071 8160  MIM2000
 	24c7  82801DB/DBL/DBM (ICH4/ICH4-L/ICH4-M) USB UHCI Controller #3
 		1014 0267  NetVista A30p
+		1014 052d  Thinkpad R50e model 1634
 		1025 005a  TravelMate 290
 		1028 0126  Optiplex GX260
 		1028 0163  Latitude D505
 		1028 0196  Inspiron 5160
 		103c 088c  NC8000 laptop
 		103c 0890  NC6000 laptop
+		103c 08b0  tc1100 tablet
 		1071 8160  MIM2000
 		1462 5800  845PE Max (MS-6580)
 		1509 2990  Averatec 5110H
@@ -10707,11 +11010,13 @@
 		4c53 1090  Cx9 / Vx9 mainboard
 		8086 4541  Latitude D400
 	24ca  82801DBM (ICH4-M) IDE Controller
+		1014 052d  Thinkpad R50e model 1634
 		1025 005a  TravelMate 290
 		1028 0163  Latitude D505
 		1028 0196  Inspiron 5160
 		103c 088c  NC8000 laptop
 		103c 0890  NC6000 laptop
+		103c 08b0  tc1100 tablet
 		1071 8160  MIM2000
 		1734 1055  Amilo M1420
 		8086 4541  Latitude D400
@@ -10726,6 +11031,7 @@
 		1734 1055  Amilo M1420
 	24cd  82801DB/DBM (ICH4/ICH4-M) USB2 EHCI Controller
 		1014 0267  NetVista A30p
+		1014 052e  Thinkpad R50e model 1634
 		1025 005a  TravelMate 290
 		1028 011d  Latitude D600
 		1028 0126  Optiplex GX260
@@ -10734,7 +11040,9 @@
 		1028 0196  Inspiron 5160
 		103c 088c  NC8000 laptop
 		103c 0890  NC6000 laptop
+		103c 08b0  tc1100 tablet
 		1071 8160  MIM2000
+		1179 ff00  Satellite 2430
 		1462 3981  845PE Max (MS-6580)
 		1509 1968  Averatec 5110H
 		1734 1004  D1451 Mainboard (SCENIC N300, i845GV)
@@ -10751,6 +11059,7 @@
 		15d9 4580  P4SCE Mainboard
 		8086 3427  S875WP1-E mainboard
 		8086 4246  Desktop Board D865GBF
+		8086 4c43  Desktop Board D865GLC
 		8086 524c  D865PERL mainboard
 	24d2  82801EB/ER (ICH5/ICH5R) USB UHCI Controller #1
 		1014 02ed  xSeries server mainboard
@@ -10766,6 +11075,7 @@
 		1734 101c  Primergy RX300 S2
 		8086 3427  S875WP1-E mainboard
 		8086 4246  Desktop Board D865GBF
+		8086 4c43  Desktop Board D865GLC
 		8086 524c  D865PERL mainboard
 	24d3  82801EB/ER (ICH5/ICH5R) SMBus Controller
 		1014 02ed  xSeries server mainboard
@@ -10778,6 +11088,7 @@
 		1734 101c  Primergy RX300 S2
 		8086 3427  S875WP1-E mainboard
 		8086 4246  Desktop Board D865GBF
+		8086 4c43  Desktop Board D865GLC
 		8086 524c  D865PERL mainboard
 	24d4  82801EB/ER (ICH5/ICH5R) USB UHCI Controller #2
 		1014 02ed  xSeries server mainboard
@@ -10793,6 +11104,7 @@
 		1734 101c  Primergy RX300 S2
 		8086 3427  S875WP1-E mainboard
 		8086 4246  Desktop Board D865GBF
+		8086 4c43  Desktop Board D865GLC
 		8086 524c  D865PERL mainboard
 	24d5  82801EB/ER (ICH5/ICH5R) AC'97 Audio Controller
 		1028 0169  Precision 470
@@ -10806,6 +11118,8 @@
 		8086 a000  D865PERL mainboard
 		8086 e000  D865PERL mainboard
 		8086 e001  Desktop Board D865GBF
+# onboard Audio in Intel 865glc MoBo
+		8086 e002  SoundMax Intergrated Digital Audio
 	24d6  82801EB/ER (ICH5/ICH5R) AC'97 Modem Controller
 		103c 006a  NX9500
 	24d7  82801EB/ER (ICH5/ICH5R) USB UHCI Controller #3
@@ -10821,6 +11135,7 @@
 		1734 101c  Primergy RX300 S2
 		8086 3427  S875WP1-E mainboard
 		8086 4246  Desktop Board D865GBF
+		8086 4c43  Desktop Board D865GLC
 		8086 524c  D865PERL mainboard
 	24db  82801EB/ER (ICH5/ICH5R) IDE Controller
 		1014 02ed  xSeries server mainboard
@@ -10837,6 +11152,7 @@
 		8086 24db  P4C800 Mainboard
 		8086 3427  S875WP1-E mainboard
 		8086 4246  Desktop Board D865GBF
+		8086 4c43  Desktop Board D865GLC
 		8086 524c  D865PERL mainboard
 	24dc  82801EB (ICH5) LPC Interface Bridge
 	24dd  82801EB/ER (ICH5/ICH5R) USB2 EHCI Controller
@@ -10851,6 +11167,7 @@
 		1462 7280  865PE Neo2 (MS-6728)
 		8086 3427  S875WP1-E mainboard
 		8086 4246  Desktop Board D865GBF
+		8086 4c43  Desktop Board D865GLC
 		8086 524c  D865PERL mainboard
 	24de  82801EB/ER (ICH5/ICH5R) USB UHCI Controller #4
 		1014 02ed  xSeries server mainboard
@@ -10862,6 +11179,7 @@
 		1734 101c  Primergy RX300 S2
 		8086 3427  S875WP1-E mainboard
 		8086 4246  Desktop Board D865GBF
+		8086 4c43  Desktop Board D865GLC
 		8086 524c  D865PERL mainboard
 	24df  82801ER (ICH5R) SATA Controller
 	2500  82820 820 (Camino) Chipset Host Bridge (MCH)
@@ -10874,6 +11192,7 @@
 	2520  82805AA MTH Memory Translator Hub
 	2521  82804AA MRH-S Memory Repeater Hub for SDRAM
 	2530  82850 850 (Tehama) Chipset Host Bridge (MCH)
+		1028 00c7  Dimension 8100
 		147b 0507  TH7II-RAID
 	2531  82860 860 (Wombat) Chipset Host Bridge (MCH)
 	2532  82850 850 (Tehama) Chipset AGP Bridge
@@ -10920,6 +11239,7 @@
 		103c 12bc  D530 sff(dc578av)
 		1043 80a5  P5P800-MX Mainboard
 		8086 4246  Desktop Board D865GBF
+		8086 4c43  Desktop Board D865GLC
 	2573  82865G/PE/P PCI to CSA Bridge
 	2576  82865G/PE/P Processor to I/O Memory Interface
 	2578  82875P/E7210 Memory Controller Hub
@@ -10956,35 +11276,50 @@
 		1043 1881  GMA 900 915GM Integrated Graphics
 	25a1  6300ESB LPC Interface Controller
 	25a2  6300ESB PATA Storage Controller
+		1775 10d0  V5D Single Board Computer IDE
+		1775 ce90  CE9
 		4c53 10b0  CL9 mainboard
 		4c53 10e0  PSL09 PrPMC
 	25a3  6300ESB SATA Storage Controller
+		1775 ce90  CE9
 		4c53 10b0  CL9 mainboard
 		4c53 10d0  Telum ASLP10 Processor AMC
 		4c53 10e0  PSL09 PrPMC
 	25a4  6300ESB SMBus Controller
+		1775 10d0  V5D Single Board Computer
+		1775 ce90  CE9
 		4c53 10b0  CL9 mainboard
 		4c53 10d0  Telum ASLP10 Processor AMC
 		4c53 10e0  PSL09 PrPMC
 	25a6  6300ESB AC'97 Audio Controller
+		1775 ce90  CE9
 		4c53 10b0  CL9 mainboard
 	25a7  6300ESB AC'97 Modem Controller
 	25a9  6300ESB USB Universal Host Controller
+		1775 10d0  V5D Single Board Computer USB
+		1775 ce90  CE9
 		4c53 10b0  CL9 mainboard
 		4c53 10d0  Telum ASLP10 Processor AMC
 		4c53 10e0  PSL09 PrPMC
 	25aa  6300ESB USB Universal Host Controller
+		1775 ce90  CE9
 		4c53 10b0  CL9 mainboard
 		4c53 10e0  PSL09 PrPMC
 	25ab  6300ESB Watchdog Timer
+		1775 10d0  V5D Single Board Computer
+		1775 ce90  CE9
 		4c53 10b0  CL9 mainboard
 		4c53 10d0  Telum ASLP10 Processor AMC
 		4c53 10e0  PSL09 PrPMC
 	25ac  6300ESB I/O Advanced Programmable Interrupt Controller
+		1775 10d0  V5D Single Board Computer
+		1775 ce90  CE9
 		4c53 10b0  CL9 mainboard
 		4c53 10d0  Telum ASLP10 Processor AMC
 		4c53 10e0  PSL09 PrPMC
 	25ad  6300ESB USB2 Enhanced Host Controller
+		1775 10d0  V5D Single Board Computer USB 2.0
+		1775 ce90  CE9
 		4c53 10b0  CL9 mainboard
 		4c53 10d0  Telum ASLP10 Processor AMC
 		4c53 10e0  PSL09 PrPMC
@@ -10992,26 +11327,25 @@
 	25b0  6300ESB SATA RAID Controller
 		4c53 10d0  Telum ASLP10 Processor AMC
 		4c53 10e0  PSL09 PrPMC
-	25c0  Workstation Memory Controller Hub
-	25d0  Server Memory Controller Hub
-	25d4  Server Memory Contoller Hub
-	25d8  Server Memory Controller Hub
-	25e2  Server PCI Express x4 Port 2
-	25e3  Server PCI Express x4 Port 3
-	25e4  Server PCI Express x4 Port 4
-	25e5  Server PCI Express x4 Port 5
-	25e6  Server PCI Express x4 Port 6
-	25e7  Server PCI Express x4 Port 7
-	25e8  Server AMB Memory Mapped Registers
-	25f0  Server Error Reporting Registers
-	25f1  Reserved Registers
-	25f3  Reserved Registers
-	25f5  Server FBD Registers
-	25f6  Server FBD Registers
-	25f7  Server PCI Express x8 Port 2-3
-	25f8  Server PCI Express x8 Port 4-5
-	25f9  Server PCI Express x8 Port 6-7
-	25fa  Server PCI Express x16 Port 4-7
+	25c0  5000X Chipset Memory Controller Hub
+	25d0  5000Z Chipset Memory Controller Hub
+	25d4  5000V Chipset Memory Controller Hub
+	25d8  5000P Chipset Memory Controller Hub
+	25e2  5000 Series Chipset PCI Express x4 Port 2
+	25e3  5000 Series Chipset PCI Express x4 Port 3
+	25e4  5000 Series Chipset PCI Express x4 Port 4
+	25e5  5000 Series Chipset PCI Express x4 Port 5
+	25e6  5000 Series Chipset PCI Express x4 Port 6
+	25e7  5000 Series Chipset PCI Express x4 Port 7
+	25f0  5000 Series Chipset Error Reporting Registers
+	25f1  5000 Series Chipset Reserved Registers
+	25f3  5000 Series Chipset Reserved Registers
+	25f5  5000 Series Chipset FBD Registers
+	25f6  5000 Series Chipset FBD Registers
+	25f7  5000 Series Chipset PCI Express x8 Port 2-3
+	25f8  5000 Series Chipset PCI Express x8 Port 4-5
+	25f9  5000 Series Chipset PCI Express x8 Port 6-7
+	25fa  5000X Chipset PCI Express x16 Port 4-7
 	2600  E8500/E8501 Hub Interface 1.5
 	2601  E8500/E8501 PCI Express x4 Port D
 	2602  E8500/E8501 PCI Express x4 Port C0
@@ -11095,12 +11429,15 @@
 		1458 5006  GA-8I915ME-G Mainboard
 		1462 7028  915P/G Neo2
 		1734 105c  Scenic W620
+		8086 265c  Dimension 3100
 	2660  82801FB/FBM/FR/FW/FRW (ICH6 Family) PCI Express Port 1
 		103c 099c  NX6110/NC6120
 	2662  82801FB/FBM/FR/FW/FRW (ICH6 Family) PCI Express Port 2
 	2664  82801FB/FBM/FR/FW/FRW (ICH6 Family) PCI Express Port 3
 	2666  82801FB/FBM/FR/FW/FRW (ICH6 Family) PCI Express Port 4
 	2668  82801FB/FBM/FR/FW/FRW (ICH6 Family) High Definition Audio Controller
+# based on the PTGD1-LA motherboard
+		103c 2a09  PufferM-UL8E
 		1043 814e  P5GD1-VW Mainboard
 	266a  82801FB/FBM/FR/FW/FRW (ICH6 Family) SMBus Controller
 		1028 0179  Optiplex GX280
@@ -11129,26 +11466,27 @@
 		1458 266f  GA-8I915ME-G Mainboard
 		1462 7028  915P/G Neo2
 		1734 105c  Scenic W620
-	2670  Enterprise Southbridge LPC
-	2680  Enterprise Southbridge SATA IDE
-	2681  Enterprise Southbridge SATA AHCI
-	2682  Enterprise Southbridge SATA RAID
-	2683  Enterprise Southbridge SATA RAID
-	2688  Enterprise Southbridge UHCI USB #1
-	2689  Enterprise Southbridge UHCI USB #2
-	268a  Enterprise Southbridge UHCI USB #3
-	268b  Enterprise Southbridge UHCI USB #4
-	268c  Enterprise Southbridge EHCI USB
-	2690  Enterprise Southbridge PCI Express Root Port 1
-	2692  Enterprise Southbridge PCI Express Root Port 2
-	2694  Enterprise Southbridge PCI Express Root Port 3
-	2696  Enterprise Southbridge PCI Express Root Port 4
-	2698  Enterprise Southbridge AC '97 Audio
-	2699  Enterprise Southbridge AC '97 Modem
-	269a  Enterprise Southbridge High Definition Audio
-	269b  Enterprise Southbridge SMBus
-	269e  Enterprise Southbridge PATA
+	2670  631xESB/632xESB/3100 Chipset LPC Interface Controller
+	2680  631xESB/632xESB/3100 Chipset SATA Storage Controller IDE
+	2681  631xESB/632xESB SATA Storage Controller AHCI
+	2682  631xESB/632xESB SATA Storage Controller RAID
+	2683  631xESB/632xESB SATA Storage Controller RAID
+	2688  631xESB/632xESB/3100 Chipset UHCI USB Controller #1
+	2689  631xESB/632xESB/3100 Chipset UHCI USB Controller #2
+	268a  631xESB/632xESB/3100 Chipset UHCI USB Controller #3
+	268b  631xESB/632xESB/3100 Chipset UHCI USB Controller #4
+	268c  631xESB/632xESB/3100 Chipset EHCI USB2 Controller
+	2690  631xESB/632xESB/3100 Chipset PCI Express Root Port 1
+	2692  631xESB/632xESB/3100 Chipset PCI Express Root Port 2
+	2694  631xESB/632xESB/3100 Chipset PCI Express Root Port 3
+	2696  631xESB/632xESB/3100 Chipset PCI Express Root Port 4
+	2698  631xESB/632xESB AC '97 Audio Controller
+	2699  631xESB/632xESB AC '97 Modem Controller
+	269a  631xESB/632xESB High Definition Audio Controller
+	269b  631xESB/632xESB/3100 Chipset SMBus Controller
+	269e  631xESB/632xESB IDE Controller
 	2770  945G/GZ/P/PL Express Memory Controller Hub
+		107b 5048  E4500
 		8086 544e  DeskTop Board D945GTP
 	2771  945G/GZ/P/PL Express PCI Express Root Port
 	2772  945G/GZ Express Integrated Graphics Controller
@@ -11168,99 +11506,121 @@
 		103c 099c  NX6110/NC6120
 		1043 1881  GMA 900 915GM Integrated Graphics
 	27a0  Mobile 945GM/PM/GMS/940GML and 945GT Express Memory Controller Hub
+		17aa 2017  Thinkpad R60e model 0657
 	27a1  Mobile 945GM/PM/GMS/940GML and 945GT Express PCI Express Root Port
 	27a2  Mobile 945GM/GMS/940GML Express Integrated Graphics Controller
+		17aa 201a  Thinkpad R60e model 0657
 	27a6  Mobile 945GM/GMS/940GML Express Integrated Graphics Controller
+		17aa 201a  Thinkpad R60e model 0657
 	27b0  82801GH (ICH7DH) LPC Interface Bridge
 	27b8  82801GB/GR (ICH7 Family) LPC Interface Bridge
+		107b 5048  E4500
 		8086 544e  DeskTop Board D945GTP
 	27b9  82801GBM (ICH7-M) LPC Interface Bridge
+		17aa 2009  Thinkpad R60e model 0657
 	27bd  82801GHM (ICH7-M DH) LPC Interface Bridge
 	27c0  82801GB/GR/GH (ICH7 Family) Serial ATA Storage Controller IDE
+		107b 5048  E4500
 		8086 544e  DeskTop Board D945GTP
 	27c1  82801GR/GH (ICH7 Family) Serial ATA Storage Controller AHCI
 	27c3  82801GR/GH (ICH7 Family) Serial ATA Storage Controller RAID
 	27c4  82801GBM/GHM (ICH7 Family) Serial ATA Storage Controller IDE
 	27c5  82801GBM/GHM (ICH7 Family) Serial ATA Storage Controller AHCI
+		17aa 200d  Thinkpad R60e model 0657
 	27c6  82801GHM (ICH7-M DH) Serial ATA Storage Controller RAID
 	27c8  82801G (ICH7 Family) USB UHCI #1
+		107b 5048  E4500
+		17aa 200a  Thinkpad R60e model 0657
 		8086 544e  DeskTop Board D945GTP
 	27c9  82801G (ICH7 Family) USB UHCI #2
+		107b 5048  E4500
+		17aa 200a  Thinkpad R60e model 0657
 		8086 544e  DeskTop Board D945GTP
 	27ca  82801G (ICH7 Family) USB UHCI #3
+		107b 5048  E4500
+		17aa 200a  Thinkpad R60e model 0657
 		8086 544e  DeskTop Board D945GTP
 	27cb  82801G (ICH7 Family) USB UHCI #4
+		107b 5048  E4500
+		17aa 200a  Thinkpad R60e model 0657
 		8086 544e  DeskTop Board D945GTP
 	27cc  82801G (ICH7 Family) USB2 EHCI Controller
+		17aa 200b  Thinkpad R60e model 0657
 		8086 544e  DeskTop Board D945GTP
 	27d0  82801G (ICH7 Family) PCI Express Port 1
 	27d2  82801G (ICH7 Family) PCI Express Port 2
 	27d4  82801G (ICH7 Family) PCI Express Port 3
 	27d6  82801G (ICH7 Family) PCI Express Port 4
 	27d8  82801G (ICH7 Family) High Definition Audio Controller
+		107b 5048  E4500
+		17aa 2010  Thinkpad R60e model 0657
 	27da  82801G (ICH7 Family) SMBus Controller
+		17aa 200f  Thinkpad R60e model 0657
 		8086 544e  DeskTop Board D945GTP
 	27dc  82801G (ICH7 Family) LAN Controller
 		8086 308d  DeskTop Board D945GTP
 	27dd  82801G (ICH7 Family) AC'97 Modem Controller
 	27de  82801G (ICH7 Family) AC'97 Audio Controller
 	27df  82801G (ICH7 Family) IDE Controller
+		107b 5048  E4500
+		17aa 200c  Thinkpad R60e model 0657
 		8086 544e  DeskTop Board D945GTP
 	27e0  82801GR/GH/GHM (ICH7 Family) PCI Express Port 5
 	27e2  82801GR/GH/GHM (ICH7 Family) PCI Express Port 6
-	2810  LPC Interface Controller
+	2810  82801HB/HR (ICH8/R) LPC Interface Controller
 	2811  Mobile LPC Interface Controller
-	2812  LPC Interface Controller
-	2814  LPC Interface Controller
+	2812  82801HH (ICH8DH) LPC Interface Controller
+	2814  82801HO (ICH8DO) LPC Interface Controller
 	2815  Mobile LPC Interface Controller
-	2820  SATA Controller 1 IDE
-	2821  SATA Controller AHCI
-	2822  SATA Controller RAID
-	2824  SATA Controller AHCI
-	2825  SATA Controller 2 IDE
-	2828  Mobile SATA Controller IDE
-	2829  Mobile SATA Controller AHCI
-	282a  Mobile SATA Controller RAID
-	2830  USB UHCI Controller #1
-	2831  USB UHCI Controller #2
-	2832  USB UHCI Controller #3
-	2834  USB UHCI Controller #4
-	2835  USB UHCI Controller #5
-	2836  USB2 EHCI Controller #1
-	283a  USB2 EHCI Controller #2
-	283e  SMBus Controller
-	283f  PCI Express Port 1
-	2841  PCI Express Port 2
-	2843  PCI Express Port 3
-	2845  PCI Express Port 4
-	2847  PCI Express Port 5
-	2849  PCI Express Port 6
-	284b  HD Audio Controller
-	284f  Thermal Subsystem
+	2820  82801H (ICH8 Family) 4 port SATA IDE Controller
+	2821  82801HB (ICH8) SATA AHCI Controller
+	2822  82801HR/HO/HH (ICH8R/DO/DH) SATA RAID Controller
+	2824  82801HR/HO/HH (ICH8R/DO/DH) SATA AHCI Controller
+	2825  82801H (ICH8 Family) 2 port SATA IDE Controller
+	2828  Mobile SATA IDE Controller
+	2829  Mobile SATA AHCI Controller
+	282a  Mobile SATA RAID Controller
+	2830  82801H (ICH8 Family) USB UHCI #1
+	2831  82801H (ICH8 Family) USB UHCI #2
+	2832  82801H (ICH8 Family) USB UHCI #3
+	2834  82801H (ICH8 Family) USB UHCI #4
+	2835  82801H (ICH8 Family) USB UHCI #5
+	2836  82801H (ICH8 Family) USB2 EHCI #1
+	283a  82801H (ICH8 Family) USB2 EHCI #2
+	283e  82801H (ICH8 Family) SMBus Controller
+	283f  82801H (ICH8 Family) PCI Express Port 1
+	2841  82801H (ICH8 Family) PCI Express Port 2
+	2843  82801H (ICH8 Family) PCI Express Port 3
+	2845  82801H (ICH8 Family) PCI Express Port 4
+	2847  82801H (ICH8 Family) PCI Express Port 5
+	2849  82801H (ICH8 Family) PCI Express Port 6
+	284b  82801H (ICH8 Family) HD Audio Controller
+	284f  82801H (ICH8 Family) Thermal Reporting Device
 	2850  Mobile IDE Controller
-	2970  Memory Controller Hub
-	2971  PCI Express Root Port
-	2972  Integrated Graphics Controller
-	2973  Integrated Graphics Controller
-	2974  HECI Controller
-	2976  PT IDER Controller
-	2977  KT Controller
-	2990  Memory Controller Hub
-	2991  PCI Express Root Port
-	2992  Integrated Graphics Controller
-	2993  Integrated Graphics Controller
-	2994  HECI Controller
-	2995  HECI Controller
-	2996  PT IDER Controller
-	2997  KT Controller
-	29a0  Memory Controller Hub
-	29a1  PCI Express Root Port
-	29a2  Integrated Graphics Controller
-	29a3  Integrated Graphics Controller
-	29a4  HECI Controller
-	29a5  HECI Controller
-	29a6  PT IDER Controller
-	29a7  KT Controller
+	2970  946GZ/PL/GL Memory Controller Hub
+	2971  946GZ/PL/GL PCI Express Root Port
+	2972  946GZ/GL Integrated Graphics Controller
+	2973  946GZ/GL Integrated Graphics Controller
+	2974  946GZ/GL HECI Controller
+	2975  946GZ/GL HECI Controller
+	2976  946GZ/GL PT IDER Controller
+	2977  946GZ/GL KT Controller
+	2990  Q963/Q965 Memory Controller Hub
+	2991  Q963/Q965 PCI Express Root Port
+	2992  Q963/Q965 Integrated Graphics Controller
+	2993  Q963/Q965 Integrated Graphics Controller
+	2994  Q963/Q965 HECI Controller
+	2995  Q963/Q965 HECI Controller
+	2996  Q963/Q965 PT IDER Controller
+	2997  Q963/Q965 KT Controller
+	29a0  P965/G965 Memory Controller Hub
+	29a1  P965/G965 PCI Express Root Port
+	29a2  G965 Integrated Graphics Controller
+	29a3  G965 Integrated Graphics Controller
+	29a4  P965/G965 HECI Controller
+	29a5  P965/G965 HECI Controller
+	29a6  P965/G965 PT IDER Controller
+	29a7  P965/G965 KT Controller
 	2a00  Mobile Memory Controller Hub
 	2a01  Mobile PCI Express Root Port
 	2a02  Mobile Integrated Graphics Controller
@@ -11271,19 +11631,20 @@
 		1025 005a  TravelMate 290
 		103c 088c  NC8000 laptop
 		103c 0890  NC6000 laptop
+		103c 08b0  HP tc1100 laptop
 	3341  82855PM Processor to AGP Controller
-	3500  Enterprise Southbridge PCI Express Upstream Port
-	3501  Enterprise Southbridge PCI Express Upstream Port
-	3504  Enterprise Southbridge IOxAPIC
-	3505  Enterprise Southbridge IOxAPIC
-	350c  Enterprise Southbridge PCI Express to PCI-X Bridge
-	350d  Enterprise Southbridge PCI Express to PCI-X Bridge
-	3510  Enterprise Southbridge PCI Express Downstream Port E1
-	3511  Enterprise Southbridge PCI Express Downstream Port E1
-	3514  Enterprise Southbridge PCI Express Downstream Port E2
-	3515  Enterprise Southbridge PCI Express Downstream Port E2
-	3518  Enterprise Southbridge PCI Express Downstream Port E3
-	3519  Enterprise Southbridge PCI Express Downstream Port E3
+	3500  6311ESB/6321ESB PCI Express Upstream Port
+	3501  6310ESB PCI Express Upstream Port
+	3504  6311ESB/6321ESB I/OxAPIC Interrupt Controller
+	3505  6310ESB I/OxAPIC Interrupt Controller
+	350c  6311ESB/6321ESB PCI Express to PCI-X Bridge
+	350d  6310ESB PCI Express to PCI-X Bridge
+	3510  6311ESB/6321ESB PCI Express Downstream Port E1
+	3511  6310ESB PCI Express Downstream Port E1
+	3514  6311ESB/6321ESB PCI Express Downstream Port E2
+	3515  6310ESB PCI Express Downstream Port E2
+	3518  6311ESB/6321ESB PCI Express Downstream Port E3
+	3519  6310ESB PCI Express Downstream Port E3
 	3575  82830 830 Chipset Host Bridge
 		0e11 0030  Evo N600c
 		1014 021d  ThinkPad A/T/X Series
@@ -11293,31 +11654,43 @@
 		1014 0513  ThinkPad A/T/X Series
 	3578  82830 830 Chipset Host Bridge
 	3580  82852/82855 GM/GME/PM/GMV Processor to I/O Controller
+		1014 055c  Thinkpad R50e model 1634
 		1028 0139  Latitude D400
 		1028 0163  Latitude D505
 		1028 0196  Inspiron 5160
 		1734 1055  Amilo M1420
+		1775 10d0  V5D Single Board Computer
+		1775 ce90  CE9
 		4c53 10b0  CL9 mainboard
 		4c53 10e0  PSL09 PrPMC
 	3581  82852/82855 GM/GME/PM/GMV Processor to AGP Controller
 		1734 1055  Amilo M1420
 	3582  82852/855GM Integrated Graphics Device
+		1014 0562  Thinkpad R50e model 1634
 		1028 0139  Latitude D400
 		1028 0163  Latitude D505
+		1775 10d0  V5D Single Board Computer VGA
+		1775 ce90  CE9
 		4c53 10b0  CL9 mainboard
 		4c53 10e0  PSL09 PrPMC
 	3584  82852/82855 GM/GME/PM/GMV Processor to I/O Controller
+		1014 055d  Thinkpad R50e model 1634
 		1028 0139  Latitude D400
 		1028 0163  Latitude D505
 		1028 0196  Inspiron 5160
 		1734 1055  Amilo M1420
+		1775 10d0  V5D Single Board Computer
+		1775 ce90  CE9
 		4c53 10b0  CL9 mainboard
 		4c53 10e0  PSL09 PrPMC
 	3585  82852/82855 GM/GME/PM/GMV Processor to I/O Controller
+		1014 055e  Thinkpad R50e model 1634
 		1028 0139  Latitude D400
 		1028 0163  Latitude D505
 		1028 0196  Inspiron 5160
 		1734 1055  Amilo M1420
+		1775 10d0  V5D Single Board Computer
+		1775 ce90  CE9
 		4c53 10b0  CL9 mainboard
 		4c53 10e0  PSL09 PrPMC
 	3590  E7520 Memory Controller Hub
@@ -11340,6 +11713,12 @@
 	359b  E7525/E7520/E7320 Extended Configuration Registers
 	359e  E7525 Memory Controller Hub
 		1028 0169  Precision 470
+	35b0  3100 Chipset Memory I/O Controller Hub
+	35b1  3100 DRAM Controller Error Reporting Registers
+	35b5  3100 Chipset Enhanced DMA Controller
+	35b6  3100 Chipset PCI Express Port A
+	35b7  3100 Chipset PCI Express Port A1
+	35c8  3100 Extended Configuration Test Overflow Registers
 	4220  PRO/Wireless 2200BG Network Connection
 	4222  PRO/Wireless 3945ABG Network Connection
 		8086 1005  PRO/Wireless 3945BG Network Connection
@@ -11349,7 +11728,7 @@
 		1351 103c  Compaq NC6220
 	4224  PRO/Wireless 2915ABG Network Connection
 	4227  PRO/Wireless 3945ABG Network Connection
-		8086 1011  Thinkpad X60s
+		8086 1011  Thinkpad  X60s, R60e model 0657
 		8086 1014  PRO/Wireless 3945BG Network Connection
 	5200  EtherExpress PRO/100 Intelligent Server
 	5201  EtherExpress PRO/100 Intelligent Server
@@ -11457,6 +11836,8 @@
 # observed, and documented in Intel revision note; new mask of 1011:0026
 	b154  21154 PCI-to-PCI Bridge
 	b555  21555 Non transparent PCI-to-PCI Bridge
+		12c7 5005  SS7HD PCI Adaptor Card
+		12c7 5006  SS7HDC cPCI Adaptor Card
 		12d9 000a  PCI VoIP Gateway
 		4c53 1050  CT7 mainboard
 		4c53 1051  CE7 mainboard
@@ -11673,6 +12054,7 @@
 		9005 0299  ASR-4800SAS
 		9005 029a  4805SAS
 	0286  AAC-RAID (Rocket)
+		1014 034d  8s
 		1014 9540  ServeRAID 8k/8k-l4
 		1014 9580  ServeRAID 8k/8k-l8
 		9005 028c  ASR-2230S + ASR-2230SLP PCI-X (Lancer)
@@ -11693,7 +12075,24 @@
 		9005 02a8  AAR-2430SA
 		9005 02a9  ICP5087AU
 		9005 02aa  ICP5047AU
+		9005 02b3  ASR-2400SAS
+		9005 02b4  ICP ICP5045AU
 		9005 0800  Callisto
+	0410  AIC-9410W SAS (Razor HBA RAID)
+		9005 0410  ASC-48300(Spirit RAID)
+		9005 0411  ASC-58300 (Oakmont RAID)                =20
+	0412  AIC-9410W SAS (Razor HBA non-RAID)
+		9005 0412  ASC-48300 (Spirit non-RAID)
+		9005 0413  ASC-58300 (Oakmont non-RAID)
+	041e  AIC-9410W SAS (Razor ASIC non-RAID)
+	041f  AIC-9410W SAS (Razor ASIC RAID)=20
+		9005 041f  AIC-9410W SAS (Razor ASIC RAID)
+	0430  AIC-9405W SAS (Razor-Lite HBA RAID)
+		9005 0430  ASC-44300 (Spirit-Lite RAID)
+	0432  AIC-9405W SAS (Razor-Lite HBA non-RAID)
+		9005 0432  ASC-44300 (Spirit-Lite non-RAID)
+	043e  AIC-9405W SAS (Razor-Lite ASIC non-RAID)=20
+	043f  AIC-9405W SAS (Razor-Lite ASIC RAID)=20
 	0500  Obsidian chipset SCSI controller
 		1014 02c1  PCI-X DDR 3Gb SAS Adapter (572A/572C)
 		1014 02c2  PCI-X DDR 3Gb SAS RAID Adapter (572B/572D)
@@ -11772,6 +12171,7 @@
 	6250  VITC/LTC Timecode Reader card [PCI-VLTC/RDR]
 affe  Sirrix AG security technologies
 	dead  Sirrix.PCI4S0 4-port ISDN S0 interface
+b10b  Uakron PCI Project
 b1b3  Shiva Europe Limited
 # Pinnacle should be 11bd, but they got it wrong several times --mj
 bd11  Pinnacle Systems, Inc. (Wrong ID)
@@ -11811,6 +12211,7 @@
 		0059 0003  128k ISDN-U Adapter
 		00a7 0001  TELES.S0/PCI 2.x ISDN Adapter
 		8086 0003  Digium X100P/X101P analogue PSTN FXO interface
+		b1d9 0003  TDM400P/A400P analogue 4xPSTN FXO/FXS interface
 	0002  Tiger100APC ISDN chipset
 e4bf  EKF Elektronik GmbH
 # Innovative and scalable network IC vendor
--- pciutils-2.2.4~pre4.orig/pcimodules.c
+++ pciutils-2.2.4~pre4/pcimodules.c
@@ -0,0 +1,185 @@
+/*
+ *	pcimodules:  Load all kernel modules for PCI device currently
+ *      plugged into any PCI slot.
+ *
+ *	Copyright 2000 Yggdrasil Computing, Incorporated
+ *	This file may be copied under the terms and conditions of version 
+ *      two of the GNU General Public License, as published by the Free
+ *      Software Foundation (Cambridge, Massachusetts, USA).
+ *
+ *      This file is based on pciutils/lib/example.c, which has the following
+ *      authorship and copyright statement:
+ *
+ *		Written by Martin Mares and put to public domain. You can do
+ *		with it anything you want, but I don't give you any warranty.
+ */
+
+#include <stdlib.h>
+#include <stdio.h>
+#include <malloc.h>
+#include <string.h>
+#include <unistd.h>
+#include <sys/utsname.h>
+#include <sys/param.h>
+#include <sys/types.h>
+
+#define _GNU_SOURCE
+#include <getopt.h>
+
+#include "pciutils.h"
+
+const char program_name[] = "pcimodules";
+
+#define MODDIR	"/lib/modules"
+#define PCIMAP	"modules.pcimap"
+
+#define LINELENGTH	8000 
+
+#define DEVICE_ANY	0xffffffff
+#define VENDOR_ANY	0xffffffff
+
+#include "lib/pci.h"
+
+struct pcimap_entry {
+	unsigned int vendor, subsys_vendor, dev, subsys_dev, class, class_mask;
+	char *module;
+	struct pcimap_entry *next;
+};
+
+static struct pcimap_entry *pcimap_list = NULL;
+
+#define OPT_STRING "h"
+static struct option long_options[] = {
+	{"class",	required_argument,	NULL, 'c'},
+	{"classmask",	required_argument,	NULL, 'm'},
+	{"help",	no_argument,		NULL, 'h'},
+	{ 0,		0,			0, 	0}
+};
+
+static unsigned long desired_class;
+static unsigned long desired_classmask; /* Default is 0: accept all classes.*/
+
+void
+read_pcimap(void)
+{
+	struct utsname utsname;
+	char filename[MAXPATHLEN];
+	FILE *pcimap_file;
+	char line[LINELENGTH];
+	struct pcimap_entry *entry;
+	unsigned int driver_data;
+	char *prevmodule = "";
+	char module[LINELENGTH];
+
+	if (uname(&utsname) < 0) {
+		perror("uname");
+		exit(1);
+	}
+	sprintf(filename, "%s/%s/%s", MODDIR, utsname.release, PCIMAP);
+	if ((pcimap_file = fopen(filename, "r")) == NULL) {
+		perror(filename);
+		exit(1);
+	}
+
+	while(fgets(line, LINELENGTH, pcimap_file) != NULL) {
+		if (line[0] == '#')
+			continue;
+
+		entry = xmalloc(sizeof(struct pcimap_entry));
+
+		if (sscanf(line, "%s 0x%x 0x%x 0x%x 0x%x 0x%x 0x%x 0x%x",
+			   module,
+			   &entry->vendor, &entry->dev,
+			   &entry->subsys_vendor, &entry->subsys_dev,
+			   &entry->class, &entry->class_mask,
+			   &driver_data) != 8) {
+			fprintf (stderr,
+				"modules.pcimap unparsable line: %s.\n", line);
+			free(entry);
+			continue;
+		}
+
+		/* Optimize memory allocation a bit, in case someday we
+		   have Linux systems with ~100,000 modules.  It also
+		   allows us to just compare pointers to avoid trying
+		   to load a module twice. */
+		if (strcmp(module, prevmodule) != 0) {
+			prevmodule = xmalloc(strlen(module)+1);
+			strcpy(prevmodule, module);
+		}
+		entry->module = prevmodule;
+		entry->next = pcimap_list;
+		pcimap_list = entry;
+	}
+	fclose(pcimap_file);
+}
+
+/* Return a filled in pci_access->dev tree, with the device classes
+   stored in dev->aux.
+*/
+static void
+match_pci_modules(void)
+{
+	struct pci_access *pacc;
+	struct pci_dev *dev;
+	unsigned int class, subsys_dev, subsys_vendor;
+	struct pcimap_entry *map;
+	const char *prevmodule = "";
+
+	pacc = pci_alloc();		/* Get the pci_access structure */
+	/* Set all options you want -- here we stick with the defaults */
+	pci_init(pacc);		/* Initialize the PCI library */
+	pci_scan_bus(pacc);	/* We want to get the list of devices */
+  	for(dev=pacc->devices; dev; dev=dev->next) {
+		pci_fill_info(dev, PCI_FILL_IDENT | PCI_FILL_BASES);
+		class = (pci_read_word(dev, PCI_CLASS_DEVICE) << 8)
+			| pci_read_byte(dev, PCI_CLASS_PROG);
+		subsys_dev = pci_read_word(dev, PCI_SUBSYSTEM_ID);
+		subsys_vendor = pci_read_word(dev,PCI_SUBSYSTEM_VENDOR_ID);
+		for(map = pcimap_list; map != NULL; map = map->next) {
+			if (((map->class ^ class) & map->class_mask) == 0 &&
+			    ((desired_class ^ class) & desired_classmask)==0 &&
+			    (map->dev == DEVICE_ANY ||
+			     map->dev == dev->device_id) &&
+			    (map->vendor == VENDOR_ANY ||
+			     map->vendor == dev->vendor_id) &&
+			    (map->subsys_dev == DEVICE_ANY ||
+			     map->subsys_dev == subsys_dev) &&
+			    (map->subsys_vendor == VENDOR_ANY ||
+			     map->subsys_vendor == subsys_vendor) &&
+			    prevmodule != map->module) {
+				printf("%s\n", map->module);
+				prevmodule = map->module;
+			}
+		}
+
+	}
+	pci_cleanup(pacc);
+}
+
+int
+main (int argc, char **argv)
+{
+	int opt_index = 0;
+	int opt;
+
+	while ((opt = getopt_long(argc, argv, OPT_STRING, long_options,
+		           &opt_index)) != -1) {
+		switch(opt) {
+			case 'c':
+				desired_class = strtol(optarg, NULL, 0);
+				break;
+			case 'm':
+				desired_classmask = strtol(optarg, NULL, 0);
+				break;
+			case 'h':
+				printf ("Usage: pcimodules [--help]\n"
+					"  Lists kernel modules corresponding to PCI devices currently plugged"
+					"  into the computer.\n");
+		}
+	}	
+
+	read_pcimap();
+	match_pci_modules();
+	return 0;
+}
--- pciutils-2.2.4~pre4.orig/pcimodules.man
+++ pciutils-2.2.4~pre4/pcimodules.man
@@ -0,0 +1,92 @@
+.TH pcimodules 8 "@TODAY@" "@VERSION@" "Linux PCI Utilities"
+.IX pcimodules
+.SH NAME
+pcimodules \- List kernel driver modules available for all currently plugged
+in PCI devices
+.SH SYNOPSIS
+.B pcimodules
+.RB [ --class class_id ]
+.RB [ --classmask mask ]
+.RB [ --help ]
+.SH DESCRIPTION
+.B pcimodules
+lists all driver modules for all currently plugged in PCI devices.
+.B pcimodules
+should be run at boot time, and whenever a PCI device is "hot plugged"
+into the system.  This can be done by the following Bourne shell syntax:
+.IP
+	for module in $(pcimodules) ; do
+.IP
+		modprobe -s -k "$module"
+.IP
+	done
+.PP
+When a PCI device is removed from the system, the Linux kernel will
+decrement a usage count on PCI driver module.  If this count drops
+to zero (i.e., there are no PCI drivers), then the
+.B modprobe -r
+process that is normally configured to run from cron every few minutes
+will eventually remove the unneeded module.
+.PP
+The --class and --classmask arguments can be used to limit the search
+to certain classes of PCI devices.  This is useful, for example, to
+generate a list of ethernet card drivers to be loaded when the kernel
+has indicated that it is trying to resolve an unknown network interface.
+.PP
+Modules are listed in the order in which the PCI devices are physically
+arranged so that the computer owner can arrange things like having scsi
+device 0 be on a controller that is not alphabetically the first scsi
+controller.
+.SH OPTIONS
+.TP
+.B --class class --classmask mask
+.PP
+--class and --classmask limit the search to PCI
+cards in particular classes.  These arguments are always used together.
+The arguments to --class and --classmask
+can be given as hexadecimal numbers by prefixing a leading "0x".
+Note that the classes used by pcimodules are in "Linux" format,
+meaning the class value that you see with lspci would be shifted
+left eight bits, with the new low eight bits programming interface ID.
+An examples of how to use class and classmask is provided below.
+.B --help, -h
+Print a help message and exit.
+.SH EXAMPLES
+.TP
+pcimodules
+lists all modules corresponding to currently plugged in PCI devices.
+.TP
+pcimodules --class 0x20000 --classmask 0xffff00
+lists all modules corresponding to currently plugged in ethernet PCI devices.
+.SH FILES
+.TP
+.B /lib/modules/<kernel-version>/modules.pcimap
+This file is automatically generated by
+.B depmod,
+and used by
+.B pcimodules
+to determine which modules correspond to which PCI ID's.
+.TP
+.B /proc/bus/pci
+An interface to PCI bus configuration space provided by the post-2.1.82 Linux
+kernels. Contains per-bus subdirectories with per-card config space files and a
+.I devices
+file containing a list of all PCI devices.
+
+.SH SEE ALSO
+.BR lspci (8)
+
+.SH MAINTAINER
+The Linux PCI Utilities are maintained by Martin Mares <mj@suse.cz>.
+
+.SH AUTHOR
+.B pcimodules
+was written by Adam J. Richter <adam@yggdrasil.com>, based on public
+domain example code by Martin Mares <mj@suse.cz>.
+
+.SH COPYRIGHT
+.B pcimodules
+is copyright 2000, Yggdrasil Computing, Incorporated, and may
+be copied under the terms and conditions of version 2 of the GNU
+General Public License as published by the Free Software Foundation
+(Cambridge, Massachusetts, United States of America).
