--- pciutils-2.1.9.orig/Makefile
+++ pciutils-2.1.9/Makefile
@@ -2,9 +2,9 @@
 # Makefile for Linux PCI Utilities
 # (c) 1998--2000 Martin Mares <mj@ucw.cz>
 
-OPT=-O2 -fomit-frame-pointer
+OPT=-O2 -fomit-frame-pointer -g
 #OPT=-O2 -g
-CFLAGS=$(OPT) -Wall -W -Wno-parentheses -Wstrict-prototypes -Werror
+CFLAGS=$(OPT) -Wall -W -Wno-parentheses -Wstrict-prototypes
 
 VERSION=2.1.9
 #SUFFIX=-pre2
@@ -24,8 +24,8 @@
 INSTALL=installbsd
 DIRINSTALL=mkdir -p
 else
-ROOT=/
-PREFIX=/usr
+ROOT=$(DESTDIR)/
+PREFIX=$(DESTDIR)/usr
 endif
 endif
 MANDIR=$(shell if [ -d $(PREFIX)/share/man ] ; then echo $(PREFIX)/share/man ; else echo $(PREFIX)/man ; fi)
@@ -60,7 +60,7 @@
 	# -c is ignored on Linux, but required on FreeBSD
 	$(DIRINSTALL) -m 755 $(ROOT)/sbin $(PREFIX)/share $(MANDIR)/man8
 	$(INSTALL) -c -m 755 -s lspci setpci $(ROOT)/sbin
-	$(INSTALL) -c -m 644 pci.ids $(PREFIX)/share
+	$(INSTALL) -c -m 644 pci.ids $(PREFIX)/share/misc
 	$(INSTALL) -c -m 644 lspci.8 setpci.8 $(MANDIR)/man8
 	# Remove relics from old versions
 	rm -f $(ROOT)/etc/pci.ids
--- pciutils-2.1.9.orig/lib/configure
+++ pciutils-2.1.9/lib/configure
@@ -80,7 +80,7 @@
 if [ -z "$ok" ] ; then
 	echo "WARNING: No real configuration access method is available."
 fi
-echo >>$c "#define PATH_PCI_IDS \"$prefix/share/pci.ids\""
+echo >>$c "#define PATH_PCI_IDS \"$prefix/share/misc/pci.ids\""
 if [ -f header.h ] ; then
 	echo >>$c '#define HAVE_OWN_HEADER_H'
 fi
--- pciutils-2.1.9.orig/lspci.man
+++ pciutils-2.1.9/lspci.man
@@ -75,7 +75,7 @@
 Use
 .B
 <file>
-as PCI ID database instead of /usr/share/pci.ids.
+as PCI ID database instead of /usr/share/misc/pci.ids.
 .TP
 .B -p <dir>
 Use
@@ -136,7 +136,7 @@
 
 .SH FILES
 .TP
-.B /usr/share/pci.ids
+.B /usr/share/misc/pci.ids
 A list of all known PCI ID's (vendors, devices, classes and subclasses).
 .TP
 .B /proc/bus/pci
