From 0766bfa0f557430a6dd3256d45a5b756499f9f7b Mon Sep 17 00:00:00 2001
From: Martin Mares <mj@ucw.cz>
Date: Mon, 1 May 2023 15:44:40 +0200
Origin: upstream, commit:0766bfa0f557430a6dd3256d45a5b756499f9f7b
Subject: [PATCH] Fix stripping in cross-compiling mode

Fixes 3d7466ef8545d37a4666e185a9f5d65ceb9c8af4.
---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 608d3ee..0e878a6 100644
--- a/Makefile
+++ b/Makefile
@@ -45,11 +45,11 @@ PKGCFDIR=$(LIBDIR)/pkgconfig
 # Commands
 INSTALL=install
 DIRINSTALL=install -d
+STRIP=-s
 ifdef CROSS_COMPILE
-STRIP=--strip-program $(CROSS_COMPILE)-strip
+STRIP+=--strip-program $(CROSS_COMPILE)strip
 CC=$(CROSS_COMPILE)gcc
 else
-STRIP=-s
 CC=cc
 endif
 AR=$(CROSS_COMPILE)ar
-- 
2.40.1

