
 pci.ids |  201 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++--------
 1 file changed, 178 insertions(+), 23 deletions(-)

--- a/pci.ids	2013-11-10 23:17:03.000000000 +1100
+++ b/pci.ids	2013-12-27 13:15:02.000000000 +1100
@@ -1,8 +1,8 @@
 #
 #	List of PCI ID's
 #
-#	Version: 2013.11.10
-#	Date:    2013-11-10 13:17:03
+#	Version: 2013.12.27
+#	Date:    2013-12-27 03:15:01
 #
 #	Maintained by Martin Mares <mj@ucw.cz> and other volunteers from the
 #	PCI ID Project at http://pci-ids.ucw.cz/.
@@ -334,6 +334,7 @@
 		1028 1f35  PERC H710 Adapter
 		1028 1f37  PERC H710 Mini (for blades)
 		1028 1f38  PERC H710 Mini (for monolithics)
+		8086 3510  RMS25PB080 RAID Controller
 		8086 3513  RMS25CB080 RAID Controller
 	005c  SAS1064A PCI-X Fusion-MPT SAS
 	005d  MegaRAID SAS-3 3108 [Invader]
@@ -1426,7 +1427,7 @@
 	6606  Mars XTX [Radeon HD 8790M]
 	6607  Mars LE [Radeon HD 8530M]
 	6610  Oland XT [Radeon HD 8670/R7 250]
-	6611  Oland [Radeon HD 8570]
+	6611  Oland [Radeon HD 8570 / R7 240 OEM]
 	6613  Oland PRO [Radeon R7 240]
 	6620  Mars
 	6621  Mars PRO
@@ -1438,11 +1439,13 @@
 	6650  Bonaire
 	6651  Bonaire
 	6658  Bonaire XTX [Radeon R7 260X]
-	665c  Bonaire XT [Radeon HD 7790/8770]
+	665c  Bonaire XT [Radeon HD 7790/8770 / R9 260 OEM]
 		1043 0452  Radeon HD 7790 DirectCU II OC
 # R7790-1GD5/OC
 		1462 2930  Radeon HD 7790 OC
 		1462 2932  Radeon HD 8770
+		1462 2934  Radeon R9 260 OEM
+		148c 9260  Radeon R9 260 OEM
 # FX-779A-CDB4 / FX-779A-CDBC
 		1682 3310  Radeon HD 7790 Black Edition 2 GB
 # 100356OCL / 11210-01-20G
@@ -1450,7 +1453,8 @@
 		1787 2329  Radeon HD 7790 TurboDuo
 	665d  Bonaire [Radeon R7 200 Series]
 	6660  Sun XT [Radeon HD 8670A/8670M/8690M]
-	6663  Sun PRO [Radeon HD 8570M]
+	6663  Sun PRO [Radeon HD 8570A/8570M]
+		1025 0846  Radeon HD 8570A
 	6664  Jet XT [Radeon R5 M200 Series]
 	6665  Jet PRO [Radeon R5 M200 Series]
 	6667  Jet ULT [Radeon R5 M200 Series]
@@ -1880,9 +1884,9 @@
 		17aa 3629  Radeon HD 6450A
 		17aa 363c  Radeon HD 6450A
 		17aa 3658  Radeon HD 7470A
-	6771  Caicos XTX [Radeon HD 8490]
+	6771  Caicos XTX [Radeon HD 8490 / R5 235X OEM]
 	6772  Caicos [Radeon HD 7450A]
-	6778  Caicos XT [Radeon HD 7470/8470]
+	6778  Caicos XT [Radeon HD 7470/8470 / R5 235 OEM]
 		1019 0024  Radeon HD 7470
 		1019 0027  Radeon HD 8470
 		1028 2120  Radeon HD 7470
@@ -1893,7 +1897,7 @@
 		1642 3c75  Radeon HD 8470
 		174b 8145  Radeon HD 8470
 		174b e145  Radeon HD 7470
-	6779  Caicos [Radeon HD 6450/7450/8450]
+	6779  Caicos [Radeon HD 6450/7450/8450 / R5 230 OEM]
 		1019 0016  Radeon HD 6450
 		1019 0017  Radeon HD 6450
 		1019 0018  Radeon HD 6450
@@ -1979,6 +1983,9 @@
 	67a0  Hawaii XT GL
 	67a1  Hawaii GL
 	67a2  Hawaii GL
+	67a8  Hawaii
+	67a9  Hawaii
+	67aa  Hawaii
 	67b0  Hawaii XT [Radeon R9 290X]
 	67b1  Hawaii PRO [Radeon R9 290]
 	67b9  Vesuvius
@@ -2010,7 +2017,7 @@
 		103c 1851  Radeon HD 7750M
 	6821  Venus XT [Radeon HD 8870M]
 		1002 031e  FirePro SX4000
-	6822  Venus PRO
+	6822  Venus PRO [Radeon E8860]
 	6823  Venus PRO [Radeon HD 8850M]
 	6825  Heathrow XT [Radeon HD 7870M]
 		8086 2111  Chelsea PRO
@@ -2025,7 +2032,7 @@
 		103c 1851  Radeon HD 7750M
 	6830  Cape Verde [Radeon HD 7800M Series]
 	6831  Cape Verde [AMD Radeon HD 7700M Series]
-	6835  Cape Verde PRX [Radeon R7 260]
+	6835  Cape Verde PRX [Radeon R9 255 OEM]
 	6837  Cape Verde LE [Radeon HD 7730/8730]
 		1462 2796  Radeon HD 8730
 		1462 8092  Radeon HD 8730
@@ -2996,7 +3003,10 @@
 # AMD A10-5800K CPU
 	9901  Trinity [Radeon HD 7660D]
 	9902  Trinity HDMI Audio Controller
+		103c 194e  ProBook 455 G1 Notebook
 	9903  Trinity [Radeon HD 7640G]
+		103c 194e  ProBook 455 G1 Notebook
+		103c 1952  ProBook 455 G1 Notebook
 	9904  Trinity [Radeon HD 7560D]
 	9905  Trinity [FirePro A300 Series Graphics]
 	9906  Trinity [FirePro A300 Series Graphics]
@@ -3710,25 +3720,42 @@
 	7800  FCH SATA Controller [IDE mode]
 	7801  FCH SATA Controller [AHCI mode]
 		103c 168b  ProBook 4535s Notebook
+		103c 194e  ProBook 455 G1 Notebook
 	7802  FCH SATA Controller [RAID mode]
 	7803  FCH SATA Controller [RAID mode]
 	7804  FCH SATA Controller [AHCI mode]
 	7805  FCH SATA Controller [RAID mode]
 	7806  FCH SD Flash Controller
 	7807  FCH USB OHCI Controller
+		103c 194e  ProBook 455 G1 Notebook
 	7808  FCH USB EHCI Controller
+		103c 194e  ProBook 455 G1 Notebook
 	7809  FCH USB OHCI Controller
+		103c 194e  ProBook 455 G1 Notebook
 	780b  FCH SMBus Controller
+		103c 194e  ProBook 455 G1 Notebook
 	780c  FCH IDE Controller
 	780d  FCH Azalia Controller
+		103c 194e  ProBook 455 G1 Notebook
 		1043 8444  F2A85-M Series
 	780e  FCH LPC Bridge
+		103c 194e  ProBook 455 G1 Notebook
 	780f  FCH PCI Bridge
 	7812  FCH USB XHCI Controller
 	7813  FCH SD Flash Controller
 	7814  FCH USB XHCI Controller
-	7900  CZ SATA Controller [IDE mode]
-	790b  CZ SMBus Controller
+		103c 194e  ProBook 455 G1 Notebook
+	7900  FCH SATA Controller [IDE mode]
+	7901  FCH SATA Controller [AHCI mode]
+	7902  FCH SATA Controller [RAID mode]
+	7903  FCH SATA Controller [RAID mode]
+	7904  FCH SATA Controller [AHCI mode]
+	7906  FCH SD Flash Controller
+	7908  FCH USB EHCI Controller
+	790b  FCH SMBus Controller
+	790e  FCH LPC Bridge
+	790f  FCH PCI Bridge
+	7914  FCH USB XHCI Controller
 	9600  RS780 Host Bridge
 		1043 82f1  M3A78-EH Motherboard
 	9601  RS880 Host Bridge
@@ -8681,8 +8708,8 @@
 # GRID K1 USM
 	0fe7  GK107GL [GRID K100 vGPU]
 		10de 101e  GRID K100
-	0fe9  GK107M [GeForce GT 750M]
-	0fea  GK107M [GeForce GT 755M]
+	0fe9  GK107M [GeForce GT 750M Mac Edition]
+	0fea  GK107M [GeForce GT 755M Mac Edition]
 	0fef  GK107GL [GRID K340]
 	0ff1  GK107 [NVS 1000]
 	0ff2  GK107GL [GRID K1]
@@ -8723,9 +8750,12 @@
 	1020  GK110GL [Tesla K20X]
 	1021  GK110GL [Tesla K20Xm]
 	1022  GK110GL [Tesla K20c]
-	1024  GK110GL [Tesla K40c]
+	1023  GK110BGL [Tesla K40m]
+	1024  GK110BGL [Tesla K40c]
 	1026  GK110GL [Tesla K20s]
+	1027  GK110BGL [Tesla K40st]
 	1028  GK110GL [Tesla K20m]
+	1029  GK110BGL [Tesla K40s]
 	103a  GK110GL [Quadro K6000]
 	1040  GF119 [GeForce GT 520]
 	1042  GF119 [GeForce 510]
@@ -8784,7 +8814,7 @@
 	10c3  GT218 [GeForce 8400 GS Rev. 3]
 	10c5  GT218 [GeForce 405]
 	10d8  GT218 [NVS 300]
-	1140  GF117M [GeForce 610M/710M / GT 620M/625M/630M/720M]
+	1140  GF117M [GeForce 610M/710M/820M / GT 620M/625M/630M/720M]
 		1025 0600  GeForce GT 620M
 		1025 0606  GeForce GT 620M
 		1025 064a  GeForce GT 620M
@@ -8911,6 +8941,7 @@
 		1462 10e9  GeForce GT 720M
 		1462 aa33  GeForce 720M
 		1462 aaa2  GeForce GT 720M
+		1462 aaa3  GeForce 820M
 		1462 ae71  GeForce GT 720M
 		152d 0926  GeForce 620M
 		152d 0982  GeForce GT 630M
@@ -8922,6 +8953,7 @@
 		152d 1072  GeForce GT 720M
 		17aa 2200  NVS 5200M
 		17aa 2213  GeForce GT 720M
+		17aa 2220  GeForce GT 720M
 		17aa 3656  GeForce GT 620M
 		17aa 3800  GeForce GT 720M
 		17aa 3801  GeForce GT 720M
@@ -8953,6 +8985,8 @@
 		17aa 502d  GeForce 710M
 		17aa 502e  GeForce GT 720M
 		17aa 502f  GeForce GT 720M
+		17aa 503e  GeForce 710M
+		17aa 503f  GeForce 820M
 		1854 0177  GeForce 710M
 		1854 0180  GeForce 710M
 		1854 0190  GeForce GT 720M
@@ -8980,8 +9014,8 @@
 	118e  GK104 [GeForce GTX 760 OEM]
 	118f  GK104GL [Tesla K10]
 	1193  GK104 [GeForce GTX 760 Ti OEM]
-	119d  GK104M [GeForce GTX 775M]
-	119e  GK104M [GeForce GTX 780M]
+	119d  GK104M [GeForce GTX 775M Mac Edition]
+	119e  GK104M [GeForce GTX 780M Mac Edition]
 	119f  GK104M [GeForce GTX 780M]
 	11a0  GK104M [GeForce GTX 680M]
 	11a1  GK104M [GeForce GTX 670MX]
@@ -9227,8 +9261,10 @@
 	0139  RTL-8139/8139C/8139C+ Ethernet Controller
 	5209  RTS5209 PCI Express Card Reader
 	5227  RTS5227 PCI Express Card Reader
+		17aa 220e  ThinkPad T440p
 	5229  RTS5229 PCI Express Card Reader
 		1025 0813  Aspire R7-571
+		103c 194e  ProBook 455 G1 Notebook
 	5288  Barossa PCI Express Card Reader
 	8029  RTL-8029(AS)
 		10b8 2011  EZ-Card (SMC1208)
@@ -9304,6 +9340,7 @@
 		1028 04b2  Vostro 3350
 		1028 04da  Vostro 3750
 		103c 1611  Pavilion DM1Z-3000
+		103c 1950  ProBook 450/455
 		1043 11f5  A6J-Q008
 		1043 16d5  U6V/U31J laptop
 		1043 81aa  P5B
@@ -10692,26 +10729,35 @@
 	0041  VIC PCIe Downstream Port
 	0042  VIC Management Controller
 		1137 0047  VIC P81E PCIe Management Controller
+		1137 0085  VIC 1225 PCIe Management Controller
+		1137 00cd  VIC 1285 PCIe Management Controller
+		1137 00ce  VIC 1225T PCIe Management Controller
 	0043  VIC Ethernet NIC
 		1137 0047  VIC P81E PCIe Ethernet NIC
 		1137 0048  VIC M81KR Mezzanine Ethernet NIC
 		1137 004f  VIC 1280 Mezzanine Ethernet NIC
 		1137 0084  VIC 1240 MLOM Ethernet NIC
 		1137 0085  VIC 1225 PCIe Ethernet NIC
+		1137 00cd  VIC 1285 PCIe Ethernet NIC
+		1137 00ce  VIC 1225T PCIe Ethernet NIC
 	0044  VIC Ethernet NIC Dynamic
 		1137 0047  VIC P81E PCIe Ethernet NIC Dynamic
 		1137 0048  VIC M81KR Mezzanine Ethernet NIC Dynamic
 		1137 004f  VIC 1280 Mezzanine Ethernet NIC Dynamic
 		1137 0084  VIC 1240 MLOM Ethernet NIC Dynamic
 		1137 0085  VIC 1225 PCIe Ethernet NIC Dynamic
+		1137 00cd  VIC 1285 PCIe Ethernet NIC Dynamic
+		1137 00ce  VIC 1225T PCIe Ethernet NIC Dynamic
 	0045  VIC FCoE HBA
 		1137 0047  VIC P81E PCIe FCoE HBA
 		1137 0048  VIC M81KR Mezzanine FCoE HBA
 		1137 004f  VIC 1280 Mezzanine FCoE HBA
+		1137 0084  VIC 1240 MLOM FCoE HBA
+		1137 0085  VIC 1225 PCIe FCoE HBA
+		1137 00cd  VIC 1285 PCIe FCoE HBA
+		1137 00ce  VIC 1225T PCIe FCoE HBA
 	004e  VIC 82 PCIe Upstream Port
 	0071  VIC SR-IOV VF
-	0084  VIC 1240 MLOM FCoE HBA
-	0085  VIC 1225 PCIe FCOE HBA
 	00cf  VIC Userspace NIC
 1138  Ziatech Corporation
 	8905  8905 [STD 32 Bridge]
@@ -11878,6 +11924,9 @@
 	7375  PM7375 [LASAR-155 ATM SAR]
 	7384  PM7384 [FREEDM - 84P672 Frm Engine & Datalink Mgr]
 	8000  PM8000  [SPC - SAS Protocol Controller]
+	8032  ATTO Celerity FC8xEN
+		117c 003b  Celerity FC-82EN Fibre Channel Adapter
+		117c 003c  Celerity FC-84EN Fibre Channel Adapter
 11f9  I-Cube Inc
 11fa  Kasan Electronics Company, Ltd.
 11fb  Datel Inc
@@ -14216,6 +14265,8 @@
 144b  Verint Systems Inc.
 144c  Catalina Research Inc
 144d  Samsung Electronics Co Ltd
+	1600  Apple PCIe SSD
+	a800  XP941 PCIe SSD
 	a820  NVMe SSD Controller 171X
 		1028 1f95  Express Flash NVMe XS1715 SSD 400GB
 		1028 1f96  Express Flash NVMe XS1715 SSD 800GB
@@ -15765,8 +15816,10 @@
 15bc  Agilent Technologies
 	0100  HPFC-5600 Tachyon DX2+ FC
 	0103  QX4 PCI Express quad 4-gigabit Fibre Channel controller
-	0105  Celerity FC-42XS Fibre Channel Adapter
+	0105  Celerity FC-44XS/FC-42XS/FC-41XS/FC-44ES/FC-42ES/FC-41ES
 		117c 0022  Celerity FC-42XS Fibre Channel Adapter
+		117c 0025  Celerity FC-44ES Fibre Channel Adapter
+		117c 0026  Celerity FC-42ES Fibre Channel Adapter
 	1100  E8001-66442 PCI Express CIC
 	2922  64 Bit, 133MHz PCI-X Exerciser & Protocol Checker
 	2928  64 Bit, 66MHz PCI Exerciser & Analyzer
@@ -16172,7 +16225,11 @@
 		103c 1838  AR9485/HB125 802.11bgn 1×1 Wi-Fi Adapter
 	0033  AR9580 Wireless Network Adapter
 	0034  AR9462 Wireless Network Adapter
+		1a56 2003  Killer Wireless-N 1202 Half-size Mini PCIe Card
 	0036  QCA9565 / AR9565 Wireless Network Adapter
+	0037  AR9485 Wireless Network Adapter
+# Also used as Gigabyte GC-WB150 on a PCIe-to-mini-PCIe converter
+		1a3b 2100  AW-NB100H 802.11n Wireless Mini PCIe Card
 	003c  QCA988x 802.11ac Wireless Network Adapter
 	0207  AR5210 Wireless Network Adapter [AR5000 802.11a]
 	1014  AR5212 802.11abg NIC
@@ -16307,6 +16364,7 @@
 	0100  NSP2000-SSL crypto accelerator
 170c  YottaYotta Inc.
 1719  EZChip Technologies
+	1000  NPA Access Network Processor Family
 # Seems to be a 2nd ID for Vitesse Semiconductor
 1725  Vitesse Semiconductor
 	7174  VSC7174 PCI/PCI-X Serial ATA Host Bus Controller
@@ -16588,7 +16646,9 @@
 	3091  RT3091 Wireless 802.11n 1T/2R PCIe
 	3092  RT3092 Wireless 802.11n 2T/2R PCIe
 	3290  RT3290 Wireless 802.11n 1T/1R PCIe
+		103c 18ec  Ralink RT3290LE 802.11bgn 1x1 Wi-Fi and Bluetooth 4.0 Combo Adapter
 	3298  RT3290 Bluetooth
+		103c 18ec  Ralink RT3290LE 802.11bgn 1x1 Wi-Fi and Bluetooth 4.0 Combo Adapter
 	3592  RT3592 Wireless 802.11abgn 2T/2R PCIe
 	5360  RT5360 Wireless 802.11n 1T/1R
 		1186 3c05  DWA-525 Wireless N 150 Desktop Adapter (rev.A2)
@@ -17004,6 +17064,8 @@
 	0086  MPC8343E
 	0087  MPC8343
 	00b4  MPC8315E
+	00b6  MPC8314E
+		1a56 1101  Killer Xeno Pro Gigabit Ethernet Controller
 	00c2  MPC8379E
 	00c3  MPC8379
 	00c4  MPC8378E
@@ -17032,6 +17094,8 @@
 	7010  MPC8641 PCI Host Bridge
 	7011  MPC8641D PCI Host Bridge
 	7018  MPC8610
+	c006  MPC8308
+		1a56 1201  Killer E2100 Gigabit Ethernet Controller
 1958  Faster Technology, LLC.
 1959  PA Semi, Inc
 	a000  PA6T Core
@@ -17368,6 +17432,7 @@
 		1043 8438  P8P67 Deluxe Motherboard
 	9172  88SE9172 SATA 6Gb/s Controller
 	917a  88SE9172 SATA III 6Gb/s RAID Controller
+	9183  88SS9183 PCIe SSD Controller
 	9192  88SE9172 SATA III 6Gb/s RAID Controller
 	91a0  88SE912x SATA 6Gb/s Controller [IDE mode]
 	91a4  88SE912x IDE Controller
@@ -17421,6 +17486,11 @@
 1c1c  Symphony
 	0001  82C101
 1c2c  Fiberblaze
+	000a  Capture
+	000f  SmartNIC
+	00a0  FBC4G Capture 4x1Gb
+	00a1  FBC4XG Capture 4x10Gb
+	00a2  FBC8XG Capture 8x10Gb
 # Used on V120 VME Crate Controller
 1c32  Highland Technology, Inc.
 1c33  Daktronics, Inc
@@ -17440,6 +17510,7 @@
 1ce4  Exablaze
 	0001  ExaNIC X4
 	0002  ExaNIC X2
+1cf7  Subspace Dynamics
 1d44  DPT
 	a400  PM2x24/PM3224
 1d5c  Fantasia Trading LLC
@@ -18205,6 +18276,7 @@
 	040a  Xeon E3-1200 v3 Processor Integrated Graphics Controller
 	0412  Xeon E3-1200 v3/4th Gen Core Processor Integrated Graphics Controller
 	0416  4th Gen Core Processor Integrated Graphics Controller
+		17aa 220e  ThinkPad T440p
 	041a  Xeon E3-1200 v3 Processor Integrated Graphics Controller
 	0433  Coleto Creek ACC - ME/CPM interface
 	0435  Coleto Creek PCIe Endpoint
@@ -18424,11 +18496,70 @@
 		8086 8270  Dual Band Wireless AC 3160
 # PowerVR SGX 545
 	08cf  Atom Processor Z2760 Integrated Graphics Controller
+	095a  Wireless 7265
+# Stone Peak 2 AC
+		8086 1010  Dual Band Wireless-AC 7265
+# Stone Peak 2 AGN
+		8086 5000  Dual Band Wireless-N 7265
+# Stone Peak 2 BGN
+		8086 5002  Wireless-N 7265
+# Stone Peak 2 AGN
+		8086 500a  Dual Band Wireless-AC 7265
+# Stone Peak 2 AC
+		8086 5010  Dual Band Wireless-AC 7265
+# Stone Peak 2 AC
+		8086 5012  Dual Band Wireless-AC 7265
+# Stone Peak 2 AGN
+		8086 5020  Dual Band Wireless-N 7265
+# Stone Peak 2 AGN
+		8086 502a  Dual Band Wireless-N 7265
+# Maple Peak AC
+		8086 5090  Dual Band Wireless-AC 7265
+# Stone Peak 2 AC
+		8086 5110  Dual Band Wireless-AC 7265
+# Maple Peak AC
+		8086 5190  Dual Band Wireless-AC 7265
+# Stone Peak 2 AGN
+		8086 5400  Dual Band Wireless-AC 7265
+# Stone Peak 2 AC
+		8086 5410  Dual Band Wireless-AC 7265
+# Stone Peak 2 AGN
+		8086 5420  Dual Band Wireless-N 7265
+# Maple Peak AC
+		8086 5490  Dual Band Wireless-AC 7265
+# Maple Peak AC
+		8086 5590  Dual Band Wireless-AC 7265
+# Stone Peak 2 AC
+		8086 9010  Dual Band Wireless-AC 7265
+# Stone Peak 2 AC
+		8086 9110  Dual Band Wireless-AC 7265
+# Stone Peak 2 AC
+		8086 9210  Dual Band Wireless-AC 7265
+# Stone Peak 2 AC
+		8086 9310  Dual Band Wireless-AC 7265
+# Stone Peak 2 AC
+		8086 9410  Dual Band Wireless-AC 7265
+# Stone Peak 2 AC
+		8086 9510  Dual Band Wireless-AC 7265
+	095b  Wireless 7265
+# Stone Peak 2 AGN
+		8086 5200  Dual Band Wireless-N 7265
+# Stone Peak 2 BGN
+		8086 5202  Wireless-N 7265
+# Stone Peak 2 AC
+		8086 5210  Dual Band Wireless-AC 7265
+# Maple Peak AC
+		8086 5290  Dual Band Wireless-AC 7265
+# Stone Peak 2 BGN
+		8086 5302  Dual Band Wireless-AC 7265
+# Stone Peak 2 AC
+		8086 5310  Dual Band Wireless-AC 7265
 	0960  80960RP (i960RP) Microprocessor/Bridge
 	0962  80960RM (i960RM) Bridge
 	0964  80960RP (i960RP) Microprocessor/Bridge
 	0a04  Haswell-ULT DRAM Controller
 	0a06  Haswell-ULT Integrated Graphics Controller
+	0a0c  Haswell-ULT HD Audio Controller
 	0a16  Haswell-ULT Integrated Graphics Controller
 	0a22  Haswell-ULT Integrated Graphics Controller
 	0a26  Haswell-ULT Integrated Graphics Controller
@@ -18462,10 +18593,12 @@
 	0c00  4th Gen Core Processor DRAM Controller
 	0c01  Xeon E3-1200 v3/4th Gen Core Processor PCI Express x16 Controller
 	0c04  Xeon E3-1200 v3/4th Gen Core Processor DRAM Controller
+		17aa 220e  ThinkPad T440p
 	0c05  Xeon E3-1200 v3/4th Gen Core Processor PCI Express x8 Controller
 	0c08  Xeon E3-1200 v3 Processor DRAM Controller
 	0c09  Xeon E3-1200 v3/4th Gen Core Processor PCI Express x4 Controller
 	0c0c  Xeon E3-1200 v3/4th Gen Core Processor HD Audio Controller
+		17aa 220e  ThinkPad T440p
 	0c46  Atom Processor S1200 PCI Express Root Port 1
 	0c47  Atom Processor S1200 PCI Express Root Port 2
 	0c48  Atom Processor S1200 PCI Express Root Port 3
@@ -19504,6 +19637,7 @@
 	1538  I210 Gigabit Network Connection
 	1539  I211 Gigabit Network Connection
 	153a  Ethernet Connection I217-LM
+		17aa 220e  ThinkPad T440p
 	153b  Ethernet Connection I217-V
 	1547  DSL3510 Thunderbolt Port [Cactus Ridge]
 	1549  DSL3510 Thunderbolt Controller [Cactus Ridge]
@@ -19817,6 +19951,7 @@
 		1043 84ca  P8H77-I Motherboard
 		1849 1e2d  Motherboard
 	1e31  7 Series/C210 Series Chipset Family USB xHCI Host Controller
+		103c 17ab  ProBook 6570b
 		1043 1477  N56VZ
 		1043 1517  Zenbook Prime UX31A
 		1043 84ca  P8H77-I Motherboard
@@ -22906,7 +23041,7 @@
 		1351 103c  Compaq NC6220
 	4224  PRO/Wireless 2915ABG [Calexico2] Network Connection
 	4227  PRO/Wireless 3945ABG [Golan] Network Connection
-		8086 1011  ThinkPad R60e/X60s
+		8086 1011  ThinkPad T60/R60e/X60s
 		8086 1014  PRO/Wireless 3945BG Network Connection
 	4229  PRO/Wireless 4965 AG or AGN [Kedron] Network Connection
 		8086 1100  Vaio VGN-SZ79SN_C
@@ -23195,6 +23330,7 @@
 	8c01  8 Series Chipset Family 4-port SATA Controller 1 [IDE mode] - Mobile
 	8c02  8 Series/C220 Series Chipset Family 6-port SATA Controller 1 [AHCI mode]
 	8c03  8 Series/C220 Series Chipset Family 6-port SATA Controller 1 [AHCI mode]
+		17aa 220e  ThinkPad T440p
 	8c04  8 Series/C220 Series Chipset Family SATA Controller 1 [RAID mode]
 	8c05  8 Series/C220 Series Chipset Family SATA Controller 1 [RAID mode]
 	8c06  8 Series/C220 Series Chipset Family SATA Controller 1 [RAID mode]
@@ -23204,8 +23340,10 @@
 	8c0e  8 Series/C220 Series Chipset Family SATA Controller 1 [RAID mode]
 	8c0f  8 Series/C220 Series Chipset Family SATA Controller 1 [RAID mode]
 	8c10  8 Series/C220 Series Chipset Family PCI Express Root Port #1
+		17aa 220e  ThinkPad T440p
 	8c11  8 Series/C220 Series Chipset Family PCI Express Root Port #1
 	8c12  8 Series/C220 Series Chipset Family PCI Express Root Port #2
+		17aa 220e  ThinkPad T440p
 	8c13  8 Series/C220 Series Chipset Family PCI Express Root Port #2
 	8c14  8 Series/C220 Series Chipset Family PCI Express Root Port #3
 	8c15  8 Series/C220 Series Chipset Family PCI Express Root Port #3
@@ -23220,16 +23358,22 @@
 	8c1e  8 Series/C220 Series Chipset Family PCI Express Root Port #8
 	8c1f  8 Series/C220 Series Chipset Family PCI Express Root Port #8
 	8c20  8 Series/C220 Series Chipset High Definition Audio Controller
+		17aa 220e  ThinkPad T440p
 	8c21  8 Series/C220 Series Chipset High Definition Audio Controller
 	8c22  8 Series/C220 Series Chipset Family SMBus Controller
+		17aa 220e  ThinkPad T440p
 	8c23  8 Series Chipset Family CHAP Counters
 	8c24  8 Series Chipset Family Thermal Management Controller
 	8c26  8 Series/C220 Series Chipset Family USB EHCI #1
+		17aa 220e  ThinkPad T440p
 	8c2d  8 Series/C220 Series Chipset Family USB EHCI #2
+		17aa 220e  ThinkPad T440p
 	8c31  8 Series/C220 Series Chipset Family USB xHCI
+		17aa 220e  ThinkPad T440p
 	8c33  8 Series/C220 Series Chipset Family LAN Controller
 	8c34  8 Series/C220 Series Chipset Family NAND Controller
 	8c3a  8 Series/C220 Series Chipset Family MEI Controller #1
+		17aa 220e  ThinkPad T440p
 	8c3b  8 Series/C220 Series Chipset Family MEI Controller #2
 	8c3c  8 Series/C220 Series Chipset Family IDE-r Controller
 	8c3d  8 Series/C220 Series Chipset Family KT Controller
@@ -23249,6 +23393,7 @@
 	8c4d  8 Series/C220 Series Chipset Family LPC Controller
 	8c4e  Q87 Express LPC Controller
 	8c4f  QM87 Express LPC Controller
+		17aa 220e  ThinkPad T440p
 	8c50  B85 Express LPC Controller
 	8c51  8 Series/C220 Series Chipset Family LPC Controller
 	8c52  C222 Series Chipset Family Server Essential SKU LPC Controller
@@ -23955,9 +24100,16 @@
 	a11d  DeckLink Studio
 	a11e  DeckLink Optical Fibre
 	a121  DeckLink HD Extreme 3D/3D+
+	a124  Intensity Extreme
+	a126  Intensity Shuttle
+	a127  UltraStudio Express
+	a129  UltraStudio Mini Monitor
+	a12a  UltraStudio Mini Recorder
+	a12d  UltraStudio 4K
 	a12e  DeckLink 4K Extreme
 	a12f  DeckLink Mini Monitor
 	a130  DeckLink Mini Recorder
+	a132  UltraStudio 4K
 c001  TSI Telsys
 c0a9  Micron/Crucial Technology
 c0de  Motorola
@@ -24011,11 +24163,14 @@
 dada  Datapath Limited
 db10  Diablo Technologies
 dcba  Dynamic Engineering
-	0046  PCIeAlteraCycloneIV
+	0046  PCIe Altera Cyclone IV
 # VPX format Receiver Controller Board
 	0047  VPX-RCB
 # PMC Format FPGA design with 8 high speed UART channels
 	0048  PMC-Biserial-III-BAE9
+	004e  PC104p-Biserial-III-NVY5
+	004f  PC104p-Biserial-III-NVY6
+	0052  PCIeBiSerialDb37 BA22 LVDS IO
 dd01  Digital Devices GmbH
 	0003  Octopus DVB Adapter
 		dd01 0001  Octopus DVB adapter
