
 pci.ids |  242 +++++++++++++++++++++++++++++++++++++++++++++++++++++++---------
 1 file changed, 208 insertions(+), 34 deletions(-)

--- a/pci.ids	2012-01-15 01:34:18.000000000 +1100
+++ b/pci.ids	2012-02-27 13:15:01.000000000 +1100
@@ -1,8 +1,8 @@
 #
 #	List of PCI ID's
 #
-#	Version: 2012.01.14
-#	Date:    2012-01-14 15:34:18
+#	Version: 2012.02.27
+#	Date:    2012-02-27 03:15:01
 #
 #	Maintained by Martin Mares <mj@ucw.cz> and other volunteers from the
 #	PCI ID Project at http://pci-ids.ucw.cz/.
@@ -464,6 +464,7 @@
 		1000 9262  MegaRAID SAS 9262-8i
 		1000 9263  MegaRAID SAS 9261-8i
 		1000 9264  MegaRAID SAS 9264-8i
+		1000 9267  MegaRAID SAS 9260CV-4i
 		1000 9268  MegaRAID SAS 9260CV-8i
 		1000 9275  MegaRAID SAS 9280-8ex
 		1000 9276  MR9260-16i
@@ -1740,6 +1741,7 @@
 	9480  M96 [Mobility Radeon HD 4650]
 		103c 3628  ATI Mobility Radeon HD 4650 [dv6-1190en]
 	9485  RV740 Pro [Radeon HD 4770]
+	9488  RV730 XT [Mobility Radeon HD 4670]
 	9489  M96 XT [Mobility FireGL V5725]
 	9490  RV730XT [Radeon HD 4670]
 		174b e880  Radeon HD 4670 512MB DDR3
@@ -1768,7 +1770,7 @@
 	94c9  Mobility Radeon HD 2400
 		1002 94c9  Radeon HD2400
 	94cb  Radeon E2400
-	94cc  RV 610LE PCI [Radeon HD 2400]
+	94cc  RV610 LE [Radeon HD 2400 Pro PCI]
 	9501  RV670 [Radeon HD 3870]
 		174b e620  Sapphire Radeon HD 3870 PCIe 2.0
 	9504  RV670 [Mobility Radeon HD 3850]
@@ -1845,6 +1847,7 @@
 	9802  Wrestler [Radeon HD 6310]
 	9804  Wrestler [Radeon HD 6250]
 	9806  Wrestler [Radeon HD 6320]
+	9807  Wrestler [Radeon HD 6290]
 	aa00  R600 Audio Device [Radeon HD 2900 Series]
 	aa08  RV630 audio device [Radeon HD 2600 Series]
 	aa10  RV610 HDMI Audio [Radeon HD 2350/2400 Series]
@@ -3033,7 +3036,7 @@
 	003e  NAPCCARD Cardbus Controller
 	0046  PowerVR PCX2 [midas]
 	005a  Vrc5074 [Nile 4]
-	0063  Firewarden
+	0063  uPD72862 [Firewarden] IEEE1394 OHCI 1.0 Link Controller
 	0067  PowerVR Neon 250 Chipset
 		1010 0020  PowerVR Neon 250 AGP 32Mb
 		1010 0080  PowerVR Neon 250 AGP 16Mb
@@ -3049,17 +3052,17 @@
 	009b  Vrc5476
 	00a5  VRC4173
 	00a6  VRC5477 AC97
-	00cd  IEEE 1394 [OrangeLink] Host Controller
+	00cd  uPD72870 [Firewarden] IEEE1394a OHCI 1.0 Link/3-port PHY Controller
 		12ee 8011  Root hub
-	00ce  IEEE 1394 Host Controller
+	00ce  uPD72871 [Firewarden] IEEE1394a OHCI 1.0 Link/1-port PHY Controller
 	00df  Vr4131
 	00e0  USB 2.0
 		12ee 7001  Root hub
 		14c2 0205  PTI-205N USB 2.0 Host Controller
 		1799 0002  Root Hub
 		807d 1043  PCI-USB2 (EHCI subsystem)
-	00e7  uPD72873 IEEE1394 OHCI 1.1 2-port Host Controller
-	00f2  uPD72874 IEEE1394 OHCI 1.1 3-port PHY-Link Ctrlr
+	00e7  uPD72873 [Firewarden] IEEE1394a OHCI 1.1 Link/2-port PHY Controller
+	00f2  uPD72874 [Firewarden] IEEE1394a OHCI 1.1 Link/3-port PHY Controller
 	00f3  uPD6113x Multimedia Decoder/Processor [EMMA2]
 	010c  VR7701
 	0125  uPD720400 PCI Express - PCI/PCI-X Bridge
@@ -3068,6 +3071,8 @@
 		1028 04b2  Vostro 3350
 		1043 8413  P8P67 Deluxe Motherboard
 		1b96 0001  USB 3.0 PCIe Card
+	01e7  uPD72873 [Firewarden] IEEE1394a OHCI 1.1 Link/2-port PHY Controller
+	01f2  uPD72874 [Firewarden] IEEE1394a OHCI 1.1 Link/3-port PHY Controller
 1034  Framatome Connectors USA Inc.
 1035  Comp. & Comm. Research Lab
 1036  Future Domain Corp.
@@ -3318,6 +3323,7 @@
 	1303  RMP-3 (Remote Management Processor)
 	1361  BCM4312 802.11a/b/g WLAN Controller
 	1371  Broadcom Corporation BCM4312 802.11a/b/g (rev 02)
+	1717  Intel 82571EB dual 1 Gb Ethernet controller
 	2910  E2910A PCIBus Exerciser
 	2925  E2925A 32 Bit, 33 MHzPCI Exerciser & Analyzer
 	3080  Pavilion ze2028ea
@@ -3569,21 +3575,28 @@
 		1102 100f  Graphics Blaster Extreme
 		3d3d 0100  Reference Permedia 2 3D
 	8000  PCILynx/PCILynx2 IEEE 1394 Link Layer Controller
+		105e 8003  FireBoard200
 		1443 8003  FireBoard200
+		1443 8005  FireBoard400
+		1443 8006  FireBoard400
 		e4bf 1010  CF1-1-SNARE
 		e4bf 1020  CF1-2-SNARE
+		e4bf 1040  FireCompact400
 	8009  TSB12LV22 IEEE-1394 Controller
 		104d 8032  8032 OHCI i.LINK (IEEE 1394) Controller
+		1443 8010  FireBoard400-OHCI
 	8017  PCI4410 FireWire Controller
 	8019  TSB12LV23 IEEE-1394 Controller
 		11bd 000a  Studio DV500-1394
 		11bd 000e  Studio DV
+		1443 8010  FireBoard400-OHCI
 		e4bf 1010  CF2-1-CYMBAL
 	8020  TSB12LV26 IEEE-1394 Controller (Link)
 		1028 00d8  Precision 530
 		104d 80e2  VAIO PCV-J200
 		11bd 000f  Studio DV500-1394
 		11bd 001c  Excalibur 4.1
+		1443 8010  FireBoard400-OHCI
 	8021  TSB43AA22 IEEE-1394 Controller (PHY/Link Integrated)
 		104d 80df  Vaio PCG-FX403
 		104d 80e7  VAIO PCG-GR214EP/GR214MP/GR215MP/GR314MP/GR315MP
@@ -3594,12 +3607,17 @@
 		1043 808b  K8N4-E Mainboard
 		1043 815b  P5W DH Deluxe Motherboard
 		1443 8023  FireCard400
+		8086 5044  Desktop Board DP35DP
 	8024  TSB43AB23 IEEE-1394a-2000 Controller (PHY/Link)
 		107d 6620  Winfast DV2000 FireWire Controller
 		1443 8024  FireBoard Blue
 		1458 1000  GA-EP45-DS5 Motherboard
 	8025  TSB82AA2 IEEE-1394b Link Layer Controller
+		1043 813c  P5P series mainboard
+		1443 8025  FireBoard800
 		1458 1000  GA-K8N Ultra-9 Mainboard
+		1546 8025  FWB-PCI01
+		17fc 8025  GIC3800
 	8026  TSB43AB21 IEEE-1394a-2000 Controller (PHY/Link)
 		1025 0035  TravelMate 660
 		1025 003c  Aspire 2001WLCi (Compaq CL50 motherboard)
@@ -4143,7 +4161,7 @@
 	ba57  eXtremeRAID 4000/5000 support Device
 		1069 0072  eXtremeRAID 5000 support Device
 106a  Aten Research Inc
-106b  Apple Computer Inc.
+106b  Apple Inc.
 	0001  Bandit PowerPC host bridge
 	0002  Grand Central I/O
 	0003  Control Video
@@ -4213,6 +4231,7 @@
 	0074  U4 HT Bridge
 	1645  Tigon3 Gigabit Ethernet NIC (BCM5701)
 106c  Hynix Semiconductor
+	8139  8139c 100BaseTX Ethernet Controller
 	8801  Dual Pentium ISA/PCI Motherboard
 	8802  PowerPC ISA/PCI Motherboard
 	8803  Dual Window Graphics Accelerator
@@ -4397,8 +4416,7 @@
 		108d 0017  OC-2250 GoCard 10/100 Ethernet Adapter
 	0021  OC-6151/6152 [RapidFire ATM 155]
 	0022  ATM Adapter
-# formerly Sun Microsystems
-108e  Oracle Corporation
+108e  Oracle/SUN
 	0001  EBUS
 	1000  EBUS
 	1001  Happy Meal 10/100 Ethernet [hme]
@@ -5453,7 +5471,7 @@
 10dd  Evans & Sutherland
 	0100  Lightning 1200
 		10dd 0023  Lightning 1200 15+16M
-10de  nVidia Corporation
+10de  NVIDIA Corporation
 	0008  NV1 [EDGE 3D]
 	0009  NV1 [EDGE 3D]
 	0010  NV2 [Mutara V08]
@@ -6009,6 +6027,7 @@
 	0215  NV48 [GeForce 6800 GT]
 	0218  NV48 [GeForce 6800 XT]
 	0221  NV44A [GeForce 6200]
+		1043 81e1  N6200/TD/256M/A
 		3842 a341  256A8N341DX
 	0222  NV44 [GeForce 6200 A-LE]
 	0240  C51PV [GeForce 6150]
@@ -6273,7 +6292,7 @@
 	0361  MCP55 LPC Bridge
 		1028 0221  PowerEdge R805 MCP55 LPC Bridge
 	0362  MCP55 LPC Bridge
-		147b 12c4  KN9-Ultra Mainboard
+		147b 1c24  KN9 series mainboard
 	0363  MCP55 LPC Bridge
 	0364  MCP55 LPC Bridge
 		1028 0221  PowerEdge R805 MCP55 LPC Bridge
@@ -6283,26 +6302,27 @@
 	0368  MCP55 SMBus
 		1028 020c  PowerEdge M605 MCP55 SMBus
 		1028 0221  PowerEdge R805 MCP55 SMBus
-		147b 12c4  KN9-Ultra Mainboard
+		147b 1c24  KN9 series mainboard
 	0369  MCP55 Memory Controller
-		147b 12c4  KN9-Ultra Mainboard
+		147b 1c24  KN9 series mainboard
 	036a  MCP55 Memory Controller
 	036b  MCP55 SMU
 	036c  MCP55 USB Controller
 		1028 020c  PowerEdge M605 MCP55 USB Controller
 		1028 0221  PowerEdge R805 MCP55 USB Controller
-		147b 12c4  KN9-Ultra Mainboard
+		147b 1c24  KN9 series mainboard
 	036d  MCP55 USB Controller
 		1028 020c  PowerEdge M605 MCP55 USB Controller
 		1028 0221  PowerEdge R805 MCP55 USB Controller
-		147b 12c4  KN9-Ultra Mainboard
+		147b 1c24  KN9 series mainboard
 	036e  MCP55 IDE
-		147b 12c4  KN9-Ultra Mainboard
+		147b 1c24  KN9 series mainboard
 	0370  MCP55 PCI bridge
 	0371  MCP55 High Definition Audio
+		147b 1c24  KN9 series mainboard
 	0372  MCP55 Ethernet
 	0373  MCP55 Ethernet
-		147b 12c4  KN9-Ultra Mainboard
+		147b 1c24  KN9 series mainboard
 	0374  MCP55 PCI Express bridge
 	0375  MCP55 PCI Express bridge
 	0376  MCP55 PCI Express bridge
@@ -6312,7 +6332,7 @@
 	037e  MCP55 SATA Controller
 	037f  MCP55 SATA Controller
 		1028 0221  PowerEdge R805 MCP55 SATA Controller
-		147b 12c4  KN9-Ultra Mainboard
+		147b 1c24  KN9 series mainboard
 	038b  G73 [GeForce 7650 GS]
 	0390  G73 [GeForce 7650 GS]
 	0391  G73 [GeForce 7600 GT]
@@ -6430,8 +6450,10 @@
 		1849 03f6  939NF6G-VSTA Board
 	03f7  MCP61 SATA Controller
 	0400  G84 [GeForce 8600 GTS]
+		1043 8241  EN8600GTS
 	0401  G84 [GeForce 8600GT]
 	0402  G84 [GeForce 8600 GT]
+		1458 3455  GV-NX86T512H
 		1462 0910  NX8600GT-T2D256EZ
 	0403  G84 [GeForce 8600 GS]
 	0404  G84 [GeForce 8400 GS]
@@ -7052,13 +7074,16 @@
 		10de 087e  Leadtek WinFast GTX 570
 	1082  GF114 [GeForce GTX 560 Ti]
 	1086  GF110 [GeForce GTX 570 HD]
+	1091  Tesla M2090
 	1094  Tesla M2075 Dual-Slot Computing Processor Module
+	1096  Tesla C2075
 	10c3  GT218 [GeForce 8400 GS]
 		10de 066d  G98 [GeForce 8400GS]
 	10c5  Dell M116N [GeForce 405]
 	10d8  GT218 [NVS 300]
 	1200  GF110 [GeForce GTX 560 Ti]
 	1201  GF114 [GeForce GTX 560]
+	1205  GF114 [GeForce GTX 460 v2]
 	1244  GF116 [GeForce GTX 550 Ti]
 	1245  GF116 [GeForce GTS 450]
 	1247  GF106 [GeForce GT 555M]
@@ -7221,6 +7246,7 @@
 		10ec 8129  RT8129 Fast Ethernet Adapter
 		11ec 8129  RT8129 Fast Ethernet Adapter
 	8136  RTL8101E/RTL8102E PCI Express Fast Ethernet controller
+		103c 2ab1  Pavillion p6774
 		103c 30cc  Pavilion dv6700
 		1179 ff64  RTL8102E PCI-E Fast Ethernet NIC
 	8138  RT8139 (B/C) Cardbus Fast Ethernet Adapter
@@ -7294,7 +7320,7 @@
 		1462 7522  X58 Pro-E
 		1775 11cc  CC11/CL11
 		1849 8168  Motherboard (one of many)
-		8086 d615  DeskTop Board D510MO
+		8086 d615  Desktop Board D510MO
 	8169  RTL-8169 Gigabit Ethernet
 		1025 0079  Aspire 5024WLMi
 		10bd 3202  EP-320G-TX1 32-bit PCI Gigabit Ethernet Adapter
@@ -7322,6 +7348,7 @@
 		1385 4700  MA521 802.11b Wireless PC Card
 		1737 0019  WPC11v4 802.11b Wireless-B Notebook Adapter
 	8185  RTL-8185 IEEE 802.11a/b/g Wireless LAN Controller
+	8190  RTL8190 802.11n Wireless LAN
 	8191  RTL8188CE 802.11b/g/n WiFi Adapter
 	8192  RTL8192E/RTL8192SE Wireless LAN Controller
 	8197  SmartLAN56 56K Modem
@@ -7332,12 +7359,14 @@
 10ee  Xilinx Corporation
 	0001  EUROCOM for PCI (ECOMP)
 	0002  Octal E1/T1 for PCI ETP Card
-	0007  Zomojo Z1
+	0007  Default PCIe endpoint ID
 	0205  Wildcard TE205P
 	0210  Wildcard TE210P
+	0300  Active Silicon Phoenix Frame Grabbers
 	0314  Wildcard TE405P/TE410P (1st Gen)
 	0405  Wildcard TE405P (2nd Gen)
 	0410  Wildcard TE410P (2nd Gen)
+	0600  Active Silicon Firebird Frame Grabbers
 	2b00  Zomojo Zcard
 	3fc0  RME Digi96
 	3fc1  RME Digi96/8
@@ -7421,6 +7450,7 @@
 		1102 1009  SB Audigy2 OEM HP
 		1102 2002  SB Audigy 2 ZS (SB0350)
 		1102 4001  E-MU 1010
+		1102 4002  E-MU 0404
 	0005  SB X-Fi
 		1102 0021  X-Fi Platinum
 		1102 002c  X-Fi XtremeGamer FATAL1TY PRO
@@ -7495,6 +7525,7 @@
 	2340  RocketRAID 2340 16 Port SATA-II Controller
 	2640  RocketRAID 2640 SAS/SATA Controller
 	2722  RocketRAID 2722
+	3120  RocketRAID 3120
 	3220  RocketRAID 3220
 	3320  RocketRAID 3320
 1104  RasterOps Corp.
@@ -9132,7 +9163,7 @@
 	e230  Memory Stick Host Controller
 	e822  MMC/SD Host Controller
 		1028 040b  Latitude E6510
-	e832  FireWire Host Controller
+	e832  R5C832 PCIe IEEE 1394 Controller
 		1028 040b  Latitude E6510
 1181  Telmatics International
 1183  Fujikura Ltd
@@ -9527,8 +9558,8 @@
 11be  International Microcircuits Inc
 11bf  Astrodesign, Inc.
 11c0  Hewlett Packard
-# Nee Lucent Microelectronics
-11c1  Agere Systems
+# nee Agere Systems nee Lucent Microelectronics
+11c1  LSI Corporation
 	0440  56k WinModem
 		1033 8015  LT WinModem 56k Data+Fax+Voice+Dsvd
 		1033 8047  LT WinModem 56k Data+Fax+Voice+Dsvd
@@ -10123,7 +10154,7 @@
 	1968  ES1968 Maestro 2
 		1028 0085  ES1968 Maestro-2 PCI
 		1033 8051  ES1968 Maestro-2 Audiodrive
-	1969  ES1969 Solo-1 Audiodrive
+	1969  ES1938/ES1946/ES1969 Solo-1 Audiodrive
 		1014 0166  ES1969 SOLO-1 AudioDrive on IBM Aptiva Mainboard
 		125d 8888  Solo-1 Audio Adapter
 		153b 111b  Terratec 128i PCI
@@ -10462,8 +10493,7 @@
 	8152  IT8152F/G Advanced RISC-to-PCI Companion Chip
 	8211  ITE 8211F Single Channel UDMA 133
 		1043 8138  P5GD1-VW Mainboard
-# PCI version seems to be IT8212, embedded seems to be ITE8212
-	8212  IT/ITE8212 Dual channel ATA RAID controller
+	8212  IT8212 Dual channel ATA RAID controller
 		1283 0001  IT/ITE8212 Dual channel ATA RAID controller
 	8213  IT8213 IDE Controller
 	8330  IT8330G
@@ -13747,6 +13777,7 @@
 		1a56 2001  Killer Wireless-N 1103 Half-size Mini PCIe Card [AR9380]
 	0032  AR9485 Wireless Network Adapter
 	0033  AR9580 Wireless Network Adapter
+	0034  AR9462 Wireless Network Adapter
 	0207  AR5210 Wireless Network Adapter [AR5000 802.11a]
 	1014  AR5212 802.11abg NIC
 		1014 058a  ThinkPad 11a/b/g Wireless LAN Mini Express Adapter (AR5BXB6)
@@ -14058,6 +14089,7 @@
 	6061  R6061 USB 2.0 Controller
 17f7  Topdek Semiconductor Inc.
 17f9  Gemtek Technology Co., Ltd
+17fc  IOGEAR, Inc.
 17fe  InProComm Inc.
 	2120  IPN 2120 802.11b
 		1737 0020  WMP11 v4 802.11b Wireless-B PCI Adapter
@@ -14300,6 +14332,7 @@
 18ee  Chenming Mold Ind. Corp.
 18f1  Spectrum GmbH
 18f4  Napatech A/S
+	0031  NT20X Network Adapter
 	0051  NT20-X Capture Card
 	0061  NT20E Capture Card
 	0064  NT20E Inline Card
@@ -14309,6 +14342,7 @@
 	0091  NT20X Capture Card [New Rev]
 	00a1  NT4E-STD Capture Card
 	00a4  NT4E-STD Inline Card
+	00c5  NT20E2 Network Adapter
 18f6  NextIO
 	1000  [Nexsis] Switch Virtual P2P PCIe Bridge
 	1050  [Nexsis] Switch Virtual P2P PCI Bridge
@@ -14387,15 +14421,21 @@
 		1924 6204  SFN5122F-R4
 		1924 6205  SFN5122F-R5
 		1924 6206  SFN5122F-R6
+		1924 6207  SFN5122F-R7
 		1924 6210  SFN5322F-R0
 		1924 6211  SFN5322F-R1
+		1924 6217  SFN5322F-R7
+		1924 6227  SFN6122F-R7
+		1924 6237  SFN6322F-R7
 		1924 6501  SFN5802K-R1
 		1924 6511  SFN5814H-R1
 		1924 6521  SFN5812H-R1
 		1924 6a05  SFN5112F-R5
 		1924 6a06  SFN5112F-R6
 		1924 7206  SFN5162F-R6
+		1924 7207  SFN5162F-R7
 		1924 7a06  SFN5152F-R6
+		1924 7a07  SFN5152F-R7
 	0813  SFL9021 [Solarstorm]
 		1924 6100  SFN5121T-R0
 		1924 6102  SFN5121T-R2
@@ -14545,6 +14585,8 @@
 	1067  L1c Gigabit Ethernet
 	1073  AR8151 v1.0 Gigabit Ethernet
 	1083  AR8151 v2.0 Gigabit Ethernet
+	1090  AR8162 Fast Ethernet
+	1091  AR8161 Gigabit Ethernet
 	2048  L2 Fast Ethernet
 	2060  AR8152 v1.1 Fast Ethernet
 	2062  AR8152 v2.0 Fast Ethernet
@@ -14709,6 +14751,7 @@
 		1a78 0037  tachIOn PCIe SSD [rev 3D]
 		1a78 0038  tachIOn PCIe SSD [rev 4]
 		1a78 0039  tachIOn PCIe SSD [rev 4D]
+		1a78 0040  FlashMAX PCIe SSD [LP Z1]
 1a84  Commex Technologies
 	0001  Vulcan SP HT6210 10-Gigabit Ethernet (rev 02)
 1a88  MEN Mikro Elektronik
@@ -14868,6 +14911,7 @@
 	0001  Spectrum Analyzer PC Card (SAgE)
 1fd4  SUNIX Co., Ltd.
 	0001  Matrix multiport serial adapter
+	1999  Multiport serial controller
 2000  Smart Link Ltd.
 	2800  SmartPCI2800 V.92 PCI Soft DFT
 2001  Temporal Research Ltd
@@ -15028,7 +15072,7 @@
 4321  Tata Power Strategic Electronics Division
 434e  CAST Navigation LLC
 4444  Internext Compression Inc
-	0016  iTVC16 (CX23416) MPEG-2 Encoder
+	0016  iTVC16 (CX23416) Video Decoder
 		0070 0003  WinTV PVR 250
 		0070 0009  WinTV PVR 150
 		0070 0801  WinTV PVR 150
@@ -15550,6 +15594,12 @@
 	0372  80333 Segment-B PCI Express-to-PCI Express Bridge
 	0373  80333 B-Bus IOAPIC
 	0374  80333 Address Translation Unit
+	0402  Haswell Integrated Graphics Controller
+	0406  Haswell Integrated Graphics Controller
+	040a  Haswell Integrated Graphics Controller
+	0412  Haswell Integrated Graphics Controller
+	0416  Haswell Integrated Graphics Controller
+	041a  Haswell Integrated Graphics Controller
 	0436  DH89xxCC Gigabit SGMII Connection
 	0438  DH8900CC Series Gigabit Network Connection
 	043a  DH8900CC Series Gigabit Fiber Network Connection
@@ -15697,6 +15747,39 @@
 	0bf5  Atom Processor D2xxx/N2xxx DRAM Controller
 	0bf6  Atom Processor D2xxx/N2xxx DRAM Controller
 	0bf7  Atom Processor D2xxx/N2xxx DRAM Controller
+	0c00  Haswell DRAM Controller
+	0c01  Haswell PCI Express x16 Controller
+	0c04  Haswell DRAM Controller
+	0c05  Haswell PCI Express x8 Controller
+	0c08  Haswell DRAM Controller
+	0c09  Haswell PCI Express x4 Controller
+	0c0c  Haswell HD Audio Controller
+	0c46  Centerton PCI Express Root Port 1
+	0c47  Centerton PCI Express Root Port 2
+	0c48  Centerton PCI Express Root Port 3
+	0c49  Centerton PCI Express Root Port 4
+	0c4e  Centerton NTB Primary
+	0c54  Centerton Internal Management
+	0c59  Centerton SMBus 2.0 Controller 0
+	0c5a  Centerton SMBus 2.0 Controller 1
+	0c5f  Centerton UART
+	0c60  Centerton Integrated Legacy Bus
+	0c70  Centerton Internal Fabric
+	0c71  Centerton Internal Fabric
+	0c72  Centerton Internal Fabric
+	0c73  Centerton Internal Fabric
+	0c74  Centerton Internal Fabric
+	0c75  Centerton Internal Fabric
+	0c76  Centerton Internal Fabric
+	0c77  Centerton Internal Fabric
+	0c78  Centerton Internal Fabric
+	0c79  Centerton Internal Fabric
+	0c7a  Centerton Internal Fabric
+	0c7b  Centerton Internal Fabric
+	0c7c  Centerton Internal Fabric
+	0c7d  Centerton Internal Fabric
+	0c7e  Centerton Internal Fabric
+	0c7f  Centerton Internal Fabric
 	1000  82542 Gigabit Ethernet Controller (Fiber)
 		0e11 b0df  NC6132 Gigabit Ethernet Adapter (1000-SX)
 		0e11 b0e0  NC6133 Gigabit Ethernet Adapter (1000-LX)
@@ -16002,8 +16085,8 @@
 		17aa 2001  ThinkPad T60
 		17aa 207e  ThinkPad X60s
 		8086 109a  PRO/1000 PL Network Connection
-		8086 309c  DeskTop Board D945GTP
-		8086 30a5  DeskTop Board D975XBX
+		8086 309c  Desktop Board D945GTP
+		8086 30a5  Desktop Board D975XBX
 	109b  82546GB PRO/1000 GF Quad Port Server Adapter
 	109e  82597EX 10GbE Ethernet Controller
 		8086 a01f  PRO/10GbE CX4 Server Adapter
@@ -16450,6 +16533,7 @@
 	154a  Ethernet Server Adapter X520-4
 		8086 011a  Ethernet Converged Network Adapter X520-4
 		8086 011b  Ethernet Converged Network Adapter X520-4
+		8086 011c  Ethernet Converged Network Adapter X520-4
 	154d  82599EB 10-Gigabit SFP+ Network Connection
 		8086 7b11  10GbE 2P X520 Adapter
 	1960  80960RP (i960RP) Microprocessor
@@ -17784,10 +17868,10 @@
 		103c 30a1  NC2400
 		1775 11cc  CC11/CL11 integrated graphics (secondary)
 		17aa 201a  ThinkPad T60/R60 series
-	27ac  Mobile 945GME Express Memory Controller Hub
+	27ac  Mobile 945GSE Express Memory Controller Hub
 		1775 11cc  CC11/CL11
-	27ad  Mobile 945GME Express PCI Express Root Port
-	27ae  Mobile 945GME Express Integrated Graphics Controller
+	27ad  Mobile 945GSE Express PCI Express Root Port
+	27ae  Mobile 945GSE Express Integrated Graphics Controller
 		1775 11cc  CC11/CL11 integrated graphics (primary)
 	27b0  82801GH (ICH7DH) LPC Interface Bridge
 		103c 2a3b  Pavilion A1512X
@@ -18212,6 +18296,7 @@
 		1028 0211  Optiplex 755
 	2916  82801IR (ICH9R) LPC Interface Controller
 		1028 020d  Inspiron 530
+		8086 5044  Desktop Board DP35DP
 	2917  ICH9M-E LPC Interface Controller
 		e4bf cc4d  CCM-BOOGIE
 	2918  82801IB (ICH9) LPC Interface Controller
@@ -18230,6 +18315,7 @@
 		1028 0237  PowerEdge T610 SATA IDE Controller
 		1462 7360  G33/P35 Neo
 	2922  82801IR/IO/IH (ICH9R/DO/DH) 6 port SATA Controller [AHCI mode]
+		8086 5044  Desktop Board DP35DP
 	2923  82801IB (ICH9) 4 port SATA Controller [AHCI mode]
 	2925  82801IR/IO (ICH9R/DO) SATA Controller [RAID mode]
 		1734 10e0  System Board D2542
@@ -18253,6 +18339,7 @@
 		1028 0211  Optiplex 755
 		103c 3628  dv6-1190en
 		1462 7360  G33/P35 Neo
+		8086 5044  Desktop Board DP35DP
 		e4bf cc4d  CCM-BOOGIE
 	2932  82801I (ICH9 Family) Thermal Subsystem
 		103c 3628  dv6-1190en
@@ -18269,6 +18356,7 @@
 		1028 029c  PowerEdge M710 USB UHCI Controller
 		1028 2011  Optiplex 755
 		1462 7360  G33/P35 Neo
+		8086 5044  Desktop Board DP35DP
 		e4bf cc4d  CCM-BOOGIE
 	2935  82801I (ICH9 Family) USB UHCI Controller #2
 		1028 020d  Inspiron 530
@@ -18282,6 +18370,7 @@
 		1028 0287  PowerEdge M610 onboard UHCI
 		1028 029c  PowerEdge M710 USB UHCI Controller
 		1462 7360  G33/P35 Neo
+		8086 5044  Desktop Board DP35DP
 		e4bf cc4d  CCM-BOOGIE
 	2936  82801I (ICH9 Family) USB UHCI Controller #3
 		1028 020d  Inspiron 530
@@ -18293,6 +18382,7 @@
 		1028 0287  PowerEdge M610 onboard UHCI
 		1028 029c  PowerEdge M710 USB UHCI Controller
 		1462 7360  G33/P35 Neo
+		8086 5044  Desktop Board DP35DP
 		e4bf cc4d  CCM-BOOGIE
 	2937  82801I (ICH9 Family) USB UHCI Controller #4
 		1028 020d  Inspiron 530
@@ -18306,6 +18396,7 @@
 		1462 7360  G33/P35 Neo
 		8086 2937  Optiplex 755
 		8086 2942  828011 (ICH9 Family ) USB UHCI Controller
+		8086 5044  Desktop Board DP35DP
 		e4bf cc4d  CCM-BOOGIE
 	2938  82801I (ICH9 Family) USB UHCI Controller #5
 		1028 020d  Inspiron 530
@@ -18317,12 +18408,14 @@
 		1028 029c  PowerEdge M710 USB UHCI Controller
 		1462 7360  G33/P35 Neo
 		8086 2938  Optiplex 755
+		8086 5044  Desktop Board DP35DP
 		e4bf cc4d  CCM-BOOGIE
 	2939  82801I (ICH9 Family) USB UHCI Controller #6
 		1028 020d  Inspiron 530
 		1028 0210  PowerEdge T300 onboard UHCI
 		1028 0237  PowerEdge T610 USB UHCI Controller
 		1462 7360  G33/P35 Neo
+		8086 5044  Desktop Board DP35DP
 		e4bf cc4d  CCM-BOOGIE
 	293a  82801I (ICH9 Family) USB2 EHCI Controller #1
 		1028 020d  Inspiron 530
@@ -18336,6 +18429,7 @@
 		1028 0287  PowerEdge M610 onboard EHCI
 		1028 029c  PowerEdge M710 USB EHCI Controller
 		1462 7360  G33/P35 Neo
+		8086 5044  Desktop Board DP35DP
 		e4bf cc4d  CCM-BOOGIE
 	293c  82801I (ICH9 Family) USB2 EHCI Controller #2
 		1028 020d  Inspiron 530
@@ -18347,6 +18441,7 @@
 		1028 029c  PowerEdge M710 USB EHCI Controller
 		1462 7360  G33/P35 Neo
 		8086 293c  Optiplex 755
+		8086 5044  Desktop Board DP35DP
 		e4bf cc4d  CCM-BOOGIE
 	293e  82801I (ICH9 Family) HD Audio Controller
 		1028 020d  Inspiron 530
@@ -18424,6 +18519,7 @@
 		1028 020d  Inspiron 530
 		1043 82b0  P5KPL-VM Motherboard
 		1462 7360  G33/P35 Neo
+		8086 5044  Desktop Board DP35DP
 	29c1  82G33/G31/P35/P31 Express PCI Express Root Port
 		1028 020d  Inspiron 530
 	29c2  82G33/G31 Express Integrated Graphics Controller
@@ -18433,6 +18529,7 @@
 		1028 020d  Inspiron 530
 		1043 82b0  P5KPL-VM Motherboard
 	29c4  82G33/G31/P35/P31 Express MEI Controller
+		8086 5044  Desktop Board DP35DP
 	29c5  82G33/G31/P35/P31 Express MEI Controller
 	29c6  82G33/G31/P35/P31 Express PT IDER Controller
 	29c7  82G33/G31/P35/P31 Express Serial KT Controller
@@ -19604,6 +19701,80 @@
 	8817  Platform Controller Hub EG20T I2C Controller
 	8818  Platform Controller Hub EG20T Controller Area Network (CAN) Controller
 	8819  Platform Controller Hub EG20T IEEE 1588 Hardware Assist
+	8c00  Lynx Point 4-Port SATA IDE Controller
+	8c01  Lynx Point 4-Port SATA IDE Controller
+	8c02  Lynx Point 6-Port SATA AHCI Controller
+	8c03  Lynx Point 6-Port SATA AHCI Controller
+	8c04  Lynx Point SATA RAID Controller
+	8c05  Lynx Point SATA RAID Controller
+	8c06  Lynx Point SATA RAID Controller
+	8c07  Lynx Point SATA RAID Controller
+	8c08  Lynx Point 2-Port SATA IDE Controller
+	8c09  Lynx Point 2-Port SATA IDE Controller
+	8c0e  Lynx Point SATA RAID Controller
+	8c0f  Lynx Point SATA RAID Controller
+	8c10  Lynx Point PCI Express Root Port 1
+	8c11  Lynx Point PCI Express Root Port 1
+	8c12  Lynx Point PCI Express Root Port 2
+	8c13  Lynx Point PCI Express Root Port 2
+	8c14  Lynx Point PCI Express Root Port 3
+	8c15  Lynx Point PCI Express Root Port 3
+	8c16  Lynx Point PCI Express Root Port 4
+	8c17  Lynx Point PCI Express Root Port 4
+	8c18  Lynx Point PCI Express Root Port 5
+	8c19  Lynx Point PCI Express Root Port 5
+	8c1a  Lynx Point PCI Express Root Port 6
+	8c1b  Lynx Point PCI Express Root Port 6
+	8c1c  Lynx Point PCI Express Root Port 7
+	8c1d  Lynx Point PCI Express Root Port 7
+	8c1e  Lynx Point PCI Express Root Port 8
+	8c1f  Lynx Point PCI Express Root Port 8
+	8c20  Lynx Point HD Audio Controller
+	8c21  Lynx Point HD Audio Controller
+	8c22  Lynx Point SMBus Controller
+	8c23  Lynx Point CHAP Counter
+	8c24  Lynx Point Thermal Management Controller
+	8c26  Lynx Point USB Enhanced Host Controller #1
+	8c2d  Lynx Point USB Enhanced Host Controller #2
+	8c31  Lynx Point USB xHCI Host Controller
+	8c33  Lynx Point LAN Controller
+	8c34  Lynx Point NAND Controller
+	8c3a  Lynx Point MEI Controller #1
+	8c3b  Lynx Point MEI Controller #2
+	8c3c  Lynx Point IDE-r Controller
+	8c3d  Lynx Point KT Controller
+	8c40  Lynx Point LPC Controller
+	8c41  Lynx Point LPC Controller
+	8c42  Lynx Point LPC Controller
+	8c43  Lynx Point LPC Controller
+	8c44  Lynx Point LPC Controller
+	8c45  Lynx Point LPC Controller
+	8c46  Lynx Point LPC Controller
+	8c47  Lynx Point LPC Controller
+	8c48  Lynx Point LPC Controller
+	8c49  Lynx Point LPC Controller
+	8c4a  Lynx Point LPC Controller
+	8c4b  Lynx Point LPC Controller
+	8c4c  Lynx Point LPC Controller
+	8c4d  Lynx Point LPC Controller
+	8c4e  Lynx Point LPC Controller
+	8c4f  Lynx Point LPC Controller
+	8c50  Lynx Point LPC Controller
+	8c51  Lynx Point LPC Controller
+	8c52  Lynx Point LPC Controller
+	8c53  Lynx Point LPC Controller
+	8c54  Lynx Point LPC Controller
+	8c55  Lynx Point LPC Controller
+	8c56  Lynx Point LPC Controller
+	8c57  Lynx Point LPC Controller
+	8c58  Lynx Point LPC Controller
+	8c59  Lynx Point LPC Controller
+	8c5a  Lynx Point LPC Controller
+	8c5b  Lynx Point LPC Controller
+	8c5c  Lynx Point LPC Controller
+	8c5d  Lynx Point LPC Controller
+	8c5e  Lynx Point LPC Controller
+	8c5f  Lynx Point LPC Controller
 	9000  IXP2000 Family Network Processor
 	9001  IXP2400 Network Processor
 	9002  IXP2300 Network Processor
@@ -20032,6 +20203,8 @@
 919a  Gigapixel Corp
 9412  Holtek
 	6565  6565
+9618  JusonTech Corporation
+	0001  JusonTech Gigabit Ethernet Controller
 9699  Omni Media Technology Inc
 	6565  6565
 9710  NetMos Technology
@@ -20107,6 +20280,7 @@ cace  CACE Technologies, Inc.
 	0023  AirPcap N
 cafe  Chrysalis-ITS
 	0003  Luna K3 Hardware Security Module
+	0006  Luna PCI-e 3000 Hardware Security Module
 cccc  Catapult Communications
 ccec  Curtiss-Wright Controls Embedded Computing
 cddd  Tyzx, Inc.
