
 pci.ids | 1006 ++++++++++++++++++++++++++++++++++++++++++++--------------------
 1 file changed, 704 insertions(+), 302 deletions(-)

--- a/pci.ids	2012-01-15 01:34:18.000000000 +1100
+++ b/pci.ids	2012-05-08 13:00:57.000000000 +1000
@@ -1,8 +1,8 @@
 #
 #	List of PCI ID's
 #
-#	Version: 2012.01.14
-#	Date:    2012-01-14 15:34:18
+#	Version: 2012.05.07
+#	Date:    2012-05-07 03:15:01
 #
 #	Maintained by Martin Mares <mj@ucw.cz> and other volunteers from the
 #	PCI ID Project at http://pci-ids.ucw.cz/.
@@ -375,6 +375,8 @@
 	0059  MegaRAID SAS 8208ELP/8208ELP
 	005a  SAS1066E PCI-Express Fusion-MPT SAS
 	005b  MegaRAID SAS 2208 [Thunderbolt]
+		1014 040b  ServeRAID M5110 SAS/SATA Controller
+		1014 0412  ServeRAID M5110e SAS/SATA Controller
 		1028 1f2d  PERC H810 Adapter
 		1028 1f30  PERC H710 Embedded
 		1028 1f31  PERC H710P Adapter
@@ -383,6 +385,7 @@
 		1028 1f35  PERC H710 Adapter
 		1028 1f37  PERC H710 Mini (for blades)
 		1028 1f38  PERC H710 Mini (for monolithics)
+		8086 3513  RMS25PB080 RAID Controller
 	005c  SAS1064A PCI-X Fusion-MPT SAS
 	005d  MegaRAID SAS-3 3108 [Invader]
 	005e  SAS1066 PCI-X Fusion-MPT SAS
@@ -464,6 +467,7 @@
 		1000 9262  MegaRAID SAS 9262-8i
 		1000 9263  MegaRAID SAS 9261-8i
 		1000 9264  MegaRAID SAS 9264-8i
+		1000 9267  MegaRAID SAS 9260CV-4i
 		1000 9268  MegaRAID SAS 9260CV-8i
 		1000 9275  MegaRAID SAS 9280-8ex
 		1000 9276  MR9260-16i
@@ -499,6 +503,7 @@
 	0085  SAS2208 PCI-Express Fusion-MPT SAS-2
 	0086  SAS2308 PCI-Express Fusion-MPT SAS-2
 	0087  SAS2308 PCI-Express Fusion-MPT SAS-2
+		1590 0044  H220i
 	008f  53c875J
 		1092 8000  FirePort 40 SCSI Controller
 		1092 8760  FirePort 40 Dual SCSI Host Adapter
@@ -605,6 +610,7 @@
 # nee ATI Technologies Inc.
 1002  Advanced Micro Devices [AMD] nee ATI
 	1314  Wrestler HDMI Audio [Radeon HD 6250/6310]
+		174b 1001  Sapphire PURE Fusion Mini
 	1714  BeaverCreek HDMI Audio [Radeon HD 6500D and 6400G-6600G series]
 	3150  M24 1P [Radeon Mobility X600]
 		103c 0934  nx8220
@@ -785,6 +791,8 @@
 		1179 ff50  Satellite P305D-S8995E
 		1458 4385  GA-MA770-DS3rev2.0 Motherboard
 		1462 7368  K9AG Neo2
+		15d9 a811  H8DGU
+		174b 1001  Sapphire PURE Fusion Mini
 		17f2 5000  KI690-AM2 Motherboard
 	4386  SB600 USB Controller (EHCI)
 		103c 280a  DC5750 Microtower
@@ -837,6 +845,7 @@
 		103c 1611  Pavilion DM1Z-3000
 		1043 82ef  M3A78-EH Motherboard
 		1043 8443  M5A88-V EVO
+		174b 1001  Sapphire PURE Fusion Mini
 	4392  SB7x0/SB8x0/SB9x0 SATA Controller [Non-RAID5 mode]
 	4393  SB7x0/SB8x0/SB9x0 SATA Controller [RAID5 mode]
 	4394  SB7x0/SB8x0/SB9x0 SATA Controller [AHCI mode]
@@ -845,21 +854,28 @@
 		103c 1611  Pavilion DM1Z-3000
 		1043 82ef  M3A78-EH Motherboard
 		1043 8443  M5A88-V EVO
+		15d9 a811  H8DGU
+		174b 1001  Sapphire PURE Fusion Mini
 	4397  SB7x0/SB8x0/SB9x0 USB OHCI0 Controller
 		103c 1611  Pavilion DM1Z-3000
 		1043 82ef  M3A78-EH Motherboard
 		1043 8443  M5A88-V EVO
+		15d9 a811  H8DGU
+		174b 1001  Sapphire PURE Fusion Mini
 	4398  SB7x0 USB OHCI1 Controller
 		1043 82ef  M3A78-EH Motherboard
+		15d9 a811  H8DGU
 	4399  SB7x0/SB8x0/SB9x0 USB OHCI2 Controller
 		1043 82ef  M3A78-EH Motherboard
 		1043 8443  M5A88-V EVO
+		174b 1001  Sapphire PURE Fusion Mini
 	439c  SB7x0/SB8x0/SB9x0 IDE Controller
 		1043 82ef  M3A78-EH Motherboard
 	439d  SB7x0/SB8x0/SB9x0 LPC host controller
 		103c 1611  Pavilion DM1Z-3000
 		1043 82ef  M3A78-EH Motherboard
 		1043 8443  M5A88-V EVO
+		174b 1001  Sapphire PURE Fusion Mini
 	43a0  SB700/SB800/SB900 PCI to PCI bridge (PCIE port 0)
 	43a1  SB700/SB800/SB900 PCI to PCI bridge (PCIE port 1)
 	43a2  SB900 PCI to PCI bridge (PCIE port 2)
@@ -1391,12 +1407,14 @@
 	5a10  RD890 Northbridge only dual slot (2x16) PCI-e GFX Hydra part
 	5a11  RD890 Northbridge only single slot PCI-e GFX Hydra part
 	5a12  RD890 Northbridge only dual slot (2x8) PCI-e GFX Hydra part
+		15d9 a811  H8DGU
 	5a13  RD890 PCI to PCI bridge (external gfx0 port A)
 	5a14  RD890 PCI to PCI bridge (external gfx0 port B)
 	5a15  RD890 PCI to PCI bridge (PCI express gpp port A)
 	5a16  RD890 PCI to PCI bridge (PCI express gpp port B)
 	5a17  RD890 PCI to PCI bridge (PCI express gpp port C)
 	5a18  RD890 PCI to PCI bridge (PCI express gpp port D)
+		15d9 a811  H8DGU
 	5a19  RD890 PCI to PCI bridge (PCI express gpp port E)
 	5a1a  RD890 PCI to PCI bridge (PCI express gpp port F)
 	5a1b  RD890 PCI to PCI bridge (PCI express gpp port G)
@@ -1404,6 +1422,7 @@
 	5a1d  RD890 PCI to PCI bridge (external gfx1 port A)
 	5a1e  RD890 PCI to PCI bridge (external gfx1 port B)
 	5a1f  RD890 PCI to PCI bridge (NB-SB link)
+		15d9 a811  H8DGU
 	5a23  RD990 I/O Memory Management Unit (IOMMU)
 	5a33  Radeon Xpress 200 Host Bridge
 	5a34  RS480 PCI-X Root Port
@@ -1485,6 +1504,7 @@
 	5e6d  RV410 [Radeon X700 (PCIE)] (Secondary)
 		148c 2117  PowerColor Bravo X700
 	5f57  R423 [Radeon X800XT (PCIE)]
+	6707  Cayman LE GL [FirePro V5900]
 	6718  Cayman XT [Radeon HD 6970]
 	6719  Cayman PRO [Radeon HD 6950]
 	671d  Antilles [AMD Radeon HD 6990]
@@ -1497,6 +1517,8 @@
 	6741  Whistler [AMD Radeon HD 6600M Series]
 	6742  Whistler LE [AMD Radeon HD 6625M Graphics]
 	6743  Whistler [Radeon E6760]
+	6749  Turks [FirePro V4900]
+	674a  Turks [AMD FirePro V3900]
 	6750  Turks [AMD Radeon HD 6570]
 	6751  Turks [Radeon HD 7600A Series]
 	6758  Turks [Radeon HD 6670]
@@ -1515,6 +1537,11 @@
 	6798  Tahiti XT [Radeon HD 7970]
 	6799  New Zealand [Radeon HD 7990]
 	679a  Tahiti PRO [Radeon HD 7950]
+	6800  WIMBLEDON XT [Radeon HD 7970M]
+	6818  PITCAIRN [Radeon HD 7800]
+	6819  PITCAIRN PRO [Radeon HD 7800 Series]
+	683d  Cape Verde [Radeon HD 7700 Series]
+	683f  Cape Verde PRO [Radeon HD 7700 Series]
 	6840  Thames XT/GL [Radeon HD 7600M Series]
 	6841  Thames [Radeon 7500M/7600M Series]
 	6843  Thames [Radeon HD 7670M]
@@ -1535,6 +1562,7 @@
 	68a8  Broadway [ATI Mobility Radeon HD 6800 Series]
 	68a9  Juniper XT [FirePro 3D V5800]
 	68b8  Juniper [Radeon HD 5700 Series]
+		106b 00cf  MacPro5,1 [Mac Pro 2.8GHz DDR3]
 	68b9  Juniper [Radeon HD 5600/5700]
 	68ba  Juniper XT [AMD Radeon HD 6000 Series]
 	68be  Juniper [Radeon HD 5700 Series]
@@ -1740,6 +1768,7 @@
 	9480  M96 [Mobility Radeon HD 4650]
 		103c 3628  ATI Mobility Radeon HD 4650 [dv6-1190en]
 	9485  RV740 Pro [Radeon HD 4770]
+	9488  RV730 XT [Mobility Radeon HD 4670]
 	9489  M96 XT [Mobility FireGL V5725]
 	9490  RV730XT [Radeon HD 4670]
 		174b e880  Radeon HD 4670 512MB DDR3
@@ -1768,7 +1797,7 @@
 	94c9  Mobility Radeon HD 2400
 		1002 94c9  Radeon HD2400
 	94cb  Radeon E2400
-	94cc  RV 610LE PCI [Radeon HD 2400]
+	94cc  RV610 LE [Radeon HD 2400 Pro PCI]
 	9501  RV670 [Radeon HD 3870]
 		174b e620  Sapphire Radeon HD 3870 PCIe 2.0
 	9504  RV670 [Mobility Radeon HD 3850]
@@ -1843,8 +1872,10 @@
 		1043 843e  M5A88-V EVO
 # Radeon HD 6250 too?
 	9802  Wrestler [Radeon HD 6310]
+		174b 1001  Sapphire PURE Fusion Mini
 	9804  Wrestler [Radeon HD 6250]
 	9806  Wrestler [Radeon HD 6320]
+	9807  Wrestler [Radeon HD 6290]
 	aa00  R600 Audio Device [Radeon HD 2900 Series]
 	aa08  RV630 audio device [Radeon HD 2600 Series]
 	aa10  RV610 HDMI Audio [Radeon HD 2350/2400 Series]
@@ -2272,7 +2303,7 @@
 		1014 033b  PCIe2 6Gb SAS RAID Adapter Quad-port (57B4)
 		1014 0355  PCIe2 3.6GB Cache 6Gb SAS RAID Adapter Quad-port (57B1)
 		1014 0357  PCIe2 6Gb SAS Adapter Quad-port (57C6)
-		1014 035d  PCIe2 3.6GB Cache 6Gb SAS RAID & SSD Adapter (575D)
+		1014 035d  PCIe3 1.8GB Cache RAID SAS Adapter Quad-port 6GB (57C8)
 		1014 035e  PCIe2 3.6GB Cache 6Gb SAS RAID Adapter Quad-port (57CE)
 	3022  QLA3022 Network Adapter
 	4022  QLA3022 Network Adapter
@@ -2376,7 +2407,9 @@
 	1418  Family 15h (Models 10h-1fh) Processor Root Port
 	1419  Family 15h (Models 10h-1fh) I/O Memory Management Unit
 	1510  Family 14h Processor Root Complex
+		174b 1001  Sapphire PURE Fusion Mini
 	1512  Family 14h Processor Root Port
+		174b 1001  Sapphire PURE Fusion Mini
 	1513  Family 14h Processor Root Port
 	1514  Family 14h Processor Root Port
 	1515  Family 14h Processor Root Port
@@ -2700,7 +2733,7 @@
 	00cf  AIC-7899P U160/m
 		1028 0106  PowerEdge 4600
 		1028 0121  PowerEdge 2650
-102b  Matrox Graphics, Inc.
+102b  Matrox Electronics Systems Ltd.
 # DJ: I've a suspicion that 0010 is a duplicate of 0d10.
 	0010  MGA-I [Impression?]
 	0100  MGA 1064SG [Mystique]
@@ -2858,6 +2891,7 @@
 		1028 028d  PowerEdge T410 MGA G200eW WPCM450
 		1028 029c  PowerEdge M710 MGA G200eW WPCM450
 		1028 02a4  PowerEdge T310 MGA G200eW WPCM450
+		15d9 a811  H8DGU
 	0533  MGA G200EH
 		103c 3381  iLO4
 	0534  G200eR2
@@ -2941,10 +2975,46 @@
 		102b 0101  Millenium P690 PCI
 		102b 0140  Millenium P690 LP PCIe x1
 		102b 0180  Display Wall IP Decode 128 MB
+	4164  Morphis QxT frame grabber
+	43b4  Morphis Qxt encoding engine
+	4510  Morphis COM port
 	4536  VIA Framegrabber
-	4cdc  Morphis Vision System Jpeg2000
-	4fc5  Morphis Vision System
-	5e10  Morphis Vision System Aux/IO
+	4686  Concord GX (customized Intel 82541)
+	475b  Solios eCL/XCL-B frame grabber
+	475d  Vio frame grabber family
+		102b 4b90  Vio Duo frame grabber (single channel)
+		102b 4b91  Vio Duo frame grabber
+		102b 4b92  Vio Analog frame grabber
+		102b 4b93  Vio SDI Frame Grabber
+		102b 4b94  Vio DVI-A frame grabber
+	475f  Solios (single-Full) CL frame grabber
+		102b 475f  Solios eCL/XCL-F frame grabber
+		102b 4d5f  Solios eV-CL (single-Full) frame grabber
+		102b 4e5f  Solios eM-CL (single-Full) frame grabber
+	47a1  Solios eA/XA frame grabber
+		102b 4be0  Solios eA/XA (single) frame grabber
+		102b 4be1  Solios eA/XA (dual) frame grabber
+		102b 4be2  Solios eA/XA (quad) frame grabber
+	47a2  Solios COM port
+	47c1  Solios (dual-Base/single-Medium) CL frame grabber
+		102b 0000  Solios frame grabber
+		102b 4b80  Solios eCL/XCL (single-Medium) frame grabber
+		102b 4b81  Solios eCL/XCL (dual-Base) frame grabber
+		102b 4d80  Solios eV-CL (single-Medium) frame grabber
+		102b 4d81  Solios eV-CL (dual-Base) frame grabber
+		102b 4e80  Solios eM-CL (single-Medium) frame grabber
+		102b 4e81  Solios eM-CL (dual-Base) frame grabber
+	47c2  Solios COM port
+	4949  Radient frame grabber family
+		102b 0010  Radient eCL (Single-full) frame grabber
+		102b 0020  Radient eCL (Dual-base) frame grabber
+		102b 0030  Radient eCL (Dual-full) frame grabber
+		102b 0040  Radient eCL (Quad-base) frame grabber
+		102b 0050  Radient eCL (Golden) frame grabber
+	4cdc  Morphis JPEG2000 accelerator
+	4f54  Morphis (e)Quad frame grabber
+	4fc5  Morphis (e)Dual frame grabber
+	5e10  Morphis aux I/O
 	6573  Shark 10/100 Multiport SwitchNIC
 102c  Chips and Technologies
 	00b8  F64310
@@ -3033,7 +3103,7 @@
 	003e  NAPCCARD Cardbus Controller
 	0046  PowerVR PCX2 [midas]
 	005a  Vrc5074 [Nile 4]
-	0063  Firewarden
+	0063  uPD72862 [Firewarden] IEEE1394 OHCI 1.0 Link Controller
 	0067  PowerVR Neon 250 Chipset
 		1010 0020  PowerVR Neon 250 AGP 32Mb
 		1010 0080  PowerVR Neon 250 AGP 16Mb
@@ -3049,17 +3119,17 @@
 	009b  Vrc5476
 	00a5  VRC4173
 	00a6  VRC5477 AC97
-	00cd  IEEE 1394 [OrangeLink] Host Controller
+	00cd  uPD72870 [Firewarden] IEEE1394a OHCI 1.0 Link/3-port PHY Controller
 		12ee 8011  Root hub
-	00ce  IEEE 1394 Host Controller
+	00ce  uPD72871 [Firewarden] IEEE1394a OHCI 1.0 Link/1-port PHY Controller
 	00df  Vr4131
 	00e0  USB 2.0
 		12ee 7001  Root hub
 		14c2 0205  PTI-205N USB 2.0 Host Controller
 		1799 0002  Root Hub
 		807d 1043  PCI-USB2 (EHCI subsystem)
-	00e7  uPD72873 IEEE1394 OHCI 1.1 2-port Host Controller
-	00f2  uPD72874 IEEE1394 OHCI 1.1 3-port PHY-Link Ctrlr
+	00e7  uPD72873 [Firewarden] IEEE1394a OHCI 1.1 Link/2-port PHY Controller
+	00f2  uPD72874 [Firewarden] IEEE1394a OHCI 1.1 Link/3-port PHY Controller
 	00f3  uPD6113x Multimedia Decoder/Processor [EMMA2]
 	010c  VR7701
 	0125  uPD720400 PCI Express - PCI/PCI-X Bridge
@@ -3068,6 +3138,8 @@
 		1028 04b2  Vostro 3350
 		1043 8413  P8P67 Deluxe Motherboard
 		1b96 0001  USB 3.0 PCIe Card
+	01e7  uPD72873 [Firewarden] IEEE1394a OHCI 1.1 Link/2-port PHY Controller
+	01f2  uPD72874 [Firewarden] IEEE1394a OHCI 1.1 Link/3-port PHY Controller
 1034  Framatome Connectors USA Inc.
 1035  Comp. & Comm. Research Lab
 1036  Future Domain Corp.
@@ -3318,6 +3390,7 @@
 	1303  RMP-3 (Remote Management Processor)
 	1361  BCM4312 802.11a/b/g WLAN Controller
 	1371  Broadcom Corporation BCM4312 802.11a/b/g (rev 02)
+	1717  Intel 82571EB dual 1 Gb Ethernet controller
 	2910  E2910A PCIBus Exerciser
 	2925  E2925A 32 Bit, 33 MHzPCI Exerciser & Analyzer
 	3080  Pavilion ze2028ea
@@ -3569,21 +3642,28 @@
 		1102 100f  Graphics Blaster Extreme
 		3d3d 0100  Reference Permedia 2 3D
 	8000  PCILynx/PCILynx2 IEEE 1394 Link Layer Controller
+		105e 8003  FireBoard200
 		1443 8003  FireBoard200
+		1443 8005  FireBoard400
+		1443 8006  FireBoard400
 		e4bf 1010  CF1-1-SNARE
 		e4bf 1020  CF1-2-SNARE
+		e4bf 1040  FireCompact400
 	8009  TSB12LV22 IEEE-1394 Controller
 		104d 8032  8032 OHCI i.LINK (IEEE 1394) Controller
+		1443 8010  FireBoard400-OHCI
 	8017  PCI4410 FireWire Controller
 	8019  TSB12LV23 IEEE-1394 Controller
 		11bd 000a  Studio DV500-1394
 		11bd 000e  Studio DV
+		1443 8010  FireBoard400-OHCI
 		e4bf 1010  CF2-1-CYMBAL
 	8020  TSB12LV26 IEEE-1394 Controller (Link)
 		1028 00d8  Precision 530
 		104d 80e2  VAIO PCV-J200
 		11bd 000f  Studio DV500-1394
 		11bd 001c  Excalibur 4.1
+		1443 8010  FireBoard400-OHCI
 	8021  TSB43AA22 IEEE-1394 Controller (PHY/Link Integrated)
 		104d 80df  Vaio PCG-FX403
 		104d 80e7  VAIO PCG-GR214EP/GR214MP/GR215MP/GR314MP/GR315MP
@@ -3594,12 +3674,17 @@
 		1043 808b  K8N4-E Mainboard
 		1043 815b  P5W DH Deluxe Motherboard
 		1443 8023  FireCard400
+		8086 5044  Desktop Board DP35DP
 	8024  TSB43AB23 IEEE-1394a-2000 Controller (PHY/Link)
 		107d 6620  Winfast DV2000 FireWire Controller
 		1443 8024  FireBoard Blue
 		1458 1000  GA-EP45-DS5 Motherboard
 	8025  TSB82AA2 IEEE-1394b Link Layer Controller
+		1043 813c  P5P series mainboard
+		1443 8025  FireBoard800
 		1458 1000  GA-K8N Ultra-9 Mainboard
+		1546 8025  FWB-PCI01
+		17fc 8025  GIC3800
 	8026  TSB43AB21 IEEE-1394a-2000 Controller (PHY/Link)
 		1025 0035  TravelMate 660
 		1025 003c  Aspire 2001WLCi (Compaq CL50 motherboard)
@@ -4143,7 +4228,7 @@
 	ba57  eXtremeRAID 4000/5000 support Device
 		1069 0072  eXtremeRAID 5000 support Device
 106a  Aten Research Inc
-106b  Apple Computer Inc.
+106b  Apple Inc.
 	0001  Bandit PowerPC host bridge
 	0002  Grand Central I/O
 	0003  Control Video
@@ -4213,6 +4298,7 @@
 	0074  U4 HT Bridge
 	1645  Tigon3 Gigabit Ethernet NIC (BCM5701)
 106c  Hynix Semiconductor
+	8139  8139c 100BaseTX Ethernet Controller
 	8801  Dual Pentium ISA/PCI Motherboard
 	8802  PowerPC ISA/PCI Motherboard
 	8803  Dual Window Graphics Accelerator
@@ -4282,6 +4368,7 @@
 		103c 12dd  4Gb Fibre Channel [AB429A]
 	2432  ISP2432-based 4Gb Fibre Channel to PCI Express HBA
 	2532  ISP2532-based 8Gb Fibre Channel to PCI Express HBA
+		1077 0167  QME2572 Dual Port FC8 HBA Mezzanine
 	3022  ISP4022-based Ethernet NIC
 	3032  ISP4032-based Ethernet IPv6 NIC
 	4010  ISP4010-based iSCSI TOE HBA
@@ -4329,9 +4416,7 @@
 107c  LG Electronics [Lucky Goldstar Co. Ltd]
 107d  LeadTek Research Inc.
 	0000  P86C850
-	204d  [GeForce 7800 GTX] Winfast PX7800 GTX TDH
 	2134  WinFast 3D S320 II
-	2971  [GeForce FX 5900] WinFast A350 TDH MyViVo
 	6609  Winfast TV 2000 XP RM
 	6654  Conexant CX23883 [WinFast DTV1800 H]
 	6f22  WinFast PxTV1200
@@ -4397,8 +4482,7 @@
 		108d 0017  OC-2250 GoCard 10/100 Ethernet Adapter
 	0021  OC-6151/6152 [RapidFire ATM 155]
 	0022  ATM Adapter
-# formerly Sun Microsystems
-108e  Oracle Corporation
+108e  Oracle/SUN
 	0001  EBUS
 	1000  EBUS
 	1001  Happy Meal 10/100 Ethernet [hme]
@@ -4892,6 +4976,9 @@
 	8664  PEX 8664 64-lane, 16-Port PCI Express Gen 2 (5.0 GT/s) Switch
 	8680  PEX 8680 80-lane, 20-Port PCI Express Gen 2 (5.0 GT/s) Multi-Root Switch
 	8696  PEX 8696 96-lane, 24-Port PCI Express Gen 2 (5.0 GT/s) Multi-Root Switch
+	8732  PEX 8732 32-lane, 8-Port PCI Express Gen 3 (8.0 GT/s) Switch
+# This is the Non-Transparent-Bridge Virtualized Port as presented by the PLX PEX 8732 chip, the physical bridges show up at 10b5:8732
+	87b0  PEX 8732 32-lane, 8-Port PCI Express Gen 3 (8.0 GT/s) Switch
 	9016  PLX 9016 8-port serial controller
 	9030  PCI9030 32-bit 33MHz PCI <-> IOBus Bridge
 		10b5 2695  Hilscher CIF50-PB Profibus Master Board
@@ -5453,7 +5540,7 @@
 10dd  Evans & Sutherland
 	0100  Lightning 1200
 		10dd 0023  Lightning 1200 15+16M
-10de  nVidia Corporation
+10de  NVIDIA Corporation
 	0008  NV1 [EDGE 3D]
 	0009  NV1 [EDGE 3D]
 	0010  NV2 [Mutara V08]
@@ -5574,6 +5661,7 @@
 		10f1 2865  Tomcat K8E (S2865)
 		1458 0c11  GA-K8N Ultra-9 Mainboard
 		1462 7100  MSI K8N Diamond
+		1462 7125  K8N Neo4-F mainboard
 		147b 1c1a  KN8-Ultra Mainboard
 		1565 3402  NF4 AM2L Mainboard
 	0051  CK804 ISA Bridge
@@ -5584,6 +5672,7 @@
 		10f1 2865  Tomcat K8E (S2865)
 		1458 0c11  GA-K8N Ultra-9 Mainboard
 		1462 7100  MSI K8N Diamond
+		1462 7125  K8N Neo4-F mainboard
 		147b 1c1a  KN8-Ultra Mainboard
 		1565 3402  NF4 AM2L Mainboard
 	0053  CK804 IDE
@@ -5591,6 +5680,7 @@
 		10f1 2865  Tomcat K8E (S2865)
 		1458 5002  GA-K8N Ultra-9 Mainboard
 		1462 7100  MSI K8N Diamond
+		1462 7125  K8N Neo4-F mainboard
 		147b 1c1a  KN8-Ultra Mainboard
 		1565 3402  NF4 AM2L Mainboard
 	0054  CK804 Serial ATA Controller
@@ -5599,6 +5689,7 @@
 		10f1 2865  Tomcat K8E (S2865)
 		1458 b003  GA-K8N Ultra-9 Mainboard
 		1462 7100  MSI K8N Diamond
+		1462 7125  K8N Neo4-F mainboard
 		147b 1c1a  KN8-Ultra Mainboard
 		1565 5401  NF4 AM2L Mainboard
 	0055  CK804 Serial ATA Controller
@@ -5606,6 +5697,7 @@
 		1043 815a  K8N4-E or A8N-E Mainboard
 		10f1 2865  Tomcat K8E (S2865)
 		1458 b003  GA-K8N Ultra-9 Mainboard
+		1462 7125  K8N Neo4-F mainboard
 		147b 1c1a  KN8-Ultra Mainboard
 		1565 5401  NF4 AM2L Mainboard
 	0056  CK804 Ethernet Controller
@@ -5615,12 +5707,14 @@
 		10f1 2865  Tomcat K8E (S2865)
 		1458 e000  GA-K8N Ultra-9 Mainboard
 		1462 7100  MSI K8N Diamond
+		1462 7125  K8N Neo4-F mainboard
 		147b 1c1a  KN8-Ultra Mainboard
 		1565 2501  NF4 AM2L Mainboard
 	0058  CK804 AC'97 Modem
 	0059  CK804 AC'97 Audio Controller
 		1043 812a  K8N4-E or A8N-E Mainboard
 		10f1 2865  Tomcat K8E (S2865)
+		1462 7585  K8N Neo4-F mainboard
 		147b 1c1a  KN8-Ultra Mainboard
 		1565 8211  NF4 AM2L Mainboard
 	005a  CK804 USB Controller
@@ -5629,6 +5723,7 @@
 		10f1 2865  Tomcat K8E (S2865)
 		1458 5004  GA-K8N Ultra-9 Mainboard
 		1462 7100  MSI K8N Diamond
+		1462 7125  K8N Neo4-F mainboard
 		147b 1c1a  KN8-Ultra Mainboard
 		1565 3402  NF4 AM2L Mainboard
 	005b  CK804 USB Controller
@@ -5637,6 +5732,7 @@
 		10f1 2865  Tomcat K8E (S2865)
 		1458 5004  GA-K8N Ultra-9 Mainboard
 		1462 7100  MSI K8N Diamond
+		1462 7125  K8N Neo4-F mainboard
 		147b 1c1a  KN8-Ultra Mainboard
 		1565 3402  NF4 AM2L Mainboard
 	005c  CK804 PCI Bridge
@@ -5648,7 +5744,8 @@
 		10f1 2865  Tomcat K8E (S2865)
 		10f1 2891  Thunder K8SRE Mainboard
 		1458 5000  GA-K8N Ultra-9 Mainboard
-		1462 7100  MSI K8N Diamond
+		1462 7100  K8N Diamond Mainboard
+		1462 7125  K8N Neo4-F Mainboard
 		147b 1c1a  KN8-Ultra Mainboard
 		1565 3402  NF4 AM2L Mainboard
 	005f  CK804 Memory Controller
@@ -6009,6 +6106,7 @@
 	0215  NV48 [GeForce 6800 GT]
 	0218  NV48 [GeForce 6800 XT]
 	0221  NV44A [GeForce 6200]
+		1043 81e1  N6200/TD/256M/A
 		3842 a341  256A8N341DX
 	0222  NV44 [GeForce 6200 A-LE]
 	0240  C51PV [GeForce 6150]
@@ -6273,7 +6371,7 @@
 	0361  MCP55 LPC Bridge
 		1028 0221  PowerEdge R805 MCP55 LPC Bridge
 	0362  MCP55 LPC Bridge
-		147b 12c4  KN9-Ultra Mainboard
+		147b 1c24  KN9 series mainboard
 	0363  MCP55 LPC Bridge
 	0364  MCP55 LPC Bridge
 		1028 0221  PowerEdge R805 MCP55 LPC Bridge
@@ -6283,26 +6381,27 @@
 	0368  MCP55 SMBus
 		1028 020c  PowerEdge M605 MCP55 SMBus
 		1028 0221  PowerEdge R805 MCP55 SMBus
-		147b 12c4  KN9-Ultra Mainboard
+		147b 1c24  KN9 series mainboard
 	0369  MCP55 Memory Controller
-		147b 12c4  KN9-Ultra Mainboard
+		147b 1c24  KN9 series mainboard
 	036a  MCP55 Memory Controller
 	036b  MCP55 SMU
 	036c  MCP55 USB Controller
 		1028 020c  PowerEdge M605 MCP55 USB Controller
 		1028 0221  PowerEdge R805 MCP55 USB Controller
-		147b 12c4  KN9-Ultra Mainboard
+		147b 1c24  KN9 series mainboard
 	036d  MCP55 USB Controller
 		1028 020c  PowerEdge M605 MCP55 USB Controller
 		1028 0221  PowerEdge R805 MCP55 USB Controller
-		147b 12c4  KN9-Ultra Mainboard
+		147b 1c24  KN9 series mainboard
 	036e  MCP55 IDE
-		147b 12c4  KN9-Ultra Mainboard
+		147b 1c24  KN9 series mainboard
 	0370  MCP55 PCI bridge
 	0371  MCP55 High Definition Audio
+		147b 1c24  KN9 series mainboard
 	0372  MCP55 Ethernet
 	0373  MCP55 Ethernet
-		147b 12c4  KN9-Ultra Mainboard
+		147b 1c24  KN9 series mainboard
 	0374  MCP55 PCI Express bridge
 	0375  MCP55 PCI Express bridge
 	0376  MCP55 PCI Express bridge
@@ -6312,7 +6411,7 @@
 	037e  MCP55 SATA Controller
 	037f  MCP55 SATA Controller
 		1028 0221  PowerEdge R805 MCP55 SATA Controller
-		147b 12c4  KN9-Ultra Mainboard
+		147b 1c24  KN9 series mainboard
 	038b  G73 [GeForce 7650 GS]
 	0390  G73 [GeForce 7650 GS]
 	0391  G73 [GeForce 7600 GT]
@@ -6430,8 +6529,10 @@
 		1849 03f6  939NF6G-VSTA Board
 	03f7  MCP61 SATA Controller
 	0400  G84 [GeForce 8600 GTS]
+		1043 8241  EN8600GTS
 	0401  G84 [GeForce 8600GT]
 	0402  G84 [GeForce 8600 GT]
+		1458 3455  GV-NX86T512H
 		1462 0910  NX8600GT-T2D256EZ
 	0403  G84 [GeForce 8600 GS]
 	0404  G84 [GeForce 8400 GS]
@@ -6868,7 +6969,8 @@
 		19da a123  IONITX-F-E
 	087e  ION LE VGA
 	087f  ION LE VGA
-	08a4  GT216 [GeForce 320M]
+	08a3  MCP89 [GeForce 320M]
+	08a4  MCP89 [GeForce 320M]
 	0a20  GT216 [GeForce GT 220]
 		1043 8311  ENGT220/DI/1GD3(LP)/V2
 	0a23  GT218 [GeForce 210]
@@ -7033,15 +7135,19 @@
 	0df2  GF108 [GeForce GT 435M]
 	0df4  GF108 [GeForce GT 540M]
 	0df5  GF108 [GeForce GT 540M]
+	0df7  GF108 [GeForce GT 520M]
 	0df8  GF108 [Quadro 600]
+	0df9  GF108 [Quadro 500M]
 	0dfa  GF108 [Quadro 1000M]
-	0e08  HDMI Audio stub
+	0e08  GF119 HDMI Audio Controller
 	0e09  GF110 High Definition Audio Controller
+	0e0a  GK104 HDMI Audio Controller
 	0e0c  GF110 High Definition Audio Controller
 	0e22  GF104 [GeForce GTX 460]
 		1462 2322  N460GTX Cyclone 1GD5/OC
 	0e3a  GF104 [Quadro 3000M]
 	0e3b  GF104 [Quadro 4000M]
+	0fd2  GK107 [GeForce GT 640M]
 	1040  GF119 [GeForce GT 520]
 	1050  GF119 [GeForce GT 520M]
 	1051  GF119 [GeForce GT 520MX]
@@ -7050,15 +7156,20 @@
 	1080  GF110 [GeForce GTX 580]
 	1081  GF110 [GeForce GTX 570]
 		10de 087e  Leadtek WinFast GTX 570
-	1082  GF114 [GeForce GTX 560 Ti]
+	1082  GF110 [GeForce GTX 560 Ti]
 	1086  GF110 [GeForce GTX 570 HD]
+	1087  GF110 [GeForce GTX 560 Ti 448 Cores]
+	1091  Tesla M2090
 	1094  Tesla M2075 Dual-Slot Computing Processor Module
+	1096  Tesla C2075
 	10c3  GT218 [GeForce 8400 GS]
 		10de 066d  G98 [GeForce 8400GS]
-	10c5  Dell M116N [GeForce 405]
-	10d8  GT218 [NVS 300]
-	1200  GF110 [GeForce GTX 560 Ti]
+	10c5  GT218 [GeForce 405]
+	10d8  GT218 [Quadro NVS 300]
+	1180  GK104 [GeForce GTX 680]
+	1200  GF114 [GeForce GTX 560 Ti]
 	1201  GF114 [GeForce GTX 560]
+	1205  GF114 [GeForce GTX 460 v2]
 	1244  GF116 [GeForce GTX 550 Ti]
 	1245  GF116 [GeForce GTS 450]
 	1247  GF106 [GeForce GT 555M]
@@ -7208,8 +7319,9 @@
 	0885  Realtek 885 High Definition Audio
 	0888  Realtek 888 High Definition Audio
 		1028 020d  Inspiron 530
-	5209  RTS5116 PCI Express Card Reader
-	5288  PCI Express Card Reader
+	5209  RTS5209 PCI Express Card Reader
+	5229  RTS5229 PCI Express Card Reader
+	5288  Barossa PCI Express Card Reader
 	8029  RTL-8029(AS)
 		10b8 2011  EZ-Card (SMC1208)
 		10ec 8029  RTL-8029(AS)
@@ -7221,6 +7333,7 @@
 		10ec 8129  RT8129 Fast Ethernet Adapter
 		11ec 8129  RT8129 Fast Ethernet Adapter
 	8136  RTL8101E/RTL8102E PCI Express Fast Ethernet controller
+		103c 2ab1  Pavillion p6774
 		103c 30cc  Pavilion dv6700
 		1179 ff64  RTL8102E PCI-E Fast Ethernet NIC
 	8138  RT8139 (B/C) Cardbus Fast Ethernet Adapter
@@ -7288,13 +7401,13 @@
 		1043 83a3  M4A785TD Motherboard
 		1043 8432  P8P67 and other motherboards
 		10ec 8168  TEG-ECTX Gigabit PCI-E Adapter [Trendnet]
-		1458 e000  GA-EP45-DS5 Motherboard
+		1458 e000  GA-EP45-DS5/GA-EG45M-DS2H Motherboard
 		1462 238c  Onboard RTL8111b on MSI P965 Platinum Mainboard
 		1462 368c  K9AG Neo2
 		1462 7522  X58 Pro-E
 		1775 11cc  CC11/CL11
 		1849 8168  Motherboard (one of many)
-		8086 d615  DeskTop Board D510MO
+		8086 d615  Desktop Board D510MO
 	8169  RTL-8169 Gigabit Ethernet
 		1025 0079  Aspire 5024WLMi
 		10bd 3202  EP-320G-TX1 32-bit PCI Gigabit Ethernet Adapter
@@ -7322,6 +7435,7 @@
 		1385 4700  MA521 802.11b Wireless PC Card
 		1737 0019  WPC11v4 802.11b Wireless-B Notebook Adapter
 	8185  RTL-8185 IEEE 802.11a/b/g Wireless LAN Controller
+	8190  RTL8190 802.11n Wireless LAN
 	8191  RTL8188CE 802.11b/g/n WiFi Adapter
 	8192  RTL8192E/RTL8192SE Wireless LAN Controller
 	8197  SmartLAN56 56K Modem
@@ -7332,12 +7446,14 @@
 10ee  Xilinx Corporation
 	0001  EUROCOM for PCI (ECOMP)
 	0002  Octal E1/T1 for PCI ETP Card
-	0007  Zomojo Z1
+	0007  Default PCIe endpoint ID
 	0205  Wildcard TE205P
 	0210  Wildcard TE210P
+	0300  Spartan 3 Designs (Xilinx IP)
 	0314  Wildcard TE405P/TE410P (1st Gen)
 	0405  Wildcard TE405P (2nd Gen)
 	0410  Wildcard TE410P (2nd Gen)
+	0600  Xilinx 6 Designs (Xilinx IP)
 	2b00  Zomojo Zcard
 	3fc0  RME Digi96
 	3fc1  RME Digi96/8
@@ -7421,6 +7537,7 @@
 		1102 1009  SB Audigy2 OEM HP
 		1102 2002  SB Audigy 2 ZS (SB0350)
 		1102 4001  E-MU 1010
+		1102 4002  E-MU 0404
 	0005  SB X-Fi
 		1102 0021  X-Fi Platinum
 		1102 002c  X-Fi XtremeGamer FATAL1TY PRO
@@ -7484,6 +7601,9 @@
 	0007  HPT371/371N
 	0008  HPT374
 	0009  HPT372N
+	0620  RocketRAID 620 2 Port SATA-III Controller
+	0622  RocketRAID 622 2 Port SATA-III Controller
+	0640  RocketRAID 640 4 Port SATA-III Controller
 	1720  RocketRAID 1720 (2x SATA II RAID Controller)
 	1740  RocketRAID 1740
 	1742  RocketRAID 1742
@@ -7495,6 +7615,13 @@
 	2340  RocketRAID 2340 16 Port SATA-II Controller
 	2640  RocketRAID 2640 SAS/SATA Controller
 	2722  RocketRAID 2722
+# SFF-8087 Mini-SAS 16 port internal
+	2740  RocketRAID 2740
+# SFF-8088 Mini-SAS 16 port external
+	2744  RocketRaid 2744
+# SFF-8088 8 port external / SFF-8087 24 port internal
+	2782  RocketRAID 2782
+	3120  RocketRAID 3120
 	3220  RocketRAID 3220
 	3320  RocketRAID 3320
 1104  RasterOps Corp.
@@ -7549,6 +7676,7 @@
 		1043 836c  P7H55
 		1043 83c7  P5KPL-AM EPU
 	0409  VX855/VX875 Host Bridge: Host Control
+	0410  VX900 Host Bridge: Host Control
 	0415  VT6415 PATA IDE Host Controller
 		1043 838f  M5A88-V EVO
 	0501  VT8501 [Apollo MVP4]
@@ -7634,6 +7762,7 @@
 	1353  VX800/VX820 Error Reporting
 	1364  CN896/VN896/P4M900 Host Bridge
 	1409  VX855/VX875 Error Reporting
+	1410  VX900 Error Reporting
 	1571  VT82C576M/VT82C586
 	1595  VT82C595/97 [Apollo VP2/97]
 	1732  VT1732 [Envy24 II] PCI Multi-Channel Audio Controller
@@ -7658,6 +7787,7 @@
 	2353  VX800/VX820 Host Bus Control
 	2364  CN896/VN896/P4M900 Host Bridge
 	2409  VX855/VX875 Host Bus Control
+	2410  VX900 CPU Bus Controller
 	287a  VT8251 PCI to PCI Bridge
 	287b  VT8251 Host Bridge
 	287c  VT8251 PCIE Root Port
@@ -7877,6 +8007,7 @@
 	3282  K8T800Pro Host Bridge
 	3287  VT8251 PCI to ISA Bridge
 	3288  VT8237A/VT8251 HDA Controller
+		19da a179  ZBOX VD01
 	3290  K8M890 Host Bridge
 	3296  P4M800 Host Bridge
 	3324  CX700/VX700 Host Bridge
@@ -7898,6 +8029,7 @@
 		1043 8374  M5A88-V EVO
 		1043 8384  P8P67 Deluxe Motherboard
 	3409  VX855/VX875 DRAM Bus Control
+	3410  VX900 DRAM Bus Control
 	4149  VIA VT6420 (ATA133) Controller
 	4204  K8M800 Host Bridge
 	4208  PT890 Host Bridge
@@ -7920,6 +8052,7 @@
 	4364  CN896/VN896/P4M900 Host Bridge
 	4397  VT1708B/1702S/1708S HD audio codec
 	4409  VX855/VX875 Power Management Control
+	4410  VX900 Power Management and Chip Testing Control
 	4428  VT1718S HD Audio Codec
 	5030  VT82C596 ACPI [Apollo PRO]
 	5122  VX855/VX875 Chrome 9 HCM Integrated Graphics
@@ -7937,12 +8070,14 @@
 	5364  CN896/VN896/P4M900 I/O APIC Interrupt Controller
 	5372  VT8237/8251 Serial ATA Controller
 	5409  VX855/VX875 APIC and Central Traffic Control
+	5410  VX900 APIC and Central Traffic Control
 	6100  VT85C100A [Rhine II]
 	6287  SATA RAID Controller
 	6327  P4M890 Security Device
 	6353  VX800/VX820 Scratch Registers
 	6364  CN896/VN896/P4M900 Security Device
 	6409  VX855/VX875 Scratch Registers
+	6410  VX900 Scratch Registers
 	7122  VX900 Graphics [Chrome9 HD]
 	7204  K8M800 Host Bridge
 	7205  KM400/KN400/P4M800 [S3 UniChrome]
@@ -7967,6 +8102,7 @@
 	7353  VX800/VX820 North-South Module Interface Control
 	7364  CN896/VN896/P4M900 Host Bridge
 	7409  VX855/VX875 North-South Module Interface Control
+	7410  VX900 North-South Module Interface Control
 	8231  VT8231 [PCI-to-ISA Bridge]
 	8235  VT8235 ACPI
 	8305  VT8363/8365 [KT133/KM133 AGP]
@@ -7975,6 +8111,8 @@
 	8391  VT8371 [KX133 AGP]
 	8400  MVP4
 	8409  VX855/VX875 Bus Control and Power Management
+	8410  VX900 Bus Control and Power Management
+		19da a179  ZBOX VD01
 	8500  KLE133/PLE133/PLE133T
 	8501  VT8501 [Apollo MVP4 AGP]
 	8596  VT82C596 [Apollo PRO AGP]
@@ -7989,6 +8127,7 @@
 	8a26  KL133/KL133A/KM133/KM133A [S3 ProSavage]
 	8d01  PN133/PN133T [S3 Twister]
 	8d04  KM266/P4M266/P4M266A/P4N266 [S3 ProSavageDDR]
+	9001  VX900 Serial ATA Controller
 	9530  Secure Digital Memory Card Controller
 	95d0  SDIO Host Controller
 	a208  PT890 PCI to PCI Bridge Controller
@@ -8010,7 +8149,7 @@
 		147b 1407  KV8-MAX3 motherboard
 	b198  VT8237/VX700 PCI Bridge
 	b213  VPX/VPX2 I/O APIC Interrupt Controller
-	b353  VX855/VX875 PCI to PCI Bridge
+	b353  VX855/VX875/VX900 PCI to PCI Bridge
 	b999  [K8T890 North / VT8237 South] PCI Bridge
 	c208  PT890 PCI to PCI Bridge Controller
 	c238  K8T890 PCI to PCI Bridge Controller
@@ -8602,6 +8741,8 @@
 1134  Mercury Computer Systems
 	0001  Raceway Bridge
 	0002  Dual PCI to RapidIO Bridge
+	000b  POET Serial RapidIO Bridge
+	000d  POET PSDMS Device
 1135  Fuji Xerox Co Ltd
 	0001  Printer controller
 1136  Momentum Data Systems
@@ -9132,8 +9273,9 @@
 	e230  Memory Stick Host Controller
 	e822  MMC/SD Host Controller
 		1028 040b  Latitude E6510
-	e832  FireWire Host Controller
+	e832  R5C832 PCIe IEEE 1394 Controller
 		1028 040b  Latitude E6510
+	e852  PCIe xD-Picture Card Controller
 1181  Telmatics International
 1183  Fujikura Ltd
 1184  Forks Inc
@@ -9286,6 +9428,7 @@
 	2a01  88W8335 [Libertas] 802.11b/g Wireless
 	2a02  88W8361 [TopDog] 802.11n Wireless
 		07d1 3b02  DIR-615 rev. A1 Mini PCI Wireless Module
+		1385 7c01  WN511T RangeMax Next 300 Mbps Wireless Notebook Adapter
 		1385 7e00  WN311T RangeMax Next 300 Mbps Wireless PCI Adapter
 		1799 801b  F5D8011 v2 802.11n N1 Wireless Notebook Card
 	2a08  88W8362e [TopDog] 802.11a/b/g/n Wireless
@@ -9433,6 +9576,7 @@
 	4368  88EC034 Ethernet Controller
 	4369  88EC042 Ethernet Controller
 	436a  88E8058 PCI-E Gigabit Ethernet Controller
+		11ab 00ba  Imac 8,1 Wired Ethernet Adapter
 	436b  88E8071 PCI-E Gigabit Ethernet Controller
 	436c  88E8072 PCI-E Gigabit Ethernet Controller
 	436d  88E8055 PCI-E Gigabit Ethernet Controller
@@ -9456,7 +9600,7 @@
 	6081  MV88SX6081 8-port SATA II PCI-X Controller
 	6101  88SE6101/6102 single-port PATA133 interface
 	6111  88SE6111 1-port PATA133(IDE) and 1-port SATA II Controllers
-	6121  88SE6121 SATA II Controller
+	6121  88SE6121 SATA II / PATA Controller
 	6141  88SE614x SATA II PCI-E controller
 	6145  88SE6145 SATA II PCI-E controller
 	6180  88F6180 [Kirkwood] ARM SoC
@@ -9527,8 +9671,8 @@
 11be  International Microcircuits Inc
 11bf  Astrodesign, Inc.
 11c0  Hewlett Packard
-# Nee Lucent Microelectronics
-11c1  Agere Systems
+# nee Agere Systems nee Lucent Microelectronics
+11c1  LSI Corporation
 	0440  56k WinModem
 		1033 8015  LT WinModem 56k Data+Fax+Voice+Dsvd
 		1033 8047  LT WinModem 56k Data+Fax+Voice+Dsvd
@@ -9644,14 +9788,14 @@
 	5801  USB
 	5802  USS-312 USB Controller
 	5803  USS-344S USB Controller
-	5811  FW322/323
+	5811  FW322/323 [TrueFire] 1394a Controller
 		1043 8294  IEEE 1394a Firewire Controller
 		8086 524c  D865PERL mainboard
 		dead 0800  FireWire Host Bus Adapter
-	5901  FW643 PCI Express 1394b Controller (PHY/Link)
+	5901  FW643 [TrueFire] PCIe 1394b Controller
 		1443 0643  FireBoard800-e V.2
-		1546 0643  FWB-PCIE1X20
-	5903  FW533 PCI Express 1394a Controller (PHY/Link)
+		1546 0643  FWB-PCIE1X2x
+	5903  FW533 [TrueFire] PCIe 1394a Controller
 	8110  T8110 H.100/H.110 TDM switch
 		12d9 000c  E1/T1 PMXc cPCI carrier card
 	ab10  WL60010 Wireless LAN MAC
@@ -9875,6 +10019,7 @@
 1217  O2 Micro, Inc.
 	00f7  Firewire (IEEE 1394)
 		1179 ff50  Satellite P305D-S8995E
+	10f7  1394 OHCI Compliant Host Controller
 	11f7  OZ600 1394a-2000 Controller
 	13f7  1394 OHCI Compliant Host Controller
 	6729  OZ6729
@@ -9912,6 +10057,8 @@
 		103c 0890  NC6000 laptop
 		10cf 11c4  Lifebook P5020D Laptop
 	7233  OZ711MP3/MS3 4-in-1 MemoryCardBus Controller
+	8120  Integrated MMC/SD Controller
+	8130  Integrated MS/MSPRO/xD Controller
 	8321  Integrated MMC/SD controller
 	8331  O2 Flash Memory Card
 1218  Hybricon Corp.
@@ -9963,7 +10110,7 @@
 	0057  Voodoo 3/3000 [Avenger]
 121b  Advanced Telecommunications Modules
 121c  Nippon Texaco., Ltd
-121d  Lippert Automationstechnik GmbH
+121d  LiPPERT ADLINK Technology GmbH
 121e  CSPI
 	0201  Myrinet 2000 Scalable Cluster Interconnect
 121f  Arcus Technology, Inc.
@@ -10123,7 +10270,7 @@
 	1968  ES1968 Maestro 2
 		1028 0085  ES1968 Maestro-2 PCI
 		1033 8051  ES1968 Maestro-2 Audiodrive
-	1969  ES1969 Solo-1 Audiodrive
+	1969  ES1938/ES1946/ES1969 Solo-1 Audiodrive
 		1014 0166  ES1969 SOLO-1 AudioDrive on IBM Aptiva Mainboard
 		125d 8888  Solo-1 Audio Adapter
 		153b 111b  Terratec 128i PCI
@@ -10462,10 +10609,10 @@
 	8152  IT8152F/G Advanced RISC-to-PCI Companion Chip
 	8211  ITE 8211F Single Channel UDMA 133
 		1043 8138  P5GD1-VW Mainboard
-# PCI version seems to be IT8212, embedded seems to be ITE8212
-	8212  IT/ITE8212 Dual channel ATA RAID controller
+	8212  IT8212 Dual channel ATA RAID controller
 		1283 0001  IT/ITE8212 Dual channel ATA RAID controller
 	8213  IT8213 IDE Controller
+		1458 b000  GA-EG45M-DS2H Mainboard
 	8330  IT8330G
 	8872  IT8874F PCI Dual Serial Port Controller
 	8888  IT8888F/G PCI to ISA Bridge with SMB [Golden Gate]
@@ -11719,6 +11866,7 @@
 	9513  OX16PCI954 (Quad 16950 UART) function 1 (parallel port)
 	9521  OX16PCI952 (Dual 16950 UART)
 	9523  OX16PCI952 Integrated Parallel Port
+	c158  OXPCIe952 Dual 16C950 UART
 	c308  EX-44016 16-port serial
 1416  Multiwave Innovation pte Ltd
 1417  Convergenet Technologies Inc
@@ -12188,6 +12336,9 @@
 		1028 028d  PowerEdge T410 BCM5716 Gigabit Ethernet
 		1028 02f1  PowerEdge R510 BCM5716 Gigabit Ethernet
 	163c  NetXtreme II BCM5716S Gigabit Ethernet
+	163d  NetXtreme II BCM57811 10 Gigabit Ethernet Multi Function
+	163e  NetXtreme II BCM57811 10 Gigabit Ethernet Multi Function
+	163f  NetXtreme II BCM57810 10 Gigabit Ethernet Virtual Function
 	1644  NetXtreme BCM5700 Gigabit Ethernet
 		1014 0277  Broadcom Vigil B5700 1000Base-T
 		1028 00d1  Broadcom BCM5700
@@ -12369,9 +12520,16 @@
 	167f  NetLink BCM5787F Fast Ethernet PCI Express
 	1680  NetXtreme BCM5761e Gigabit Ethernet PCIe
 	1681  NetXtreme BCM5761 Gigabit Ethernet PCIe
+	1682  NetXtreme BCM57762 Gigabit Ethernet PCIe
 	1684  NetXtreme BCM5764M Gigabit Ethernet PCIe
 	1685  NetXtreme II BCM57500S Gigabit Ethernet
-	168a  NetXtreme II BCM57800 10 Gigabit Ethernet
+	1686  NetXtreme BCM57766 Gigabit Ethernet PCIe
+# The Broadcom 57800 device has two 1Gig ports and two 10Gig ports. The subsystem information can be used to differentiate.
+	168a  NetXtreme II BCM57800 1/10 Gigabit Ethernet
+		1028 1f5c  BCM57800 10-Gigabit Ethernet
+		1028 1f5d  BCM57800 10-Gigabit Ethernet
+		1028 1f67  BCM57800 1-Gigabit Ethernet
+		1028 1f68  BCM57800 1-Gigabit Ethernet
 	168d  NetXtreme II BCM57840 10/20 Gigabit Ethernet
 	168e  NetXtreme II BCM57810 10 Gigabit Ethernet
 	1690  NetXtreme BCM57760 Gigabit Ethernet PCIe
@@ -12397,7 +12555,12 @@
 		1462 590c  KT6 Delta-FIS2R (MS-6590)
 	169d  NetLink BCM5789 Gigabit Ethernet PCI Express
 	16a0  NetLink BCM5785 Fast Ethernet
-	16a5  NetXtreme II BCM57800 10 Gigabit Ethernet Multi Function
+# The Broadcom 57800 device has two 1Gig ports and two 10Gig ports. The subsystem information can be used to differentiate.
+	16a5  NetXtreme II BCM57800 1/10 Gigabit Ethernet Multi Function
+		1028 1f5c  NetXtreme II BCM57800 10-Gigabit Ethernet Multi Function
+		1028 1f5d  NetXtreme II BCM57800 10-Gigabit Ethernet Multi Function
+		1028 1f67  NetXtreme II BCM57800 1-Gigabit Ethernet Multi Function
+		1028 1f68  NetXtreme II BCM57800 1-Gigabit Ethernet Multi Function
 	16a6  NetXtreme BCM5702X Gigabit Ethernet
 		0e11 00bb  NC7760 Gigabit Server Adapter (PCI-X, 10/100/1000-T)
 		1028 0126  BCM5702 1000Base-T
@@ -12416,7 +12579,12 @@
 		10a9 8014  Dual Port Gigabit Ethernet (PCI-X,Fiber)
 		10a9 801c  Quad Port Gigabit Ethernet (PCI-E,Fiber)
 		10b7 2001  3C998-SX Dual Port 1000-SX PCI-X
-	16a9  NetXtreme II BCM57800 10 Gigabit Ethernet Virtual Function
+# The Broadcom 57800 device has two 1Gig ports and two 10Gig ports. The subsystem information can be used to differentiate.
+	16a9  NetXtreme II BCM57800 1/10 Gigabit Ethernet Virtual Function
+		1028 1f5c  NetXtreme II BCM57800 10-Gigabit Ethernet Virtual Function
+		1028 1f5d  NetXtreme II BCM57800 10-Gigabit Ethernet Virtual Function
+		1028 1f67  NetXtreme II BCM57800 1-Gigabit Ethernet Virtual Function
+		1028 1f68  NetXtreme II BCM57800 1-Gigabit Ethernet Virtual Function
 	16aa  NetXtreme II BCM5706S Gigabit Ethernet
 		103c 3102  NC370F MultifuNCtion Gigabit Server Adapter
 		103c 310c  NC370i Multifunction Gigabit Server Adapter
@@ -12435,9 +12603,11 @@
 	16b0  NetXtreme BCM57761 Gigabit Ethernet PCIe
 	16b1  NetLink BCM57781 Gigabit Ethernet PCIe
 	16b2  NetLink BCM57791 Gigabit Ethernet PCIe
+	16b3  NetXtreme BCM57786 Gigabit Ethernet PCIe
 	16b4  NetXtreme BCM57765 Gigabit Ethernet PCIe
 	16b5  NetLink BCM57785 Gigabit Ethernet PCIe
 	16b6  NetLink BCM57795 Gigabit Ethernet PCIe
+	16b7  NetXtreme BCM57782 Gigabit Ethernet PCIe
 	16bc  NetXtreme BCM57765 Memory Card Reader
 	16c6  NetXtreme BCM5702A3 Gigabit Ethernet
 		10b7 1100  3C1000B-T 10/100/1000 PCI
@@ -13747,6 +13917,7 @@
 		1a56 2001  Killer Wireless-N 1103 Half-size Mini PCIe Card [AR9380]
 	0032  AR9485 Wireless Network Adapter
 	0033  AR9580 Wireless Network Adapter
+	0034  AR9462 Wireless Network Adapter
 	0207  AR5210 Wireless Network Adapter [AR5000 802.11a]
 	1014  AR5212 802.11abg NIC
 		1014 058a  ThinkPad 11a/b/g Wireless LAN Mini Express Adapter (AR5BXB6)
@@ -13829,6 +14000,7 @@
 	5702  PMC-SLX150-1M: Reconfigurable Spartan-6 FPGA with plug-in I/O
 	5801  XMC-VLX85 Reconfigurable Virtex-5 FPGA with plug-in I/O
 	5802  XMC-VLX110 Reconfigurable Virtex-5 FPGA with plug-in I/O
+	5803  XMC-VSX95 Reconfigurable Virtex-5 FPGA with plug-in I/O
 	5804  XMC-VLX155 Reconfigurable Virtex-5 FPGA with plug-in I/O
 	5807  XMC-SLX150: Reconfigurable Spartan-6 FPGA with plug-in I/O
 	5808  XMC-SLX150-1M: Reconfigurable Spartan-6 FPGA with plug-in I/O
@@ -13963,7 +14135,6 @@
 	8083  GL880 USB 1.1 UHCI controller
 	8084  GL880 USB 2.0 EHCI controller
 17aa  Lenovo
-	20b1  ThinkPad T61p
 17ab  Phillips Components
 17af  Hightech Information System Ltd.
 17b3  Hawking Technologies
@@ -14058,6 +14229,7 @@
 	6061  R6061 USB 2.0 Controller
 17f7  Topdek Semiconductor Inc.
 17f9  Gemtek Technology Co., Ltd
+17fc  IOGEAR, Inc.
 17fe  InProComm Inc.
 	2120  IPN 2120 802.11b
 		1737 0020  WMP11 v4 802.11b Wireless-B PCI Adapter
@@ -14081,6 +14253,7 @@
 	0200  RT2500 802.11g PCI [PC54G2]
 	0201  RT2500 Wireless 802.11bg
 		1043 130f  WL-130g
+		1186 3c00  DWL-G650X Wireless 11g CardBus Adapter
 		1371 001e  CWC-854 Wireless-G CardBus Adapter
 		1371 001f  CWM-854 Wireless-G Mini PCI Adapter
 		1371 0020  CWP-854 Wireless-G PCI Adapter
@@ -14094,6 +14267,7 @@
 		1814 2560  RT2500 Wireless 802.11bg
 		182d 9073  WL-115 Wireless Network PCI Adapter
 		185f 22a0  CN-WF513 Wireless Cardbus Adapter
+		18eb 5312  WL531P IEEE 802.11g PCI Card-EU
 		1948 3c00  C54RC v1 Wireless 11g CardBus Adapter
 		1948 3c01  C54Ri v1 Wireless 11g PCI Adapter
 	0300  Wireless Adapter Canyon CN-WF511
@@ -14122,7 +14296,7 @@
 	0601  RT2800 802.11n PCI
 		1799 801c  F5D8011 v3 802.11n N1 Wireless Notebook Card
 		187e 3412  NWD-310N 802.11n Wireless PCI Adapter
-	0681  RT2860 Wireless 802.11n PCIe
+	0681  RT2890 Wireless 802.11n PCIe
 		1458 e939  GN-WS30N-RH 802.11bgn Mini PCIe Card
 	0701  RT2760 Wireless 802.11n 1T/2R
 		1737 0074  WMP110 v2 802.11n RangePlus Wireless PCI Adapter
@@ -14134,6 +14308,8 @@
 		13bd 1057  GN-WS32L-RH Half-size Mini PCIe Card
 	3091  RT3091 Wireless 802.11n 1T/2R PCIe
 	3092  RT3092 Wireless 802.11n 2T/2R PCIe
+	5360  RT5360 Wireless 802.11n 1T/1R
+		1186 3c05  DWA-525 Wireless N 150 Desktop Adapter (rev.A2)
 	5390  RT5390 Wireless 802.11n 1T/1R PCIe
 		103c 1636  U98Z077.00 Half-size Mini PCIe Card
 	539f  RT5390 [802.11 b/g/n 1T1R G-band PCI Express Single Chip]
@@ -14143,6 +14319,7 @@
 1820  InfiniCon Systems Inc.
 1822  Twinhan Technology Co. Ltd
 	4e35  Mantis DTV PCI Bridge Controller [Ver 1.0]
+182c  BCM943228HM4L 802.11a/b/g/n 2x2 Wi-Fi Adapter
 182d  SiteCom Europe BV
 # HFC-based ISDN card
 	3069  ISDN PCI DC-105V2
@@ -14300,6 +14477,7 @@
 18ee  Chenming Mold Ind. Corp.
 18f1  Spectrum GmbH
 18f4  Napatech A/S
+	0031  NT20X Network Adapter
 	0051  NT20-X Capture Card
 	0061  NT20E Capture Card
 	0064  NT20E Inline Card
@@ -14309,6 +14487,9 @@
 	0091  NT20X Capture Card [New Rev]
 	00a1  NT4E-STD Capture Card
 	00a4  NT4E-STD Inline Card
+	00c5  NT20E2 Network Adapter 2x10Gb
+	00d5  NT40E2-4 Network Adapter 4x10Gb
+	00e5  NT40E2-1 Network Adapter 1x40Gb
 18f6  NextIO
 	1000  [Nexsis] Switch Virtual P2P PCIe Bridge
 	1050  [Nexsis] Switch Virtual P2P PCI Bridge
@@ -14387,15 +14568,21 @@
 		1924 6204  SFN5122F-R4
 		1924 6205  SFN5122F-R5
 		1924 6206  SFN5122F-R6
+		1924 6207  SFN5122F-R7
 		1924 6210  SFN5322F-R0
 		1924 6211  SFN5322F-R1
+		1924 6217  SFN5322F-R7
+		1924 6227  SFN6122F-R7
+		1924 6237  SFN6322F-R7
 		1924 6501  SFN5802K-R1
 		1924 6511  SFN5814H-R1
 		1924 6521  SFN5812H-R1
 		1924 6a05  SFN5112F-R5
 		1924 6a06  SFN5112F-R6
 		1924 7206  SFN5162F-R6
+		1924 7207  SFN5162F-R7
 		1924 7a06  SFN5152F-R6
+		1924 7a07  SFN5152F-R7
 	0813  SFL9021 [Solarstorm]
 		1924 6100  SFN5121T-R0
 		1924 6102  SFN5121T-R2
@@ -14505,6 +14692,7 @@
 	0111  P1022
 	0118  P1013E
 	0119  P1013
+	0128  P1010
 	0400  P4080E
 	0401  P4080
 	0408  P4040E
@@ -14545,6 +14733,8 @@
 	1067  L1c Gigabit Ethernet
 	1073  AR8151 v1.0 Gigabit Ethernet
 	1083  AR8151 v2.0 Gigabit Ethernet
+	1090  AR8162 Fast Ethernet
+	1091  AR8161 Gigabit Ethernet
 	2048  L2 Fast Ethernet
 	2060  AR8152 v1.1 Fast Ethernet
 	2062  AR8152 v2.0 Fast Ethernet
@@ -14598,6 +14788,8 @@
 	8001  RapidFile
 198a  Nallatech Ltd.
 1993  Innominate Security Technologies AG
+1999  A-Logics
+	a900  AM-7209 Video Processor
 199a  Pulse-LINK, Inc.
 199d  Xsigo Systems
 	8209  Virtual NIC Device
@@ -14704,11 +14896,13 @@
 1a76  Wavesat
 1a77  Lightfleet Corporation
 1a78  Virident Systems Inc.
-	0031  Virident tachIOn Drive
-		1a78 0034  tachIOn PCIe SSD [rev 3]
-		1a78 0037  tachIOn PCIe SSD [rev 3D]
-		1a78 0038  tachIOn PCIe SSD [rev 4]
-		1a78 0039  tachIOn PCIe SSD [rev 4D]
+	0031  Virident FlashMAX Drive
+		1a78 0034  FlashMAX PCIe SSD [rev 3]
+		1a78 0037  FlashMAX PCIe SSD [rev 3D]
+		1a78 0038  FlashMAX PCIe SSD [rev 4]
+		1a78 0039  FlashMAX PCIe SSD [rev 4D]
+	0040  Virident FlashMAX Drive V2
+		1a78 0040  PCIe SSD [LP Z1]
 1a84  Commex Technologies
 	0001  Vulcan SP HT6210 10-Gigabit Ethernet (rev 02)
 1a88  MEN Mikro Elektronik
@@ -14724,6 +14918,7 @@
 1ab6  CalDigit, Inc.
 	6201  RAID Card
 1ab9  Espia Srl
+1acc  Point of View B.V
 1ad7  Spectracom Corporation
 	8000  TSync-PCIe Time Code Processor
 	9100  TPRO-PCI-66U Timecode Reader/Generator
@@ -14784,6 +14979,7 @@
 	0601  NumaChip N601
 	0602  NumaChip N602
 1b4b  Marvell Technology Group Ltd.
+	0640  88SE9128 SATA III 6Gb/s RAID Controller
 	9120  88SE9120 SATA 6Gb/s Controller
 	9123  88SE9123 PCIe SATA 6.0 Gb/s controller
 	9125  88SE9125 PCIe SATA 6.0 Gb/s controller
@@ -14791,8 +14987,10 @@
 	9130  88SE9128 PCIe SATA 6 Gb/s RAID controller with HyperDuo
 		1043 8438  P8P67 Deluxe Motherboard
 	9172  88SE9172 SATA 6Gb/s Controller
+	917a  88SE9172 SATA III 6Gb/s RAID Controller
+	9192  88SE9172 SATA III 6Gb/s RAID Controller
 	91a0  88SE91A0 SATA 6Gb/s Controller
-	91a4  88SE91A4 SATA 6Gb/s Controller
+	91a4  88SE9128 IDE Controller
 	9480  88SE9480 SAS/SATA 6Gb/s RAID controller
 1b55  NetUP Inc.
 	2a2c  Dual DVB-S2-CI card
@@ -14801,6 +14999,7 @@
 	7023  EJ168 USB 3.0 Host Controller
 1b73  Fresco Logic
 	1000  FL1000G USB 3.0 Host Controller
+		1d5c 1000  Anker USB 3.0 Express Card
 1b85  OCZ Technology Group, Inc.
 	1041  RevoDrive 3 X2 PCI-Express SSD 240 GB (Marvell Controller)
 1b96  Western Digital
@@ -14817,16 +15016,21 @@
 	5308  BC-H08240A 8 port H.264 video and audio encoder / decoder
 	5310  BC-H16480A 16 port H.264 video and audio encoder / decoder
 1bb5  Quantenna Communications, Inc.
+1bbf  Maxeler Technologies Ltd.
+	0003  MAX3
 1bf4  VTI Instruments Corporation
 1c1c  Symphony
 	0001  82C101
 1c2c  Fiberblaze
 # Used on V120 VME Crate Controller
 1c32  Highland Technology, Inc.
+1c3b  Accensus, LLC
+	0200  Telas2
 1c44  Enmotus Inc
 	8000  8000 Storage IO Controller
 1d44  DPT
 	a400  PM2x24/PM3224
+1d5c  Fantasia Trading LLC
 1de1  Tekram Technology Co.,Ltd.
 	0391  TRM-S1040
 	2020  DC-390
@@ -14868,6 +15072,7 @@
 	0001  Spectrum Analyzer PC Card (SAgE)
 1fd4  SUNIX Co., Ltd.
 	0001  Matrix multiport serial adapter
+	1999  Multiport serial controller
 2000  Smart Link Ltd.
 	2800  SmartPCI2800 V.92 PCI Soft DFT
 2001  Temporal Research Ltd
@@ -15028,7 +15233,7 @@
 4321  Tata Power Strategic Electronics Division
 434e  CAST Navigation LLC
 4444  Internext Compression Inc
-	0016  iTVC16 (CX23416) MPEG-2 Encoder
+	0016  iTVC16 (CX23416) Video Decoder
 		0070 0003  WinTV PVR 250
 		0070 0009  WinTV PVR 150
 		0070 0801  WinTV PVR 150
@@ -15510,23 +15715,23 @@
 	0122  2nd Generation Core Processor Family Integrated Graphics Controller
 	0126  2nd Generation Core Processor Family Integrated Graphics Controller
 		1028 04cc  Vostro 3350
-	0150  Ivy Bridge DRAM Controller
-	0151  Ivy Bridge PCI Express Root Port
-	0152  Ivy Bridge Graphics Controller
-	0154  Ivy Bridge DRAM Controller
-	0155  Ivy Bridge PCI Express Root Port
-	0156  Ivy Bridge Graphics Controller
+	0150  3rd Gen Core processor DRAM Controller
+	0151  3rd Gen Core processor PCI Express Root Port
+	0152  3rd Gen Core processor Graphics Controller
+	0154  3rd Gen Core processor DRAM Controller
+	0155  3rd Gen Core processor PCI Express Root Port
+	0156  3rd Gen Core processor Graphics Controller
 	0158  Ivy Bridge DRAM Controller
-	0159  Ivy Bridge PCI Express Root Port
+	0159  3rd Gen Core processor PCI Express Root Port
 	015a  Ivy Bridge Graphics Controller
-	015c  Ivy Bridge DRAM Controller
-	015d  Ivy Bridge PCI Express Root Port
-	015e  Ivy Bridge Graphics Controller
-	0162  Ivy Bridge Graphics Controller
-	0166  Ivy Bridge Graphics Controller
-	016a  Ivy Bridge Graphics Controller
-	0172  Ivy Bridge Graphics Controller
-	0176  Ivy Bridge Graphics Controller
+	015c  3rd Gen Core processor DRAM Controller
+	015d  3rd Gen Core processor PCI Express Root Port
+	015e  3rd Gen Core processor Graphics Controller
+	0162  3rd Gen Core processor Graphics Controller
+	0166  3rd Gen Core processor Graphics Controller
+	016a  3rd Gen Core processor Graphics Controller
+	0172  3rd Gen Core processor Graphics Controller
+	0176  3rd Gen Core processor Graphics Controller
 	0309  80303 I/O Processor PCI-to-PCI Bridge
 	030d  80312 I/O Companion Chip PCI-to-PCI Bridge
 	0326  6700/6702PXH I/OxAPIC Interrupt Controller A
@@ -15550,6 +15755,12 @@
 	0372  80333 Segment-B PCI Express-to-PCI Express Bridge
 	0373  80333 B-Bus IOAPIC
 	0374  80333 Address Translation Unit
+	0402  Haswell Integrated Graphics Controller
+	0406  Haswell Integrated Graphics Controller
+	040a  Haswell Integrated Graphics Controller
+	0412  Haswell Integrated Graphics Controller
+	0416  Haswell Integrated Graphics Controller
+	041a  Haswell Integrated Graphics Controller
 	0436  DH89xxCC Gigabit SGMII Connection
 	0438  DH8900CC Series Gigabit Network Connection
 	043a  DH8900CC Series Gigabit Fiber Network Connection
@@ -15697,6 +15908,39 @@
 	0bf5  Atom Processor D2xxx/N2xxx DRAM Controller
 	0bf6  Atom Processor D2xxx/N2xxx DRAM Controller
 	0bf7  Atom Processor D2xxx/N2xxx DRAM Controller
+	0c00  Haswell DRAM Controller
+	0c01  Haswell PCI Express x16 Controller
+	0c04  Haswell DRAM Controller
+	0c05  Haswell PCI Express x8 Controller
+	0c08  Haswell DRAM Controller
+	0c09  Haswell PCI Express x4 Controller
+	0c0c  Haswell HD Audio Controller
+	0c46  Centerton PCI Express Root Port 1
+	0c47  Centerton PCI Express Root Port 2
+	0c48  Centerton PCI Express Root Port 3
+	0c49  Centerton PCI Express Root Port 4
+	0c4e  Centerton NTB Primary
+	0c54  Centerton Internal Management
+	0c59  Centerton SMBus 2.0 Controller 0
+	0c5a  Centerton SMBus 2.0 Controller 1
+	0c5f  Centerton UART
+	0c60  Centerton Integrated Legacy Bus
+	0c70  Centerton Internal Fabric
+	0c71  Centerton Internal Fabric
+	0c72  Centerton Internal Fabric
+	0c73  Centerton Internal Fabric
+	0c74  Centerton Internal Fabric
+	0c75  Centerton Internal Fabric
+	0c76  Centerton Internal Fabric
+	0c77  Centerton Internal Fabric
+	0c78  Centerton Internal Fabric
+	0c79  Centerton Internal Fabric
+	0c7a  Centerton Internal Fabric
+	0c7b  Centerton Internal Fabric
+	0c7c  Centerton Internal Fabric
+	0c7d  Centerton Internal Fabric
+	0c7e  Centerton Internal Fabric
+	0c7f  Centerton Internal Fabric
 	1000  82542 Gigabit Ethernet Controller (Fiber)
 		0e11 b0df  NC6132 Gigabit Ethernet Adapter (1000-SX)
 		0e11 b0e0  NC6133 Gigabit Ethernet Adapter (1000-LX)
@@ -16002,8 +16246,8 @@
 		17aa 2001  ThinkPad T60
 		17aa 207e  ThinkPad X60s
 		8086 109a  PRO/1000 PL Network Connection
-		8086 309c  DeskTop Board D945GTP
-		8086 30a5  DeskTop Board D975XBX
+		8086 309c  Desktop Board D945GTP
+		8086 30a5  Desktop Board D975XBX
 	109b  82546GB PRO/1000 GF Quad Port Server Adapter
 	109e  82597EX 10GbE Ethernet Controller
 		8086 a01f  PRO/10GbE CX4 Server Adapter
@@ -16067,6 +16311,7 @@
 		103c 323f  NC362i Integrated Dual port Gigabit Server Adapter
 		10a9 8028  UV-BaseIO dual-port GbE
 		13a3 0037  DS4100 Secure Multi-Gigabit Server Adapter with Compression
+		15d9 a811  H8DGU
 		8086 a01c  Gigabit ET Dual Port Server Adapter
 		8086 a03c  Gigabit ET Dual Port Server Adapter
 		8086 a04c  Gigabit ET Dual Port Server Adapter
@@ -16081,6 +16326,8 @@
 		8086 0001  Gigabit CT2 Desktop Adapter
 		8086 a01f  Gigabit CT Desktop Adapter
 		e4bf 50c1  PC1-GROOVE
+		e4bf 50c2  PC2-LIMBO
+	10d4  Matrox Concord GE (customized Intel 82574)
 	10d5  82571PT Gigabit PT Quad Port Server ExpressModule
 	10d6  82575GB Gigabit Network Connection
 		8086 10d6  Gigabit VT Quad Port Server Adapter
@@ -16392,6 +16639,7 @@
 	1511  82580 Gigabit SFP Connection
 	1514  82599EB 10 Gigabit KX4 Network Connection
 		8086 000b  Ethernet X520 10GbE Dual Port KX4 Mezz
+	1515  X540 Ethernet Controller Virtual Function
 	1516  82580 Gigabit Network Connection
 		8086 12b1  Ethernet Server Adapter I340-T2
 		8086 12b2  Ethernet Server Adapter I340-T2
@@ -16400,6 +16648,7 @@
 	1518  82576NS SerDes Gigabit Network Connection
 	151c  82599EB 10 Gigabit TN Network Connection
 		108e 7b13  Dual 10GBASE-T LP
+	1520  I350 Ethernet Controller Virtual Function
 	1521  I350 Gigabit Network Connection
 		1028 1f60  Intel GbE 4P I350crNDC
 		1028 1f62  Intel GbE 2P I350crNDC
@@ -16436,7 +16685,7 @@
 	1527  82580 Gigabit Fiber Network Connection
 		8086 0001  Ethernet Server Adapter I340-F4
 		8086 0002  Ethernet Server Adapter I340-F4
-	1528  Ethernet Controller 10 Gigabit X540-AT2
+	1528  Ethernet Controller 10-Gigabit X540-AT2
 		108e 7b14  Sun Dual Port 10 GbE PCIe 2.0 ExpressModule, Base-T
 		108e 7b15  Sun Dual Port 10 GbE PCIe 2.0 Low Profile Adapter, Base-T
 		8086 0001  Ethernet Converged Network Adapter X540-T2
@@ -16450,6 +16699,7 @@
 	154a  Ethernet Server Adapter X520-4
 		8086 011a  Ethernet Converged Network Adapter X520-4
 		8086 011b  Ethernet Converged Network Adapter X520-4
+		8086 011c  Ethernet Converged Network Adapter X520-4
 	154d  82599EB 10-Gigabit SFP+ Network Connection
 		8086 7b11  10GbE 2P X520 Adapter
 	1960  80960RP (i960RP) Microprocessor
@@ -16579,133 +16829,132 @@
 	1c5d  6 Series/C200 Series Chipset Family LPC Controller
 	1c5e  6 Series/C200 Series Chipset Family LPC Controller
 	1c5f  6 Series/C200 Series Chipset Family LPC Controller
-	1d00  X79 series chipset 4-Port SATA IDE Controller
-	1d02  X79 series chipset 6-Port SATA AHCI Controller
-	1d04  X79 series chipset SATA RAID Controller
-	1d06  Patsburg SATA Premium RAID Controller
-	1d08  X79 series chipset 2-Port SATA IDE Controller
-	1d10  X79 series chipset PCI Express Root Port 1
-	1d11  X79 series chipset PCI Express Root Port 1
-	1d12  X79 series chipset PCI Express Root Port 2
-	1d13  X79 series chipset PCI Express Root Port 2
-	1d14  X79 series chipset PCI Express Root Port 3
-	1d15  X79 series chipset PCI Express Root Port 3
-	1d16  X79 series chipset PCI Express Root Port 4
-	1d17  X79 series chipset PCI Express Root Port 4
-	1d18  X79 series chipset PCI Express Root Port 5
-	1d19  X79 series chipset PCI Express Root Port 5
-	1d1a  X79 series chipset PCI Express Root Port 6
-	1d1b  X79 series chipset PCI Express Root Port 6
-	1d1c  X79 series chipset PCI Express Root Port 7
-	1d1d  X79 series chipset PCI Express Root Port 7
-	1d1e  X79 series chipset PCI Express Root Port 8
-	1d1f  X79 series chipset PCI Express Root Port 8
-	1d20  X79 series chipset High Definition Audio Controller
-	1d22  X79 series chipset SMBus Host Controller
-	1d24  X79 series chipset Thermal Management Controller
-	1d25  X79 series chipset DMI to PCI Bridge
-	1d26  X79 series chipset USB2 Enhanced Host Controller #1
-	1d2d  X79 series chipset USB2 Enhanced Host Controller #2
-	1d33  X79 series chipset LAN Controller
-	1d35  Patsburg VECI Controller
-	1d3a  X79 series chipset HECI Controller #2
-	1d3b  X79 series chipset IDE-r Controller
-	1d3d  X79 series chipset KT Controller
-	1d3e  Patsburg PCI Express Virtual Root Port
-	1d3f  Patsburg PCI Express Virtual Switch Port
-	1d40  X79 series chipset LPC Controller
-	1d41  X79 series chipset LPC Controller
-	1d50  Patsburg Dual 4-Port SATA/SAS Storage Control Unit
-	1d54  Patsburg Dual 4-Port SATA/SAS Storage Control Unit
-	1d55  X79 series chipset 4-Port SATA/SAS Storage Control Unit
-	1d58  Patsburg Dual 4-Port SATA/SAS Storage Control Unit
-	1d59  X79 series chipset 4-Port SATA/SAS Storage Control Unit
-	1d5a  Patsburg Dual 4-Port SATA Storage Control Unit
-	1d5b  Patsburg 4-Port SATA Storage Control Unit
-	1d5c  Patsburg Dual 4-Port SATA/SAS Storage Control Unit
-	1d5d  Patsburg 4-Port SATA/SAS Storage Control Unit
-	1d5e  Patsburg Dual 4-Port SATA Storage Control Unit
-	1d5f  Patsburg 4-Port SATA Storage Control Unit
-	1d60  Patsburg Dual 4-Port SATA/SAS Storage Control Unit
-	1d64  Patsburg Dual 4-Port SATA/SAS Storage Control Unit
-	1d65  X79 series chipset 4-Port SATA/SAS Storage Control Unit
-	1d68  Patsburg Dual 4-Port SATA/SAS Storage Control Unit
-	1d69  X79 series chipset 4-Port SATA/SAS Storage Control Unit
-	1d6a  Patsburg Dual 4-Port SATA Storage Control Unit
-	1d6b  Patsburg 4-Port SATA Storage Control Unit
-	1d6c  Patsburg Dual 4-Port SATA/SAS Storage Control Unit
-	1d6d  Patsburg 4-Port SATA/SAS Storage Control Unit
-	1d6e  Patsburg Dual 4-Port SATA Storage Control Unit
-	1d6f  Patsburg 4-Port SATA Storage Control Unit
-	1d70  X79 series chipset SMBus Controller 0
-	1d71  Patsburg SMBus Controller 1
-	1d72  Patsburg SMBus Controller 2
-	1d73  Patsburg Integrated NVSRAM Controller
-	1d74  Patsburg PCI Express Upstream Port
-	1d76  Patsburg Multi-Function Glue
-	1e00  Panther Point 4 port SATA Controller [IDE mode]
-	1e01  Panther Point 4 port SATA Controller [IDE mode]
-	1e02  Panther Point 6 port SATA Controller [AHCI mode]
-	1e03  Panther Point 6 port SATA Controller [AHCI mode]
-	1e04  Panther Point SATA Controller [RAID mode]
+	1d00  C600/X79 series chipset 4-Port SATA IDE Controller
+	1d02  C600/X79 series chipset 6-Port SATA AHCI Controller
+	1d04  C600/X79 series chipset SATA RAID Controller
+	1d06  C600/X79 series chipset SATA Premium RAID Controller
+	1d08  C600/X79 series chipset 2-Port SATA IDE Controller
+	1d10  C600/X79 series chipset PCI Express Root Port 1
+	1d11  C600/X79 series chipset PCI Express Root Port 1
+	1d12  C600/X79 series chipset PCI Express Root Port 2
+	1d13  C600/X79 series chipset PCI Express Root Port 2
+	1d14  C600/X79 series chipset PCI Express Root Port 3
+	1d15  C600/X79 series chipset PCI Express Root Port 3
+	1d16  C600/X79 series chipset PCI Express Root Port 4
+	1d17  C600/X79 series chipset PCI Express Root Port 4
+	1d18  C600/X79 series chipset PCI Express Root Port 5
+	1d19  C600/X79 series chipset PCI Express Root Port 5
+	1d1a  C600/X79 series chipset PCI Express Root Port 6
+	1d1b  C600/X79 series chipset PCI Express Root Port 6
+	1d1c  C600/X79 series chipset PCI Express Root Port 7
+	1d1d  C600/X79 series chipset PCI Express Root Port 7
+	1d1e  C600/X79 series chipset PCI Express Root Port 8
+	1d1f  C600/X79 series chipset PCI Express Root Port 8
+	1d20  C600/X79 series chipset High Definition Audio Controller
+	1d22  C600/X79 series chipset SMBus Host Controller
+	1d24  C600/X79 series chipset Thermal Management Controller
+	1d25  C600/X79 series chipset DMI to PCI Bridge
+	1d26  C600/X79 series chipset USB2 Enhanced Host Controller #1
+	1d2d  C600/X79 series chipset USB2 Enhanced Host Controller #2
+	1d33  C600/X79 series chipset LAN Controller
+	1d35  C600/X79 series chipset VECI Controller
+	1d3a  C600/X79 series chipset MEI Controller #1
+	1d3b  C600/X79 series chipset MEI Controller #2
+	1d3c  C600/X79 series chipset IDE-r Controller
+	1d3d  C600/X79 series chipset KT Controller
+	1d3e  C600/X79 series chipset PCI Express Virtual Root Port
+	1d3f  C608/C606/X79 series chipset PCI Express Virtual Switch Port
+	1d40  C600/X79 series chipset LPC Controller
+	1d41  C600/X79 series chipset LPC Controller
+	1d50  C608 chipset Dual 4-Port SATA/SAS Storage Control Unit
+	1d54  C600/X79 series chipset Dual 4-Port SATA/SAS Storage Control Unit
+	1d55  C600/X79 series chipset 4-Port SATA/SAS Storage Control Unit
+	1d58  C606 chipset Dual 4-Port SATA/SAS Storage Control Unit
+	1d59  C604/X79 series chipset 4-Port SATA/SAS Storage Control Unit
+	1d5a  C600/X79 series chipset Dual 4-Port SATA Storage Control Unit
+	1d5b  C602 chipset 4-Port SATA Storage Control Unit
+	1d5c  C600/X79 series chipset Dual 4-Port SATA/SAS Storage Control Unit
+	1d5d  C600/X79 series chipset 4-Port SATA/SAS Storage Control Unit
+	1d5e  C600/X79 series chipset Dual 4-Port SATA Storage Control Unit
+	1d5f  C600/X79 series chipset 4-Port SATA Storage Control Unit
+	1d60  C608 chipset Dual 4-Port SATA/SAS Storage Control Unit
+	1d64  C600/X79 series chipset Dual 4-Port SATA/SAS Storage Control Unit
+	1d65  C600/X79 series chipset 4-Port SATA/SAS Storage Control Unit
+	1d68  C606 chipset Dual 4-Port SATA/SAS Storage Control Unit
+	1d69  C604/X79 series chipset 4-Port SATA/SAS Storage Control Unit
+	1d6a  C600/X79 series chipset Dual 4-Port SATA Storage Control Unit
+	1d6b  C602 chipset 4-Port SATA Storage Control Unit
+	1d6c  C600/X79 series chipset Dual 4-Port SATA/SAS Storage Control Unit
+	1d6d  C600/X79 series chipset 4-Port SATA/SAS Storage Control Unit
+	1d6e  C600/X79 series chipset Dual 4-Port SATA Storage Control Unit
+	1d6f  C600/X79 series chipset 4-Port SATA Storage Control Unit
+	1d70  C600/X79 series chipset SMBus Controller 0
+	1d71  C608/C606/X79 series chipset SMBus Controller 1
+	1d72  C608 chipset SMBus Controller 2
+	1d74  C608/C606/X79 series chipset PCI Express Upstream Port
+	1d76  C600/X79 series chipset Multi-Function Glue
+	1e00  7 Series Chipset Family 4-port SATA IDE Controller
+	1e01  7 Series Chipset Family 4-port SATA IDE Controller
+	1e02  7 Series Chipset Family 6-port SATA AHCI Controller
+	1e03  7 Series Chipset Family 6-port SATA AHCI Controller
+	1e04  7 Series Chipset Family SATA RAID Controller
 	1e05  Panther Point SATA Controller [RAID mode]
-	1e06  Panther Point SATA Controller [Premium RAID mode]
-	1e07  Panther Point SATA Controller [Premium RAID mode]
-	1e08  Panther Point 2 port SATA Controller [IDE mode]
-	1e09  Panther Point 2 port SATA Controller [IDE mode]
-	1e0e  Panther Point SATA Controller [RAID mode]
-	1e10  Panther Point PCI Express Root Port 1
-	1e12  Panther Point PCI Express Root Port 2
-	1e14  Panther Point PCI Express Root Port 3
-	1e16  Panther Point PCI Express Root Port 4
-	1e18  Panther Point PCI Express Root Port 5
-	1e1a  Panther Point PCI Express Root Port 6
-	1e1c  Panther Point PCI Express Root Port 7
-	1e1e  Panther Point PCI Express Root Port 8
-	1e20  Panther Point High Definition Audio Controller
-	1e22  Panther Point SMBus Controller
-	1e24  Panther Point Thermal Management Controller
-	1e25  Panther Point DMI to PCI Bridge
-	1e26  Panther Point USB Enhanced Host Controller #1
-	1e2d  Panther Point USB Enhanced Host Controller #2
-	1e31  Panther Point USB xHCI Host Controller
-	1e33  Panther Point LAN Controller
-	1e3a  Panther Point MEI Controller #1
-	1e3b  Panther Point MEI Controller #2
-	1e3c  Panther Point IDE-r Controller
-	1e3d  Panther Point KT Controller
-	1e40  Panther Point LPC Controller
-	1e41  Panther Point LPC Controller
-	1e42  Panther Point LPC Controller
-	1e43  Panther Point LPC Controller
-	1e44  Panther Point LPC Controller
-	1e45  Panther Point LPC Controller
-	1e46  Panther Point LPC Controller
-	1e47  Panther Point LPC Controller
-	1e48  Panther Point LPC Controller
-	1e49  Panther Point LPC Controller
-	1e4a  Panther Point LPC Controller
-	1e4b  Panther Point LPC Controller
-	1e4c  Panther Point LPC Controller
-	1e4d  Panther Point LPC Controller
-	1e4e  Panther Point LPC Controller
-	1e4f  Panther Point LPC Controller
-	1e50  Panther Point LPC Controller
-	1e51  Panther Point LPC Controller
-	1e52  Panther Point LPC Controller
+	1e06  7 Series Chipset Family SATA Premium RAID Controller
+	1e07  7 Series Chipset Family SATA Premium RAID Controller
+	1e08  7 Series Chipset Family 2-port SATA IDE Controller
+	1e09  7 Series Chipset Family 2-port SATA IDE Controller
+	1e0e  7 Series Chipset Family SATA RAID Controller
+	1e10  7 Series Chipset Family PCI Express Root Port 1
+	1e12  7 Series Chipset Family PCI Express Root Port 2
+	1e14  7 Series Chipset Family PCI Express Root Port 3
+	1e16  7 Series Chipset Family PCI Express Root Port 4
+	1e18  7 Series Chipset Family PCI Express Root Port 5
+	1e1a  7 Series Chipset Family PCI Express Root Port 6
+	1e1c  7 Series Chipset Family PCI Express Root Port 7
+	1e1e  7 Series Chipset Family PCI Express Root Port 8
+	1e20  7 Series Chipset Family High Definition Audio Controller
+	1e22  7 Series Chipset Family SMBus Controller
+	1e24  7 Series Chipset Family Thermal Management Controller
+	1e25  7 Series Chipset Family DMI to PCI Bridge
+	1e26  7 Series Chipset Family USB Enhanced Host Controller #1
+	1e2d  7 Series Chipset Family USB Enhanced Host Controller #2
+	1e31  7 Series Chipset Family USB xHCI Host Controller
+	1e33  7 Series Chipset Family LAN Controller
+	1e3a  7 Series Chipset Family MEI Controller #1
+	1e3b  7 Series Chipset Family MEI Controller #2
+	1e3c  7 Series Chipset Family IDE-r Controller
+	1e3d  7 Series Chipset Family KT Controller
+	1e41  7 Series Chipset Family LPC Controller
+	1e42  7 Series Chipset Family LPC Controller
+	1e43  7 Series Chipset Family LPC Controller
+	1e44  Z77 Express Chipset LPC Controller
+	1e45  7 Series Chipset Family LPC Controller
+	1e46  Z75 Express Chipset LPC Controller
+	1e47  Q77 Express Chipset LPC Controller
+	1e48  Q75 Express Chipset LPC Controller
+	1e49  B75 Express Chipset LPC Controller
+	1e4a  H77 Express Chipset LPC Controller
+	1e4b  7 Series Chipset Family LPC Controller
+	1e4c  7 Series Chipset Family LPC Controller
+	1e4d  7 Series Chipset Family LPC Controller
+	1e4e  7 Series Chipset Family LPC Controller
+	1e4f  7 Series Chipset Family LPC Controller
+	1e50  7 Series Chipset Family LPC Controller
+	1e51  7 Series Chipset Family LPC Controller
+	1e52  7 Series Chipset Family LPC Controller
 	1e53  Panther Point LPC Controller
-	1e54  Panther Point LPC Controller
-	1e55  Panther Point LPC Controller
-	1e56  Panther Point LPC Controller
-	1e57  Panther Point LPC Controller
-	1e58  Panther Point LPC Controller
-	1e59  Panther Point LPC Controller
-	1e5a  Panther Point LPC Controller
-	1e5b  Panther Point LPC Controller
-	1e5c  Panther Point LPC Controller
-	1e5d  Panther Point LPC Controller
-	1e5e  Panther Point LPC Controller
-	1e5f  Panther Point LPC Controller
+	1e54  7 Series Chipset Family LPC Controller
+	1e55  QM77 Express Chipset LPC Controller
+	1e56  QS77 Express Chipset LPC Controller
+	1e57  HM77 Express Chipset LPC Controller
+	1e58  UM77 Express Chipset LPC Controller
+	1e59  HM76 Express Chipset LPC Controller
+	1e5a  7 Series Chipset Family LPC Controller
+	1e5b  UM77 Express Chipset LPC Controller
+	1e5c  7 Series Chipset Family LPC Controller
+	1e5d  HM75 Express Chipset LPC Controller
+	1e5e  7 Series Chipset Family LPC Controller
+	1e5f  7 Series Chipset Family LPC Controller
 	2310  DH89xxCC LPC Controller
 	2323  DH89xxCC 4 Port SATA AHCI Controller
 	2330  DH89xxCC SMBus Controller
@@ -16802,6 +17051,7 @@
 		1025 1016  Travelmate 612 TX
 		104d 80df  Vaio PCG-FX403
 	2448  82801 Mobile PCI Bridge
+		1028 040b  Latitude E6510
 # (rev d3) (prog-if (rev d3) (prog-if 01 [Subtractive decode])
 		103c 0934  HP Compaq nw8240 Mobile Workstation
 		103c 099c  NX6110/NC6120
@@ -16865,6 +17115,7 @@
 		1028 020d  Inspiron 530
 		1028 0211  Optiplex 755
 		103c 2a3b  Pavilion A1512X
+		103c 330b  ProLiant ML150 G6 Server
 		1458 5000  GA-EP45-DS5 Motherboard
 		1775 11cc  CC11/CL11
 	2450  82801E ISA Bridge (LPC)
@@ -17697,6 +17948,7 @@
 		15d9 8680  X7DVL-E-O motherboard
 		8086 3476  Intel S5000PSLSATA Server Board
 	2682  631xESB/632xESB SATA RAID Controller
+		103c 31fe  Adaptec Serial ATA HostRAID
 	2683  631xESB/632xESB SATA RAID Controller
 	2688  631xESB/632xESB/3100 Chipset UHCI USB Controller #1
 		1028 01bb  PowerEdge 1955 onboard USB
@@ -17784,10 +18036,10 @@
 		103c 30a1  NC2400
 		1775 11cc  CC11/CL11 integrated graphics (secondary)
 		17aa 201a  ThinkPad T60/R60 series
-	27ac  Mobile 945GME Express Memory Controller Hub
+	27ac  Mobile 945GSE Express Memory Controller Hub
 		1775 11cc  CC11/CL11
-	27ad  Mobile 945GME Express PCI Express Root Port
-	27ae  Mobile 945GME Express Integrated Graphics Controller
+	27ad  Mobile 945GSE Express PCI Express Root Port
+	27ae  Mobile 945GSE Express Integrated Graphics Controller
 		1775 11cc  CC11/CL11 integrated graphics (primary)
 	27b0  82801GH (ICH7DH) LPC Interface Bridge
 		103c 2a3b  Pavilion A1512X
@@ -18028,7 +18280,7 @@
 	2825  82801HR/HO/HH (ICH8R/DO/DH) 2 port SATA Controller [IDE mode]
 		1028 01da  OptiPlex 745
 		1462 7235  P965 Neo MS-7235 mainboard
-	2826  Patsburg SATA RAID Controller
+	2826  C600/X79 series chipset SATA RAID Controller
 	2828  82801HM/HEM (ICH8M/ICH8M-E) SATA Controller [IDE mode]
 		1028 01f3  Inspiron 1420
 		103c 30c0  Compaq 6710b
@@ -18212,6 +18464,7 @@
 		1028 0211  Optiplex 755
 	2916  82801IR (ICH9R) LPC Interface Controller
 		1028 020d  Inspiron 530
+		8086 5044  Desktop Board DP35DP
 	2917  ICH9M-E LPC Interface Controller
 		e4bf cc4d  CCM-BOOGIE
 	2918  82801IB (ICH9) LPC Interface Controller
@@ -18230,6 +18483,7 @@
 		1028 0237  PowerEdge T610 SATA IDE Controller
 		1462 7360  G33/P35 Neo
 	2922  82801IR/IO/IH (ICH9R/DO/DH) 6 port SATA Controller [AHCI mode]
+		8086 5044  Desktop Board DP35DP
 	2923  82801IB (ICH9) 4 port SATA Controller [AHCI mode]
 	2925  82801IR/IO (ICH9R/DO) SATA Controller [RAID mode]
 		1734 10e0  System Board D2542
@@ -18253,6 +18507,7 @@
 		1028 0211  Optiplex 755
 		103c 3628  dv6-1190en
 		1462 7360  G33/P35 Neo
+		8086 5044  Desktop Board DP35DP
 		e4bf cc4d  CCM-BOOGIE
 	2932  82801I (ICH9 Family) Thermal Subsystem
 		103c 3628  dv6-1190en
@@ -18269,6 +18524,7 @@
 		1028 029c  PowerEdge M710 USB UHCI Controller
 		1028 2011  Optiplex 755
 		1462 7360  G33/P35 Neo
+		8086 5044  Desktop Board DP35DP
 		e4bf cc4d  CCM-BOOGIE
 	2935  82801I (ICH9 Family) USB UHCI Controller #2
 		1028 020d  Inspiron 530
@@ -18282,6 +18538,7 @@
 		1028 0287  PowerEdge M610 onboard UHCI
 		1028 029c  PowerEdge M710 USB UHCI Controller
 		1462 7360  G33/P35 Neo
+		8086 5044  Desktop Board DP35DP
 		e4bf cc4d  CCM-BOOGIE
 	2936  82801I (ICH9 Family) USB UHCI Controller #3
 		1028 020d  Inspiron 530
@@ -18293,6 +18550,7 @@
 		1028 0287  PowerEdge M610 onboard UHCI
 		1028 029c  PowerEdge M710 USB UHCI Controller
 		1462 7360  G33/P35 Neo
+		8086 5044  Desktop Board DP35DP
 		e4bf cc4d  CCM-BOOGIE
 	2937  82801I (ICH9 Family) USB UHCI Controller #4
 		1028 020d  Inspiron 530
@@ -18306,6 +18564,7 @@
 		1462 7360  G33/P35 Neo
 		8086 2937  Optiplex 755
 		8086 2942  828011 (ICH9 Family ) USB UHCI Controller
+		8086 5044  Desktop Board DP35DP
 		e4bf cc4d  CCM-BOOGIE
 	2938  82801I (ICH9 Family) USB UHCI Controller #5
 		1028 020d  Inspiron 530
@@ -18317,12 +18576,14 @@
 		1028 029c  PowerEdge M710 USB UHCI Controller
 		1462 7360  G33/P35 Neo
 		8086 2938  Optiplex 755
+		8086 5044  Desktop Board DP35DP
 		e4bf cc4d  CCM-BOOGIE
 	2939  82801I (ICH9 Family) USB UHCI Controller #6
 		1028 020d  Inspiron 530
 		1028 0210  PowerEdge T300 onboard UHCI
 		1028 0237  PowerEdge T610 USB UHCI Controller
 		1462 7360  G33/P35 Neo
+		8086 5044  Desktop Board DP35DP
 		e4bf cc4d  CCM-BOOGIE
 	293a  82801I (ICH9 Family) USB2 EHCI Controller #1
 		1028 020d  Inspiron 530
@@ -18336,6 +18597,7 @@
 		1028 0287  PowerEdge M610 onboard EHCI
 		1028 029c  PowerEdge M710 USB EHCI Controller
 		1462 7360  G33/P35 Neo
+		8086 5044  Desktop Board DP35DP
 		e4bf cc4d  CCM-BOOGIE
 	293c  82801I (ICH9 Family) USB2 EHCI Controller #2
 		1028 020d  Inspiron 530
@@ -18347,6 +18609,7 @@
 		1028 029c  PowerEdge M710 USB EHCI Controller
 		1462 7360  G33/P35 Neo
 		8086 293c  Optiplex 755
+		8086 5044  Desktop Board DP35DP
 		e4bf cc4d  CCM-BOOGIE
 	293e  82801I (ICH9 Family) HD Audio Controller
 		1028 020d  Inspiron 530
@@ -18424,6 +18687,7 @@
 		1028 020d  Inspiron 530
 		1043 82b0  P5KPL-VM Motherboard
 		1462 7360  G33/P35 Neo
+		8086 5044  Desktop Board DP35DP
 	29c1  82G33/G31/P35/P31 Express PCI Express Root Port
 		1028 020d  Inspiron 530
 	29c2  82G33/G31 Express Integrated Graphics Controller
@@ -18433,6 +18697,7 @@
 		1028 020d  Inspiron 530
 		1043 82b0  P5KPL-VM Motherboard
 	29c4  82G33/G31/P35/P31 Express MEI Controller
+		8086 5044  Desktop Board DP35DP
 	29c5  82G33/G31/P35/P31 Express MEI Controller
 	29c6  82G33/G31/P35/P31 Express PT IDER Controller
 	29c7  82G33/G31/P35/P31 Express Serial KT Controller
@@ -18687,11 +18952,13 @@
 	2e16  4 Series Chipset PT IDER Controller
 	2e17  4 Series Chipset Serial KT Controller
 	2e20  4 Series Chipset DRAM Controller
-		1458 5000  GA-EP45-DS5 Motherboard
+		1458 5000  GA-EP45-DS5/GA-EG45M-DS2H Motherboard
 	2e21  4 Series Chipset PCI Express Root Port
 		1458 5000  GA-EP45-DS5 Motherboard
 	2e22  4 Series Chipset Integrated Graphics Controller
+		1458 d000  GA-EG45M-DS2H Mainboard
 	2e23  4 Series Chipset Integrated Graphics Controller
+		1458 d000  GA-EG45M-DS2H Mainboard
 	2e24  4 Series Chipset HECI Controller
 	2e25  4 Series Chipset HECI Controller
 	2e26  4 Series Chipset PT IDER Controller
@@ -18775,17 +19042,22 @@
 		1028 0287  PowerEdge M610 I/O Hub to ESI Port
 		1028 028c  PowerEdge R410 I/O Hub to ESI Port
 		1028 028d  PowerEdge T410 I/O Hub to ESI Port
+		103c 330b  ProLiant ML150 G6 Server
 	3404  5520/5500/X58 I/O Hub to ESI Port
 	3405  5520/5500/X58 I/O Hub to ESI Port
 	3406  5520 I/O Hub to ESI Port
+		103c 330b  ProLiant DL160 G6
 	3407  5520/5500/X58 I/O Hub to ESI Port
 	3408  5520/5500/X58 I/O Hub PCI Express Root Port 1
+		103c 330b  ProLiant ML150 G6 Server
 	3409  5520/5500/X58 I/O Hub PCI Express Root Port 2
 	340a  5520/5500/X58 I/O Hub PCI Express Root Port 3
+		103c 330b  ProLiant ML150 G6 Server
 	340b  5520/X58 I/O Hub PCI Express Root Port 4
 	340c  5520/X58 I/O Hub PCI Express Root Port 5
 	340d  5520/X58 I/O Hub PCI Express Root Port 6
 	340e  5520/5500/X58 I/O Hub PCI Express Root Port 7
+		103c 330b  ProLiant ML150 G6 Server
 	340f  5520/5500/X58 I/O Hub PCI Express Root Port 8
 	3410  5520/5500/X58 I/O Hub PCI Express Root Port 9
 	3411  5520/5500/X58 I/O Hub PCI Express Root Port 10
@@ -18794,7 +19066,9 @@
 	3420  5500 Non-Legacy I/O Hub PCI Express Root Port 0
 	3421  5520 Non-Legacy I/O Hub PCI Express Root Port 0
 	3422  5520/5500/X58 I/O Hub GPIO and Scratch Pad Registers
+		103c 330b  ProLiant DL160 G6
 	3423  5520/5500/X58 I/O Hub Control Status and RAS Registers
+		103c 330b  ProLiant DL160 G6
 	3425  5520/5500/X58 Physical and Link Layer Registers Port 0
 	3426  5520/5500/X58 Routing and Protocol Layer Registers Port 0
 	3427  5520/5500 Physical and Link Layer Registers Port 1
@@ -18805,6 +19079,7 @@
 	342c  5520/5500/X58 Chipset QuickData Technology Device
 	342d  5520/5500/X58 I/O Hub I/OxAPIC Interrupt Controller
 	342e  5520/5500/X58 I/O Hub System Management Registers
+		103c 330b  ProLiant DL160 G6
 	342f  5520/5500/X58 Trusted Execution Technology Registers
 	3430  5520/5500/X58 Chipset QuickData Technology Device
 	3431  5520/5500/X58 Chipset QuickData Technology Device
@@ -18823,14 +19098,14 @@
 	3515  6310ESB PCI Express Downstream Port E2
 	3518  6311ESB/6321ESB PCI Express Downstream Port E3
 	3519  6310ESB PCI Express Downstream Port E3
-	3575  82830 830 Chipset Host Bridge
+	3575  82830M/MG/MP Host Bridge
 		0e11 0030  Evo N600c
 		1014 021d  ThinkPad A/T/X Series
 		104d 80e7  VAIO PCG-GR214EP/GR214MP/GR215MP/GR314MP/GR315MP
-	3576  82830 830 Chipset AGP Bridge
-	3577  82830 CGC [Chipset Graphics Controller]
+	3576  82830M/MP AGP Bridge
+	3577  82830M/MG Integrated Graphics Controller
 		1014 0513  ThinkPad A/T/X Series
-	3578  82830 830 Chipset Host Bridge
+	3578  82830M/MG/MP Host Bridge
 	3580  82852/82855 GM/GME/PM/GMV Processor to I/O Controller
 		1014 055c  ThinkPad R50e
 		1028 0139  Latitude D400
@@ -18996,6 +19271,7 @@
 	3a16  82801JIR (ICH10R) LPC Interface Controller
 		1028 028c  PowerEdge R410 LPC Interface Controller
 		1028 028d  PowerEdge T410 LPC Interface Controller
+		103c 330b  ProLiant G6 series
 		1458 5001  GA-EP45-DS5 Motherboard
 	3a18  82801JIB (ICH10) LPC Interface Controller
 	3a1a  82801JD (ICH10D) LPC Interface Controller
@@ -19003,8 +19279,9 @@
 		1028 028c  PowerEdge R410 SATA IDE Controller
 		1028 028d  PowerEdge T410 SATA IDE Controller
 	3a22  82801JI (ICH10 Family) SATA AHCI Controller
+		103c 330b  ProLiant G6 series
 		1043 82d4  P5Q Deluxe Motherboard
-		1458 b005  GA-EP45-DS5 Motherboard
+		1458 b005  GA-EP45-DS5/GA-EG45M-DS2H Motherboard
 	3a25  82801JIR (ICH10R) SATA RAID Controller
 		1028 028c  PERC S100 Controller (PE R410)
 		1028 028d  PERC S100 Controller (PE T410)
@@ -19013,48 +19290,57 @@
 		1028 028c  PowerEdge R410 SATA IDE Controller
 		1028 028d  PowerEdge T410 SATA IDE Controller
 	3a30  82801JI (ICH10 Family) SMBus Controller
-		1458 5001  GA-EP45-DS5 Motherboard
+		1458 5001  GA-EP45-DS5/GA-EG45M-DS2H Motherboard
 	3a32  82801JI (ICH10 Family) Thermal Subsystem
 	3a34  82801JI (ICH10 Family) USB UHCI Controller #1
 		1028 028c  PowerEdge R410 USB UHCI Controller
 		1028 028d  PowerEdge T410 USB UHCI Controller
+		103c 330b  ProLiant G6 series
 		1458 5004  GA-EP45-DS5 Motherboard
 	3a35  82801JI (ICH10 Family) USB UHCI Controller #2
 		1028 028c  PowerEdge R410 USB UHCI Controller
 		1028 028d  PowerEdge T410 USB UHCI Controller
+		103c 330b  ProLiant G6 series
 		1458 5004  GA-EP45-DS5 Motherboard
 	3a36  82801JI (ICH10 Family) USB UHCI Controller #3
 		1028 028c  PowerEdge R410 USB UHCI Controller
 		1028 028d  PowerEdge T410 USB UHCI Controller
+		103c 330b  ProLiant G6 series
 		1458 5004  GA-EP45-DS5 Motherboard
 	3a37  82801JI (ICH10 Family) USB UHCI Controller #4
 		1028 028c  PowerEdge R410 USB UHCI Controller
 		1028 028d  PowerEdge T410 USB UHCI Controller
-		1458 5004  GA-EP45-DS5 Motherboard
+		103c 330b  ProLiant G6 series
+		1458 5004  GA-EP45-DS5/GA-EG45M-DS2H Motherboard
 	3a38  82801JI (ICH10 Family) USB UHCI Controller #5
 		1028 028c  PowerEdge R410 USB UHCI Controller
 		1028 028d  PowerEdge T410 USB UHCI Controller
-		1458 5004  GA-EP45-DS5 Motherboard
+		103c 330b  ProLiant ML150 G6 Server
+		1458 5004  GA-EP45-DS5/GA-EG45M-DS2H Motherboard
 	3a39  82801JI (ICH10 Family) USB UHCI Controller #6
 		1028 028c  PowerEdge R410 USB UHCI Controller
 		1028 028d  PowerEdge T410 USB UHCI Controller
+		103c 330b  ProLiant ML150 G6 Server
 		1458 5004  GA-EP45-DS5 Motherboard
 	3a3a  82801JI (ICH10 Family) USB2 EHCI Controller #1
 		1028 028c  PowerEdge R410 USB EHCI Controller
 		1028 028d  PowerEdge T410 USB EHCI Controller
+		103c 330b  ProLiant G6 series
 		1458 5006  GA-EP45-DS5 Motherboard
 	3a3c  82801JI (ICH10 Family) USB2 EHCI Controller #2
 		1028 028c  PowerEdge R410 USB EHCI Controller
 		1028 028d  PowerEdge T410 USB EHCI Controller
+		103c 330b  ProLiant G6 series
 		1458 5006  GA-EP45-DS5 Motherboard
 	3a3e  82801JI (ICH10 Family) HD Audio Controller
 		1458 a002  GA-EP45-UD3R Motherboard
-		1458 a102  GA-EP45-DS5 Motherboard
+		1458 a102  GA-EP45-DS5/GA-EG45M-DS2H Motherboard
 	3a40  82801JI (ICH10 Family) PCI Express Root Port 1
 		1028 028c  PowerEdge R410 PCI Express Port 1
 		1028 028d  PowerEdge T410 PCI Express Port 1
+		103c 330b  ProLiant ML150 G6 Server
 		1043 82ea  P6T DeLuxe Motherboard
-		1458 5001  GA-EP45-DS5 Motherboard
+		1458 5001  GA-EP45-DS5/GA-EG45M-DS2H Motherboard
 	3a42  82801JI (ICH10 Family) PCI Express Port 2
 	3a44  82801JI (ICH10 Family) PCI Express Root Port 3
 		1043 82ea  P6T DeLuxe Motherboard
@@ -19062,11 +19348,13 @@
 		1043 82ea  P6T DeLuxe Motherboard
 		1458 5001  GA-EP45-DS5 Motherboard
 	3a48  82801JI (ICH10 Family) PCI Express Root Port 5
+		103c 330b  ProLiant ML150 G6 Server
 		1043 82ea  P6T Deluxe Motherboard
 		1458 5001  GA-EP45-DS5 Motherboard
 	3a4a  82801JI (ICH10 Family) PCI Express Root Port 6
+		103c 330b  ProLiant ML150 G6 Server
 		1043 82ea  P6T DeLuxe Motherboard
-		1458 5001  GA-EP45-DS5 Motherboard
+		1458 5001  GA-EP45-DS5/GA-EG45M-DS2H Motherboard
 	3a4c  82801JI (ICH10 Family) Gigabit Ethernet Controller
 	3a51  82801JDO (ICH10DO) VECI Controller
 	3a55  82801JD/DO (ICH10 Family) Virtual SATA Controller
@@ -19138,6 +19426,7 @@
 	3b2e  5 Series/3400 Series Chipset 4 port SATA IDE Controller
 		e4bf 50c1  PC1-GROOVE
 	3b2f  5 Series/3400 Series Chipset 6 port SATA AHCI Controller
+		1028 040b  Latitude E6510
 		e4bf 50c1  PC1-GROOVE
 	3b30  5 Series/3400 Series Chipset SMBus Controller
 		1025 0347  Aspire 7740G
@@ -19164,9 +19453,12 @@
 	3b40  5 Series/3400 Series Chipset USB Universal Host Controller
 	3b41  5 Series/3400 Series Chipset LAN Controller
 	3b42  5 Series/3400 Series Chipset PCI Express Root Port 1
+		1028 040b  Latitude E6510
 	3b44  5 Series/3400 Series Chipset PCI Express Root Port 2
+		1028 040b  Latitude E6510
 	3b46  5 Series/3400 Series Chipset PCI Express Root Port 3
 	3b48  5 Series/3400 Series Chipset PCI Express Root Port 4
+		1028 040b  Latitude E6510
 	3b4a  5 Series/3400 Series Chipset PCI Express Root Port 5
 	3b4c  5 Series/3400 Series Chipset PCI Express Root Port 6
 	3b4e  5 Series/3400 Series Chipset PCI Express Root Port 7
@@ -19184,85 +19476,81 @@
 	3b66  5 Series/3400 Series Chipset PT IDER Controller
 	3b67  5 Series/3400 Series Chipset KT Controller
 		e4bf 50c1  PC1-GROOVE
-	3c00  Sandy Bridge DMI2
-	3c01  Sandy Bridge DMI2 in PCI Express Mode
-	3c02  Sandy Bridge IIO PCI Express Root Port 1a
-	3c03  Sandy Bridge IIO PCI Express Root Port 1b
-	3c04  Sandy Bridge IIO PCI Express Root Port 2a
-	3c05  Sandy Bridge IIO PCI Express Root Port 2b
-	3c06  Sandy Bridge IIO PCI Express Root Port 2c
-	3c07  Sandy Bridge IIO PCI Express Root Port 2d
-	3c08  Sandy Bridge IIO PCI Express Root Port 3a in PCI Express Mode
-	3c09  Sandy Bridge IIO PCI Express Root Port 3b
-	3c0a  Sandy Bridge IIO PCI Express Root Port 3c
-	3c0b  Sandy Bridge IIO PCI Express Root Port 3d
-	3c0d  Sandy Bridge Non-Transparent Bridge
-	3c0e  Sandy Bridge Non-Transparent Bridge
-	3c0f  Sandy Bridge Non-Transparent Bridge
-	3c20  Sandy Bridge DMA Channel 0
-	3c21  Sandy Bridge DMA Channel 1
-	3c22  Sandy Bridge DMA Channel 2
-	3c23  Sandy Bridge DMA Channel 3
-	3c24  Sandy Bridge DMA Channel 4
-	3c25  Sandy Bridge DMA Channel 5
-	3c26  Sandy Bridge DMA Channel 6
-	3c27  Sandy Bridge DMA Channel 7
-	3c28  Sandy Bridge Address Map, VTd_Misc, System Management
-	3c2a  Sandy Bridge Control Status and Global Errors
-	3c2c  Sandy Bridge I/O APIC
-	3c2e  Sandy Bridge DMA
-	3c2f  Sandy Bridge DMA
-	3c40  Sandy Bridge IIO Switch and IRP Performance Monitor
-	3c41  Sandy Bridge QPI Port 0 Performance Monitor
-	3c42  Sandy Bridge QPI Port 1 Performance Monitor
-	3c43  Sandy Bridge Ring to PCI Express Performance Monitor
-	3c44  Sandy Bridge Ring to QuickPath Interconnect Link 0 Performance Monitor
-	3c45  Sandy Bridge Ring to QuickPath Interconnect Link 1 Performance Monitor
-	3c46  Sandy Bridge Processor Home Agent Performance Monitoring
-	3c71  Sandy Bridge Integrated Memory Controller RAS Registers
-	3c80  Sandy Bridge QPI Link 0
-	3c83  Sandy Bridge QPI Link Reut 0
-	3c84  Sandy Bridge QPI Link Reut 0
-	3c86  Sandy Bridge QPI Port 0 DFX Link
-	3c90  Sandy Bridge QPI Link 1
-	3c93  Sandy Bridge QPI Link Reut 1
-	3c94  Sandy Bridge QPI Link Reut 1
-	3c96  Sandy Bridge QPI Port 1 DFX Link
-	3ca0  Sandy Bridge Processor Home Agent
-	3ca8  Sandy Bridge Integrated Memory Controller Registers
-	3caa  Sandy Bridge Integrated Memory Controller Target Address Decoder 0
-	3cab  Sandy Bridge Integrated Memory Controller Target Address Decoder 1
-	3cac  Sandy Bridge Integrated Memory Controller Target Address Decoder 2
-	3cad  Sandy Bridge Integrated Memory Controller Target Address Decoder 3
-	3cae  Sandy Bridge Integrated Memory Controller Target Address Decoder 4
-	3cb0  Sandy Bridge Integrated Memory Controller Channel 0-3 Thermal Control 0
-	3cb1  Sandy Bridge Integrated Memory Controller Channel 0-3 Thermal Control 1
-	3cb2  Sandy Bridge Integrated Memory Controller ERROR Registers 0
-	3cb3  Sandy Bridge Integrated Memory Controller ERROR Registers 1
-	3cb4  Sandy Bridge Integrated Memory Controller Channel 0-3 Thermal Control 2
-	3cb5  Sandy Bridge Integrated Memory Controller Channel 0-3 Thermal Control 3
-	3cb6  Sandy Bridge Integrated Memory Controller ERROR Registers 2
-	3cb7  Sandy Bridge Integrated Memory Controller ERROR Registers 3
-	3cb8  Sandy Bridge DDRIO
-	3cc0  Sandy Bridge Power Control Unit 0
-	3cc1  Sandy Bridge Power Control Unit 1
-	3cc2  Sandy Bridge Power Control Unit 2
-	3cd0  Sandy Bridge Power Control Unit 3
-	3ce0  Sandy Bridge Interrupt Control Registers
-	3ce3  Sandy Bridge Semaphore and Scratchpad Configuration Registers
-	3ce4  Sandy Bridge R2PCIe
-	3ce6  Sandy Bridge QuickPath Interconnect Agent Ring Registers
-	3ce8  Sandy Bridge Unicast Register 0
-	3ce9  Sandy Bridge Unicast Register 5
-	3cea  Sandy Bridge Unicast Register 1
-	3ceb  Sandy Bridge Unicast Register 6
-	3cec  Sandy Bridge Unicast Register 3
-	3ced  Sandy Bridge Unicast Register 7
-	3cee  Sandy Bridge Unicast Register 4
-	3cef  Sandy Bridge Unicast Register 8
-	3cf4  Sandy Bridge Integrated Memory Controller System Address Decoder 0
-	3cf5  Sandy Bridge Integrated Memory Controller System Address Decoder 1
-	3cf6  Sandy Bridge System Address Decoder
+	3c00  Xeon E5/Core i7 DMI2
+	3c01  Xeon E5/Core i7 DMI2 in PCI Express Mode
+	3c02  Xeon E5/Core i7 IIO PCI Express Root Port 1a
+	3c03  Xeon E5/Core i7 IIO PCI Express Root Port 1b
+	3c04  Xeon E5/Core i7 IIO PCI Express Root Port 2a
+	3c05  Xeon E5/Core i7 IIO PCI Express Root Port 2b
+	3c06  Xeon E5/Core i7 IIO PCI Express Root Port 2c
+	3c07  Xeon E5/Core i7 IIO PCI Express Root Port 2d
+	3c08  Xeon E5/Core i7 IIO PCI Express Root Port 3a in PCI Express Mode
+	3c09  Xeon E5/Core i7 IIO PCI Express Root Port 3b
+	3c0a  Xeon E5/Core i7 IIO PCI Express Root Port 3c
+	3c0b  Xeon E5/Core i7 IIO PCI Express Root Port 3d
+	3c0d  Xeon E5/Core i7 Non-Transparent Bridge
+	3c0e  Xeon E5/Core i7 Non-Transparent Bridge
+	3c0f  Xeon E5/Core i7 Non-Transparent Bridge
+	3c20  Xeon E5/Core i7 DMA Channel 0
+	3c21  Xeon E5/Core i7 DMA Channel 1
+	3c22  Xeon E5/Core i7 DMA Channel 2
+	3c23  Xeon E5/Core i7 DMA Channel 3
+	3c24  Xeon E5/Core i7 DMA Channel 4
+	3c25  Xeon E5/Core i7 DMA Channel 5
+	3c26  Xeon E5/Core i7 DMA Channel 6
+	3c27  Xeon E5/Core i7 DMA Channel 7
+	3c28  Xeon E5/Core i7 Address Map, VTd_Misc, System Management
+	3c2a  Xeon E5/Core i7 Control Status and Global Errors
+	3c2c  Xeon E5/Core i7 I/O APIC
+	3c2e  Xeon E5/Core i7 DMA
+	3c2f  Xeon E5/Core i7 DMA
+	3c40  Xeon E5/Core i7 IIO Switch and IRP Performance Monitor
+	3c43  Xeon E5/Core i7 Ring to PCI Express Performance Monitor
+	3c44  Xeon E5/Core i7 Ring to QuickPath Interconnect Link 0 Performance Monitor
+	3c45  Xeon E5/Core i7 Ring to QuickPath Interconnect Link 1 Performance Monitor
+	3c46  Xeon E5/Core i7 Processor Home Agent Performance Monitoring
+	3c71  Xeon E5/Core i7 Integrated Memory Controller RAS Registers
+	3c80  Xeon E5/Core i7 QPI Link 0
+	3c83  Xeon E5/Core i7 QPI Link Reut 0
+	3c84  Xeon E5/Core i7 QPI Link Reut 0
+	3c90  Xeon E5/Core i7 QPI Link 1
+	3c93  Xeon E5/Core i7 QPI Link Reut 1
+	3c94  Xeon E5/Core i7 QPI Link Reut 1
+	3ca0  Xeon E5/Core i7 Processor Home Agent
+	3ca8  Xeon E5/Core i7 Integrated Memory Controller Registers
+	3caa  Xeon E5/Core i7 Integrated Memory Controller Target Address Decoder 0
+	3cab  Xeon E5/Core i7 Integrated Memory Controller Target Address Decoder 1
+	3cac  Xeon E5/Core i7 Integrated Memory Controller Target Address Decoder 2
+	3cad  Xeon E5/Core i7 Integrated Memory Controller Target Address Decoder 3
+	3cae  Xeon E5/Core i7 Integrated Memory Controller Target Address Decoder 4
+	3cb0  Xeon E5/Core i7 Integrated Memory Controller Channel 0-3 Thermal Control 0
+	3cb1  Xeon E5/Core i7 Integrated Memory Controller Channel 0-3 Thermal Control 1
+	3cb2  Xeon E5/Core i7 Integrated Memory Controller ERROR Registers 0
+	3cb3  Xeon E5/Core i7 Integrated Memory Controller ERROR Registers 1
+	3cb4  Xeon E5/Core i7 Integrated Memory Controller Channel 0-3 Thermal Control 2
+	3cb5  Xeon E5/Core i7 Integrated Memory Controller Channel 0-3 Thermal Control 3
+	3cb6  Xeon E5/Core i7 Integrated Memory Controller ERROR Registers 2
+	3cb7  Xeon E5/Core i7 Integrated Memory Controller ERROR Registers 3
+	3cb8  Xeon E5/Core i7 DDRIO
+	3cc0  Xeon E5/Core i7 Power Control Unit 0
+	3cc1  Xeon E5/Core i7 Power Control Unit 1
+	3cc2  Xeon E5/Core i7 Power Control Unit 2
+	3cd0  Xeon E5/Core i7 Power Control Unit 3
+	3ce0  Xeon E5/Core i7 Interrupt Control Registers
+	3ce3  Xeon E5/Core i7 Semaphore and Scratchpad Configuration Registers
+	3ce4  Xeon E5/Core i7 R2PCIe
+	3ce6  Xeon E5/Core i7 QuickPath Interconnect Agent Ring Registers
+	3ce8  Xeon E5/Core i7 Unicast Register 0
+	3ce9  Xeon E5/Core i7 Unicast Register 5
+	3cea  Xeon E5/Core i7 Unicast Register 1
+	3ceb  Xeon E5/Core i7 Unicast Register 6
+	3cec  Xeon E5/Core i7 Unicast Register 3
+	3ced  Xeon E5/Core i7 Unicast Register 7
+	3cee  Xeon E5/Core i7 Unicast Register 4
+	3cef  Xeon E5/Core i7 Unicast Register 8
+	3cf4  Xeon E5/Core i7 Integrated Memory Controller System Address Decoder 0
+	3cf5  Xeon E5/Core i7 Integrated Memory Controller System Address Decoder 1
+	3cf6  Xeon E5/Core i7 System Address Decoder
 	4000  5400 Chipset Memory Controller Hub
 	4001  5400 Chipset Memory Controller Hub
 	4003  5400 Chipset Memory Controller Hub
@@ -19604,6 +19892,80 @@
 	8817  Platform Controller Hub EG20T I2C Controller
 	8818  Platform Controller Hub EG20T Controller Area Network (CAN) Controller
 	8819  Platform Controller Hub EG20T IEEE 1588 Hardware Assist
+	8c00  Lynx Point 4-Port SATA IDE Controller
+	8c01  Lynx Point 4-Port SATA IDE Controller
+	8c02  Lynx Point 6-Port SATA AHCI Controller
+	8c03  Lynx Point 6-Port SATA AHCI Controller
+	8c04  Lynx Point SATA RAID Controller
+	8c05  Lynx Point SATA RAID Controller
+	8c06  Lynx Point SATA RAID Controller
+	8c07  Lynx Point SATA RAID Controller
+	8c08  Lynx Point 2-Port SATA IDE Controller
+	8c09  Lynx Point 2-Port SATA IDE Controller
+	8c0e  Lynx Point SATA RAID Controller
+	8c0f  Lynx Point SATA RAID Controller
+	8c10  Lynx Point PCI Express Root Port 1
+	8c11  Lynx Point PCI Express Root Port 1
+	8c12  Lynx Point PCI Express Root Port 2
+	8c13  Lynx Point PCI Express Root Port 2
+	8c14  Lynx Point PCI Express Root Port 3
+	8c15  Lynx Point PCI Express Root Port 3
+	8c16  Lynx Point PCI Express Root Port 4
+	8c17  Lynx Point PCI Express Root Port 4
+	8c18  Lynx Point PCI Express Root Port 5
+	8c19  Lynx Point PCI Express Root Port 5
+	8c1a  Lynx Point PCI Express Root Port 6
+	8c1b  Lynx Point PCI Express Root Port 6
+	8c1c  Lynx Point PCI Express Root Port 7
+	8c1d  Lynx Point PCI Express Root Port 7
+	8c1e  Lynx Point PCI Express Root Port 8
+	8c1f  Lynx Point PCI Express Root Port 8
+	8c20  Lynx Point HD Audio Controller
+	8c21  Lynx Point HD Audio Controller
+	8c22  Lynx Point SMBus Controller
+	8c23  Lynx Point CHAP Counter
+	8c24  Lynx Point Thermal Management Controller
+	8c26  Lynx Point USB Enhanced Host Controller #1
+	8c2d  Lynx Point USB Enhanced Host Controller #2
+	8c31  Lynx Point USB xHCI Host Controller
+	8c33  Lynx Point LAN Controller
+	8c34  Lynx Point NAND Controller
+	8c3a  Lynx Point MEI Controller #1
+	8c3b  Lynx Point MEI Controller #2
+	8c3c  Lynx Point IDE-r Controller
+	8c3d  Lynx Point KT Controller
+	8c40  Lynx Point LPC Controller
+	8c41  Lynx Point LPC Controller
+	8c42  Lynx Point LPC Controller
+	8c43  Lynx Point LPC Controller
+	8c44  Lynx Point LPC Controller
+	8c45  Lynx Point LPC Controller
+	8c46  Lynx Point LPC Controller
+	8c47  Lynx Point LPC Controller
+	8c48  Lynx Point LPC Controller
+	8c49  Lynx Point LPC Controller
+	8c4a  Lynx Point LPC Controller
+	8c4b  Lynx Point LPC Controller
+	8c4c  Lynx Point LPC Controller
+	8c4d  Lynx Point LPC Controller
+	8c4e  Lynx Point LPC Controller
+	8c4f  Lynx Point LPC Controller
+	8c50  Lynx Point LPC Controller
+	8c51  Lynx Point LPC Controller
+	8c52  Lynx Point LPC Controller
+	8c53  Lynx Point LPC Controller
+	8c54  Lynx Point LPC Controller
+	8c55  Lynx Point LPC Controller
+	8c56  Lynx Point LPC Controller
+	8c57  Lynx Point LPC Controller
+	8c58  Lynx Point LPC Controller
+	8c59  Lynx Point LPC Controller
+	8c5a  Lynx Point LPC Controller
+	8c5b  Lynx Point LPC Controller
+	8c5c  Lynx Point LPC Controller
+	8c5d  Lynx Point LPC Controller
+	8c5e  Lynx Point LPC Controller
+	8c5f  Lynx Point LPC Controller
 	9000  IXP2000 Family Network Processor
 	9001  IXP2400 Network Processor
 	9002  IXP2300 Network Processor
@@ -19644,6 +20006,7 @@
 	d136  Core Processor DMI
 	d137  Core Processor DMI
 	d138  Core Processor PCI Express Root Port 1
+		1028 040b  Latitude E6510
 	d139  Core Processor PCI Express Root Port 2
 	d13a  Core Processor PCI Express Root Port 3
 	d13b  Core Processor PCI Express Root Port 4
@@ -19964,6 +20327,31 @@
 		9005 02b3  2400
 		9005 02b4  ICP ICP5045AL
 		9005 0800  Callisto
+	028b  Series 6 products - SRC 8x6G SAS/ PCIe 2.0 [Tupelo]
+		9005 0200  Series 6 Entry Level - ASR-6405E - 4 internal 6G SAS ports
+		9005 0201  Series 6 Entry Level - ASR-6805E - 8 internal 6G SAS ports
+		9005 0300  Series 6 - ASR-6405 - 4 internal 6G SAS ports
+		9005 0301  Series 6 - ASR-6805 - 8 internal 6G SAS ports
+		9005 0302  Series 6 - ASR-6445 - 4 internal and 4 external 6G SAS ports
+		9005 0310  Series 6 Connectors on Top - ASR-6405T - 4 internal 6G SAS ports
+		9005 0311  Series 6 Connectors on Top - ASR-6805T - 8 internal 6G SAS
+		9005 0400  Series 6 - ASR-61205 - 12 internal 6G SAS ports
+		9005 0401  Series 6 - ASR-61605 - 16 internal 6G SAS ports
+		9005 0403  Series 6 - ASR-62405 - 24 internal 6G SAS ports
+	028c  Series 7 products - SRCv 16x6G SAS/ PCIe 3.0 [Denali]
+		9005 0500  Series 7 - ASR-7805 - 8 internal 6G SAS Port/PCIe 3.0
+		9005 0501  Series 7 - ASR-71605 - 16 internal 6G SAS Port/PCIe 3.0
+		9005 0502  Series 7 - ASR-71685 - 16 internal 8 external 6G SAS Port/PCIe 3.0
+		9005 0503  Series 7 - ASR-72405 - 24 internal 0 external 6G SAS Port/PCIe 3.0
+		9005 0504  Series 7 - ASR-7885 - 8 internal 8 external 6G SAS Port/PCIe 3.0
+		9005 0505  Series 7 Entry Level - ASR-71685E - 16 internal 8 external 6G SAS Port/PCIe 3.0
+		9005 0506  Series 7 Entry Level - ASR-72405E - 24 internal 0 external 6G SAS Port/PCIe 3.0
+	028d  Series 8 products - ASIC 16x12G SAS/PCIe 3.0 [Yosemite]
+		9005 0550  Series 8 - ASR-82405 - 24 internal 0 external 12G SAS Port/PCIe 3.0
+		9005 0551  Series 8 - ASR-81605 - 16 internal 0 external 12G SAS Port/PCIe 3.0
+		9005 0552  Series 8 - ASR-8805 - 8 internal 0 external 12G SAS Port/PCIe 3.0
+		9005 0553  Series 8 - ASR-8085 - 0 internal 8 external 12G SAS Port/PCIe 3.0
+		9005 0554  Series 8 - ASR-8885 - 8 internal 8 external 12G SAS Port/PCIe 3.0
 	0410  AIC-9410W SAS (Razor HBA RAID)
 		9005 0410  ASC-48300(Spirit RAID)
 		9005 0411  ASC-58300 (Oakmont RAID)
@@ -20032,6 +20420,8 @@
 919a  Gigapixel Corp
 9412  Holtek
 	6565  6565
+9618  JusonTech Corporation
+	0001  JusonTech Gigabit Ethernet Controller
 9699  Omni Media Technology Inc
 	6565  6565
 9710  NetMos Technology
@@ -20057,6 +20447,8 @@
 	9865  PCI 9865 Multi-I/O Controller
 	9901  PCIe 9901 Multi-I/O Controller
 	9904  4-Port PCIe Serial Adapter
+# 2-port Serial 1-port Parallel Adaptor
+	9912  PCIe 9912 Multi-I/O Controller
 	9922  PCIe 9922 Multi-I/O Controller
 	9990  MCS9990 PCIe to 4‐Port USB 2.0 Host Controller
 9902  Stargen Inc.
@@ -20107,6 +20499,7 @@ cace  CACE Technologies, Inc.
 	0023  AirPcap N
 cafe  Chrysalis-ITS
 	0003  Luna K3 Hardware Security Module
+	0006  Luna PCI-e 3000 Hardware Security Module
 cccc  Catapult Communications
 ccec  Curtiss-Wright Controls Embedded Computing
 cddd  Tyzx, Inc.
@@ -20167,6 +20560,15 @@ e159  Tiger Jet Network Inc.
 	0002  Tiger100APC ISDN chipset
 e1c5  Elcus
 e4bf  EKF Elektronik GmbH
+	0ccd  CCD-CALYPSO
+	0cd1  CD1-OPERA
+	0cd2  CD2-BEBOP
+	0cd3  CD3-JIVE
+	50c1  PC1-GROOVE
+	50c2  PC2-LIMBO
+	53c1  SC1-ALLEGRO
+	cc47  CCG-RUMBA
+	cc4d  CCM-BOOGIE
 e55e  Essence Technology, Inc.
 ea01  Eagle Technology
 	000a  PCI-773 Temperature Card
