http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=674708

From: Bjarni Ingi Gislason <bjarniig@rhi.hi.is>

--- a/setpci.man	2012-05-25 21:07:04.000000000 +0000
+++ b/setpci.man	2012-05-25 22:07:36.000000000 +0000
@@ -1,5 +1,5 @@
 .TH setpci 8 "@TODAY@" "@VERSION@" "The PCI Utilities"
-.IX setpci
+.\".IX setpci
 .SH NAME
 setpci \- configure PCI devices
 .SH SYNOPSIS
@@ -115,7 +115,7 @@
 .PP
 There are two kinds of operations: reads and writes. To read a register, just specify
 its name. Writes have the form
-.IR name = value , value ...
+.IR name = value , value ...\&
 where each
 .I value
 is either a hexadecimal number or an expression of type
