http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=674708

From: Bjarni Ingi Gislason <bjarniig@rhi.hi.is>

Index: pciutils-3.2.1/setpci.man
===================================================================
--- pciutils-3.2.1.orig/setpci.man
+++ pciutils-3.2.1/setpci.man
@@ -1,5 +1,5 @@
 .TH setpci 8 "@TODAY@" "@VERSION@" "The PCI Utilities"
-.IX setpci
+.\".IX setpci
 .SH NAME
 setpci \- configure PCI devices
 .SH SYNOPSIS
@@ -115,7 +115,7 @@ options of the same kind are specified,
 .PP
 There are two kinds of operations: reads and writes. To read a register, just specify
 its name. Writes have the form
-.IR name = value , value ...
+.IR name = value , value ...\&
 where each
 .I value
 is either a hexadecimal number or an expression of type
